<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        
        
        <link rel="canonical" href="https://genomicsstandardsconsortium.github.io/mixs/SoilMIMARKSSpecimen/">
        <link rel="shortcut icon" href="../img/favicon.ico">
        <title>Class: soil MIMARKS specimen - MIxS</title>
        <link href="../css/bootstrap.min.css" rel="stylesheet">
        <link href="../css/font-awesome.min.css" rel="stylesheet">
        <link href="../css/base.css" rel="stylesheet">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/styles/github.min.css">

        <script src="../js/jquery-1.10.2.min.js" defer></script>
        <script src="../js/bootstrap.min.js" defer></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/highlight.min.js"></script>
        <script>hljs.initHighlightingOnLoad();</script> 
    </head>

    <body>
        <div class="navbar fixed-top navbar-expand-lg navbar-dark bg-dark">
            <div class="container">
                <a class="navbar-brand" href="..">MIxS</a>
                <!-- Expander button -->
                <button type="button" class="navbar-toggler" data-toggle="collapse" data-target="#navbar-collapse">
                    <span class="navbar-toggler-icon"></span>
                </button>

                <!-- Expanded navigation -->
                <div id="navbar-collapse" class="navbar-collapse collapse">
                        <!-- Main navigation -->
                        <ul class="nav navbar-nav">
                            <li class="navitem">
                                <a href=".." class="nav-link">About</a>
                            </li>
                            <li class="dropdown">
                                <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown">Introduction <b class="caret"></b></a>
                                <ul class="dropdown-menu">
                                    
<li>
    <a href="../introduction/background/" class="dropdown-item">Background</a>
</li>
                                    
<li>
    <a href="../introduction/citation/" class="dropdown-item">Cite</a>
</li>
                                    
<li>
    <a href="../introduction/contents/" class="dropdown-item">Contents</a>
</li>
                                </ul>
                            </li>
                        </ul>

                    <ul class="nav navbar-nav ml-auto">
                        <li class="nav-item">
                            <a href="#" class="nav-link" data-toggle="modal" data-target="#mkdocs_search_modal">
                                <i class="fa fa-search"></i> Search
                            </a>
                        </li>
                            <li class="nav-item">
                                <a href="https://github.com/GenomicsStandardsConsortium/mixs/edit/master/docs/SoilMIMARKSSpecimen.md" class="nav-link"><i class="fa fa-github"></i> Edit on GitHub</a>
                            </li>
                    </ul>
                </div>
            </div>
        </div>

        <div class="container">
            <div class="row">
                    <div class="col-md-3"><div class="navbar-light navbar-expand-md bs-sidebar hidden-print affix" role="complementary">
    <div class="navbar-header">
        <button type="button" class="navbar-toggler collapsed" data-toggle="collapse" data-target="#toc-collapse" title="Table of Contents">
            <span class="fa fa-angle-down"></span>
        </button>
    </div>

    
    <div id="toc-collapse" class="navbar-collapse collapse card bg-secondary">
        <ul class="nav flex-column">
            
            <li class="nav-item" data-level="1"><a href="#class-soil-mimarks-specimen" class="nav-link">Class: soil MIMARKS specimen</a>
              <ul class="nav flex-column">
            <li class="nav-item" data-level="2"><a href="#parents" class="nav-link">Parents</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#uses-checklist" class="nav-link">Uses Checklist</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#attributes" class="nav-link">Attributes</a>
              <ul class="nav flex-column">
              </ul>
            </li>
              </ul>
            </li>
        </ul>
    </div>
</div></div>
                    <div class="col-md-9" role="main">

<h1 id="class-soil-mimarks-specimen">Class: soil MIMARKS specimen</h1>
<p>Combinatorial checklist Minimal Information about a Marker Specimen: specimen with environmental package soil</p>
<p>URI: <a href="https://w3id.org/mixs/vocab/SoilMIMARKSSpecimen">mixs.vocab:SoilMIMARKSSpecimen</a></p>
<p><a href="https://yuml.me/diagram/nofunky;dir:TB/class/[SoilMIMARKSSpecimen&#124;samp_taxon_id:string;experimental_factor:string%20%3F;neg_cont_type:neg_cont_type_enum%20%3F;pos_cont_type:string%20%3F;source_mat_id:string%20%3F;rel_to_oxygen:rel_to_oxygen_enum%20%3F;samp_collec_device:string%20%3F;samp_collec_method:string%20%3F;samp_mat_process:string%20%3F;size_frac:string%20%3F;nucl_acid_ext:string%20%3F;nucl_acid_amp:string%20%3F;lib_size:integer%20%3F;lib_reads_seqd:integer%20%3F;lib_layout:lib_layout_enum%20%3F;lib_vector:string%20%3F;lib_screen:string%20%3F;target_gene:string;target_subfragment:string%20%3F;pcr_primers:string%20%3F;mid:string%20%3F;adapters:string%20%3F;pcr_cond:string%20%3F;seq_meth:string;seq_quality_check:string%20%3F;chimera_check:string%20%3F;assembly_software:string%20%3F;associated_resource:string%20%3F;sop:string%20%3F;lat_lon(i):string%20%3F;geo_loc_name(i):string%20%3F;collection_date(i):date%20%3F;env_broad_scale(i):string%20%3F;env_local_scale(i):string%20%3F;env_medium(i):string%20%3F;cur_land_use(i):cur_land_use_enum%20%3F;cur_vegetation(i):string%20%3F;cur_vegetation_meth(i):string%20%3F;previous_land_use(i):string%20%3F;prev_land_use_meth(i):string%20%3F;crop_rotation(i):string%20%3F;agrochem_addition(i):string%20*;tillage(i):tillage_enum%20*;fire(i):date%20%3F;flooding(i):date%20%3F;extreme_event(i):date%20%3F;soil_horizon(i):soil_horizon_enum%20%3F;horizon_meth(i):string%20%3F;sieving(i):string%20%3F;water_cont_soil_meth(i):string%20%3F;pool_dna_extracts(i):string%20%3F;store_cond(i):string%20%3F;link_climate_info(i):string%20%3F;link_class_info(i):string%20%3F;fao_class(i):fao_class_enum%20%3F;local_class(i):string%20%3F;local_class_meth(i):string%20%3F;soil_type(i):string%20%3F;soil_type_meth(i):string%20%3F;profile_position(i):profile_position_enum%20%3F;drainage_class(i):drainage_class_enum%20%3F;soil_texture_meth(i):string%20%3F;ph(i):double%20%3F;ph_meth(i):string%20%3F;tot_org_c_meth(i):string%20%3F;tot_nitro_cont_meth(i):string%20%3F;micro_biomass_meth(i):string%20%3F;link_addit_analys(i):string%20%3F;salinity_meth(i):string%20%3F;heavy_metals(i):string%20*;heavy_metals_meth(i):string%20%3F;al_sat_meth(i):string%20%3F;misc_param(i):string%20*;samp_name(i):string;project_name(i):string]uses%20-.-&gt;[MIMARKSSpecimen],[Soil]^-[SoilMIMARKSSpecimen],[Soil],[QuantityValue],[MIMARKSSpecimen]"><img alt="img" src="https://yuml.me/diagram/nofunky;dir:TB/class/[SoilMIMARKSSpecimen&#124;samp_taxon_id:string;experimental_factor:string%20%3F;neg_cont_type:neg_cont_type_enum%20%3F;pos_cont_type:string%20%3F;source_mat_id:string%20%3F;rel_to_oxygen:rel_to_oxygen_enum%20%3F;samp_collec_device:string%20%3F;samp_collec_method:string%20%3F;samp_mat_process:string%20%3F;size_frac:string%20%3F;nucl_acid_ext:string%20%3F;nucl_acid_amp:string%20%3F;lib_size:integer%20%3F;lib_reads_seqd:integer%20%3F;lib_layout:lib_layout_enum%20%3F;lib_vector:string%20%3F;lib_screen:string%20%3F;target_gene:string;target_subfragment:string%20%3F;pcr_primers:string%20%3F;mid:string%20%3F;adapters:string%20%3F;pcr_cond:string%20%3F;seq_meth:string;seq_quality_check:string%20%3F;chimera_check:string%20%3F;assembly_software:string%20%3F;associated_resource:string%20%3F;sop:string%20%3F;lat_lon(i):string%20%3F;geo_loc_name(i):string%20%3F;collection_date(i):date%20%3F;env_broad_scale(i):string%20%3F;env_local_scale(i):string%20%3F;env_medium(i):string%20%3F;cur_land_use(i):cur_land_use_enum%20%3F;cur_vegetation(i):string%20%3F;cur_vegetation_meth(i):string%20%3F;previous_land_use(i):string%20%3F;prev_land_use_meth(i):string%20%3F;crop_rotation(i):string%20%3F;agrochem_addition(i):string%20*;tillage(i):tillage_enum%20*;fire(i):date%20%3F;flooding(i):date%20%3F;extreme_event(i):date%20%3F;soil_horizon(i):soil_horizon_enum%20%3F;horizon_meth(i):string%20%3F;sieving(i):string%20%3F;water_cont_soil_meth(i):string%20%3F;pool_dna_extracts(i):string%20%3F;store_cond(i):string%20%3F;link_climate_info(i):string%20%3F;link_class_info(i):string%20%3F;fao_class(i):fao_class_enum%20%3F;local_class(i):string%20%3F;local_class_meth(i):string%20%3F;soil_type(i):string%20%3F;soil_type_meth(i):string%20%3F;profile_position(i):profile_position_enum%20%3F;drainage_class(i):drainage_class_enum%20%3F;soil_texture_meth(i):string%20%3F;ph(i):double%20%3F;ph_meth(i):string%20%3F;tot_org_c_meth(i):string%20%3F;tot_nitro_cont_meth(i):string%20%3F;micro_biomass_meth(i):string%20%3F;link_addit_analys(i):string%20%3F;salinity_meth(i):string%20%3F;heavy_metals(i):string%20*;heavy_metals_meth(i):string%20%3F;al_sat_meth(i):string%20%3F;misc_param(i):string%20*;samp_name(i):string;project_name(i):string]uses%20-.-&gt;[MIMARKSSpecimen],[Soil]^-[SoilMIMARKSSpecimen],[Soil],[QuantityValue],[MIMARKSSpecimen]" /></a></p>
<h2 id="parents">Parents</h2>
<ul>
<li>is_a: <a href="../Soil/">Soil</a> - soil</li>
</ul>
<h2 id="uses-checklist">Uses Checklist</h2>
<ul>
<li>mixin: <a href="../MIMARKSSpecimen/">MIMARKSSpecimen</a> - Minimal Information about a Marker Specimen: specimen</li>
</ul>
<h2 id="attributes">Attributes</h2>
<h3 id="inherited-from-soil">Inherited from soil:</h3>
<ul>
<li><a href="../lat_lon/">lat_lon</a>  <sub>0..1</sub><ul>
<li>Description: The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 50.586825 6.408977 None</li>
</ul>
</li>
<li><a href="../soil_depth/">soil➞depth</a>  <sub>1..1</sub><ul>
<li>Description: The vertical distance below local surface, e.g. for sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 10 meter None</li>
</ul>
</li>
<li><a href="../alt/">alt</a>  <sub>0..1</sub><ul>
<li>Description: Altitude is a term used to identify heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth's surface. In this context, the altitude measurement is the vertical distance between the earth's surface above sea level and the sampled position in the air</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 100 meter None</li>
</ul>
</li>
<li><a href="../soil_elev/">soil➞elev</a>  <sub>1..1</sub><ul>
<li>Description: Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth's surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 100 meter None</li>
</ul>
</li>
<li><a href="../temp/">temp</a>  <sub>0..1</sub><ul>
<li>Description: Temperature of the sample at the time of sampling.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 25 degree Celsius None</li>
</ul>
</li>
<li><a href="../geo_loc_name/">geo_loc_name</a>  <sub>0..1</sub><ul>
<li>Description: The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: USA: Maryland, Bethesda None</li>
</ul>
</li>
<li><a href="../collection_date/">collection_date</a>  <sub>0..1</sub><ul>
<li>Description: The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant</li>
<li>Range: <a href="../types/Date/">Date</a></li>
<li>Example: 2018-05-11T10:00:00+01:00; 2018-05-11 None</li>
</ul>
</li>
<li><a href="../env_broad_scale/">env_broad_scale</a>  <sub>0..1</sub><ul>
<li>Description: Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO’s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: oceanic epipelagic zone biome [ENVO:01000033] for annotating a water sample from the photic zone in middle of the Atlantic Ocean None</li>
</ul>
</li>
<li><a href="../env_local_scale/">env_local_scale</a>  <sub>0..1</sub><ul>
<li>Description: Report the entity or entities which are in the sample or specimen’s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: litter layer [ENVO:01000338]; Annotating a pooled sample taken from various vegetation layers in a forest consider: canopy [ENVO:00000047]|herb and fern layer [ENVO:01000337]|litter layer [ENVO:01000338]|understory [01000335]|shrub layer [ENVO:01000336]. None</li>
</ul>
</li>
<li><a href="../env_medium/">env_medium</a>  <sub>0..1</sub><ul>
<li>Description: Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of 'environmental material' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: soil [ENVO:00001998]; Annotating a fish swimming in the upper 100 m of the Atlantic Ocean, consider: ocean water [ENVO:00002151]. Example: Annotating a duck on a pond consider: pond water [ENVO:00002228]|air [ENVO_00002005] None</li>
</ul>
</li>
<li><a href="../soil_cur_land_use/">soil➞cur_land_use</a>  <sub>0..1</sub><ul>
<li>Description: Present state of sample site</li>
<li>Range: <a href="../cur_land_use_enum/">cur_land_use_enum</a></li>
<li>Example: conifers None</li>
</ul>
</li>
<li><a href="../soil_cur_vegetation/">soil➞cur_vegetation</a>  <sub>0..1</sub><ul>
<li>Description: Vegetation classification from one or more standard classification systems, or agricultural crop</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_cur_vegetation_meth/">soil➞cur_vegetation_meth</a>  <sub>0..1</sub><ul>
<li>Description: Reference or method used in vegetation classification</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_previous_land_use/">soil➞previous_land_use</a>  <sub>0..1</sub><ul>
<li>Description: Previous land use and dates</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_prev_land_use_meth/">soil➞prev_land_use_meth</a>  <sub>0..1</sub><ul>
<li>Description: Reference or method used in determining previous land use and dates</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_crop_rotation/">soil➞crop_rotation</a>  <sub>0..1</sub><ul>
<li>Description: Whether or not crop is rotated, and if yes, rotation schedule</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: yes;R2/2017-01-01/2018-12-31/P6M None</li>
</ul>
</li>
<li><a href="../soil_agrochem_addition/">soil➞agrochem_addition</a>  <sub>0..*</sub><ul>
<li>Description: Addition of fertilizers, pesticides, etc. - amount and time of applications</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: roundup;5 milligram per liter;2018-06-21 None</li>
</ul>
</li>
<li><a href="../soil_tillage/">soil➞tillage</a>  <sub>0..*</sub><ul>
<li>Description: Note method(s) used for tilling</li>
<li>Range: <a href="../tillage_enum/">tillage_enum</a></li>
<li>Example: chisel None</li>
</ul>
</li>
<li><a href="../soil_fire/">soil➞fire</a>  <sub>0..1</sub><ul>
<li>Description: Historical and/or physical evidence of fire</li>
<li>Range: <a href="../types/Date/">Date</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_flooding/">soil➞flooding</a>  <sub>0..1</sub><ul>
<li>Description: Historical and/or physical evidence of flooding</li>
<li>Range: <a href="../types/Date/">Date</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_extreme_event/">soil➞extreme_event</a>  <sub>0..1</sub><ul>
<li>Description: Unusual physical events that may have affected microbial populations</li>
<li>Range: <a href="../types/Date/">Date</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_soil_horizon/">soil➞soil_horizon</a>  <sub>0..1</sub><ul>
<li>Description: Specific layer in the land area which measures parallel to the soil surface and possesses physical characteristics which differ from the layers above and beneath</li>
<li>Range: <a href="../soil_horizon_enum/">soil_horizon_enum</a></li>
<li>Example: A horizon None</li>
</ul>
</li>
<li><a href="../soil_horizon_meth/">soil➞horizon_meth</a>  <sub>0..1</sub><ul>
<li>Description: Reference or method used in determining the horizon</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_sieving/">soil➞sieving</a>  <sub>0..1</sub><ul>
<li>Description: Collection design of pooled samples and/or sieve size and amount of sample sieved</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_water_content/">soil➞water_content</a>  <sub>0..1</sub><ul>
<li>Description: Water content measurement</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_water_cont_soil_meth/">soil➞water_cont_soil_meth</a>  <sub>0..1</sub><ul>
<li>Description: Reference or method used in determining the water content of soil</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_pool_dna_extracts/">soil➞pool_dna_extracts</a>  <sub>0..1</sub><ul>
<li>Description: Indicate whether multiple DNA extractions were mixed. If the answer yes, the number of extracts that were pooled should be given</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: yes;5 None</li>
</ul>
</li>
<li><a href="../soil_store_cond/">soil➞store_cond</a>  <sub>0..1</sub><ul>
<li>Description: Explain how and for how long the soil sample was stored before DNA extraction (fresh/frozen/other).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: -20 degree Celsius freezer;P2Y10D None</li>
</ul>
</li>
<li><a href="../soil_link_climate_info/">soil➞link_climate_info</a>  <sub>0..1</sub><ul>
<li>Description: Link to climate resource</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_annual_temp/">soil➞annual_temp</a>  <sub>0..1</sub><ul>
<li>Description: Mean annual temperature</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 12.5 degree Celsius None</li>
</ul>
</li>
<li><a href="../soil_season_temp/">soil➞season_temp</a>  <sub>0..1</sub><ul>
<li>Description: Mean seasonal temperature</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 18 degree Celsius None</li>
</ul>
</li>
<li><a href="../soil_annual_precpt/">soil➞annual_precpt</a>  <sub>0..1</sub><ul>
<li>Description: The average of all annual precipitation values known, or an estimated equivalent value derived by such methods as regional indexes or Isohyetal maps.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_season_precpt/">soil➞season_precpt</a>  <sub>0..1</sub><ul>
<li>Description: The average of all seasonal precipitation values known, or an estimated equivalent value derived by such methods as regional indexes or Isohyetal maps.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_link_class_info/">soil➞link_class_info</a>  <sub>0..1</sub><ul>
<li>Description: Link to digitized soil maps or other soil classification information</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_fao_class/">soil➞fao_class</a>  <sub>0..1</sub><ul>
<li>Description: Soil classification from the FAO World Reference Database for Soil Resources. The list can be found at http://www.fao.org/nr/land/sols/soil/wrb-soil-maps/reference-groups</li>
<li>Range: <a href="../fao_class_enum/">fao_class_enum</a></li>
<li>Example: Luvisols None</li>
</ul>
</li>
<li><a href="../soil_local_class/">soil➞local_class</a>  <sub>0..1</sub><ul>
<li>Description: Soil classification based on local soil classification system</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_local_class_meth/">soil➞local_class_meth</a>  <sub>0..1</sub><ul>
<li>Description: Reference or method used in determining the local soil classification</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_soil_type/">soil➞soil_type</a>  <sub>0..1</sub><ul>
<li>Description: Description of the soil type or classification. This field accepts terms under soil (http://purl.obolibrary.org/obo/ENVO_00001998).  Multiple terms can be separated by pipes.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: plinthosol [ENVO:00002250] None</li>
</ul>
</li>
<li><a href="../soil_soil_type_meth/">soil➞soil_type_meth</a>  <sub>0..1</sub><ul>
<li>Description: Reference or method used in determining soil series name or other lower-level classification</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_slope_gradient/">soil➞slope_gradient</a>  <sub>0..1</sub><ul>
<li>Description: Commonly called 'slope'. The angle between ground surface and a horizontal line (in percent). This is the direction that overland water would flow. This measure is usually taken with a hand level meter or clinometer</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_slope_aspect/">soil➞slope_aspect</a>  <sub>0..1</sub><ul>
<li>Description: The direction a slope faces. While looking down a slope use a compass to record the direction you are facing (direction or degrees); e.g., nw or 315 degrees. This measure provides an indication of sun and wind exposure that will influence soil temperature and evapotranspiration.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_profile_position/">soil➞profile_position</a>  <sub>0..1</sub><ul>
<li>Description: Cross-sectional position in the hillslope where sample was collected.sample area position in relation to surrounding areas</li>
<li>Range: <a href="../profile_position_enum/">profile_position_enum</a></li>
<li>Example: summit None</li>
</ul>
</li>
<li><a href="../soil_drainage_class/">soil➞drainage_class</a>  <sub>0..1</sub><ul>
<li>Description: Drainage classification from a standard system such as the USDA system</li>
<li>Range: <a href="../drainage_class_enum/">drainage_class_enum</a></li>
<li>Example: well None</li>
</ul>
</li>
<li><a href="../soil_soil_text_measure/">soil➞soil_text_measure</a>  <sub>0..1</sub><ul>
<li>Description: The relative proportion of different grain sizes of mineral particles in a soil, as described using a standard system; express as % sand (50 um to 2 mm), silt (2 um to 50 um), and clay (&lt;2 um) with textural name (e.g., silty clay loam) optional.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_soil_texture_meth/">soil➞soil_texture_meth</a>  <sub>0..1</sub><ul>
<li>Description: Reference or method used in determining soil texture</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="soil_ph.md">soil➞ph</a>  <sub>0..1</sub><ul>
<li>Description: Ph measurement of the sample, or liquid portion of sample, or aqueous phase of the fluid</li>
<li>Range: <a href="../types/Double/">Double</a></li>
<li>Example: 7.2 None</li>
</ul>
</li>
<li><a href="../soil_ph_meth/">soil➞ph_meth</a>  <sub>0..1</sub><ul>
<li>Description: Reference or method used in determining ph</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_tot_org_carb/">soil➞tot_org_carb</a>  <sub>0..1</sub><ul>
<li>Description: Definition for soil: total organic carbon content of the soil, definition otherwise: total organic carbon content</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_tot_org_c_meth/">soil➞tot_org_c_meth</a>  <sub>0..1</sub><ul>
<li>Description: Reference or method used in determining total organic carbon</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_tot_nitro_content/">soil➞tot_nitro_content</a>  <sub>0..1</sub><ul>
<li>Description: Total nitrogen content of the sample</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_tot_nitro_cont_meth/">soil➞tot_nitro_cont_meth</a>  <sub>0..1</sub><ul>
<li>Description: Reference or method used in determining the total nitrogen</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_microbial_biomass/">soil➞microbial_biomass</a>  <sub>0..1</sub><ul>
<li>Description: The part of the organic matter in the soil that constitutes living microorganisms smaller than 5-10 micrometer. If you keep this, you would need to have correction factors used for conversion to the final units</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_micro_biomass_meth/">soil➞micro_biomass_meth</a>  <sub>0..1</sub><ul>
<li>Description: Reference or method used in determining microbial biomass</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_link_addit_analys/">soil➞link_addit_analys</a>  <sub>0..1</sub><ul>
<li>Description: Link to additional analysis results performed on the sample</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_salinity_meth/">soil➞salinity_meth</a>  <sub>0..1</sub><ul>
<li>Description: Reference or method used in determining salinity</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_heavy_metals/">soil➞heavy_metals</a>  <sub>0..*</sub><ul>
<li>Description: Heavy metals present in the sequenced sample and their concentrations. For multiple heavy metals and concentrations, add multiple copies of this field.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: mercury;0.09 micrograms per gram None</li>
</ul>
</li>
<li><a href="../soil_heavy_metals_meth/">soil➞heavy_metals_meth</a>  <sub>0..1</sub><ul>
<li>Description: Reference or method used in determining heavy metals</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_al_sat/">soil➞al_sat</a>  <sub>0..1</sub><ul>
<li>Description: Aluminum saturation (esp. For tropical soils)</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_al_sat_meth/">soil➞al_sat_meth</a>  <sub>0..1</sub><ul>
<li>Description: Reference or method used in determining Al saturation</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../soil_misc_param/">soil➞misc_param</a>  <sub>0..*</sub><ul>
<li>Description: Any other measurement performed or parameter collected, that is not listed here</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Bicarbonate ion concentration;2075 micromole per kilogram None</li>
</ul>
</li>
<li><a href="../soil_samp_name/">soil➞samp_name</a>  <sub>1..1</sub><ul>
<li>Description: A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: ISDsoil1 None</li>
</ul>
</li>
<li><a href="../soil_project_name/">soil➞project_name</a>  <sub>1..1</sub><ul>
<li>Description: Name of the project within which the sequencing was organized</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Forest soil metagenome None</li>
</ul>
</li>
<li><a href="../soil_samp_vol_we_dna_ext/">soil➞samp_vol_we_dna_ext</a>  <sub>0..1</sub><ul>
<li>Description: Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001).</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 1500 milliliter None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_samp_taxon_id/">MIMARKS specimen➞samp_taxon_id</a>  <sub>1..1</sub><ul>
<li>Description: NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa sample. Use 'synthetic metagenome’ for mock community/positive controls, or 'blank sample' for negative controls.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Gut Metagenome [NCBI:txid749906] None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_1">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_experimental_factor/">MIMARKS specimen➞experimental_factor</a>  <sub>0..1</sub><ul>
<li>Description: Experimental factors are essentially the variable aspects of an experiment design which can be used to describe an experiment, or set of experiments, in an increasingly detailed manner. This field accepts ontology terms from Experimental Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI). For a browser of EFO (v 2.95) terms, please see http://purl.bioontology.org/ontology/EFO; for a browser of OBI (v 2018-02-12) terms please see http://purl.bioontology.org/ontology/OBI</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: time series design [EFO:EFO_0001779] None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_2">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_neg_cont_type/">MIMARKS specimen➞neg_cont_type</a>  <sub>0..1</sub><ul>
<li>Description: The substance or equipment used as a negative control in an investigation</li>
<li>Range: <a href="../neg_cont_type_enum/">neg_cont_type_enum</a></li>
<li>Example:  None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_3">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_pos_cont_type/">MIMARKS specimen➞pos_cont_type</a>  <sub>0..1</sub><ul>
<li>Description: The substance, mixture, product, or apparatus used to verify that a process which is part of an investigation delivers a true positive.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_4">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_source_mat_id/">MIMARKS specimen➞source_mat_id</a>  <sub>0..1</sub><ul>
<li>Description: A unique identifier assigned to a material sample (as defined by http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular digital record of a material sample) used for extracting nucleic acids, and subsequent sequencing. The identifier can refer either to the original material collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher, /bio_material, or /culture_collection may or may not share the same value as the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id may both contain 'UAM:Herps:14' , referring to both the specimen voucher and sampled tissue with the same identifier. However, the /culture_collection qualifier may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id would refer to an identifier from some derived culture from which the nucleic acids were extracted (e.g. xatc123 or ark:/2154/R2).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: MPI012345 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_5">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_rel_to_oxygen/">MIMARKS specimen➞rel_to_oxygen</a>  <sub>0..1</sub><ul>
<li>Description: Is this organism an aerobe, anaerobe? Please note that aerobic and anaerobic are valid descriptors for microbial environments</li>
<li>Range: <a href="../rel_to_oxygen_enum/">rel_to_oxygen_enum</a></li>
<li>Example: aerobe None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_6">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_samp_collec_device/">MIMARKS specimen➞samp_collec_device</a>  <sub>0..1</sub><ul>
<li>Description: The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: swab, biopsy, niskin bottle, push core, drag swab [GENEPIO:0002713] None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_7">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_samp_collec_method/">MIMARKS specimen➞samp_collec_method</a>  <sub>0..1</sub><ul>
<li>Description: The method employed for collecting the sample.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: swabbing None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_8">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_samp_mat_process/">MIMARKS specimen➞samp_mat_process</a>  <sub>0..1</sub><ul>
<li>Description: A brief description of any processing applied to the sample during or after retrieving the sample from environment, or a link to the relevant protocol(s) performed.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: filtering of seawater, storing samples in ethanol None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_9">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_size_frac/">MIMARKS specimen➞size_frac</a>  <sub>0..1</sub><ul>
<li>Description: Filtering pore size used in sample preparation</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 0-0.22 micrometer None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_10">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_samp_size/">MIMARKS specimen➞samp_size</a>  <sub>0..1</sub><ul>
<li>Description: The total amount or size (volume (ml), mass (g) or area (m2) ) of sample collected.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 5 liter None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_11">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_nucl_acid_ext/">MIMARKS specimen➞nucl_acid_ext</a>  <sub>0..1</sub><ul>
<li>Description: A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the material separation to recover the nucleic acid fraction from a sample</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: https://mobio.com/media/wysiwyg/pdfs/protocols/12888.pdf None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_12">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_nucl_acid_amp/">MIMARKS specimen➞nucl_acid_amp</a>  <sub>0..1</sub><ul>
<li>Description: A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the enzymatic amplification (PCR, TMA, NASBA) of specific nucleic acids</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: https://phylogenomics.me/protocols/16s-pcr-protocol/ None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_13">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_lib_size/">MIMARKS specimen➞lib_size</a>  <sub>0..1</sub><ul>
<li>Description: Total number of clones in the library prepared for the project</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 50 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_14">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_lib_reads_seqd/">MIMARKS specimen➞lib_reads_seqd</a>  <sub>0..1</sub><ul>
<li>Description: Total number of clones sequenced from the library</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 20 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_15">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_lib_layout/">MIMARKS specimen➞lib_layout</a>  <sub>0..1</sub><ul>
<li>Description: Specify whether to expect single, paired, or other configuration of reads</li>
<li>Range: <a href="../lib_layout_enum/">lib_layout_enum</a></li>
<li>Example: paired None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_16">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_lib_vector/">MIMARKS specimen➞lib_vector</a>  <sub>0..1</sub><ul>
<li>Description: Cloning vector type(s) used in construction of libraries</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Bacteriophage P1 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_17">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_lib_screen/">MIMARKS specimen➞lib_screen</a>  <sub>0..1</sub><ul>
<li>Description: Specific enrichment or screening methods applied before and/or after creating libraries</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: enriched, screened, normalized None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_18">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_target_gene/">MIMARKS specimen➞target_gene</a>  <sub>1..1</sub><ul>
<li>Description: Targeted gene or locus name for marker gene studies</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 16S rRNA, 18S rRNA, nif, amoA, rpo None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_19">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_target_subfragment/">MIMARKS specimen➞target_subfragment</a>  <sub>0..1</sub><ul>
<li>Description: Name of subfragment of a gene or locus. Important to e.g. identify special regions on marker genes like V6 on 16S rRNA</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: V6, V9, ITS None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_20">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_pcr_primers/">MIMARKS specimen➞pcr_primers</a>  <sub>0..1</sub><ul>
<li>Description: PCR primers that were used to amplify the sequence of the targeted gene, locus or subfragment. This field should contain all the primers used for a single PCR reaction if multiple forward or reverse primers are present in a single PCR reaction. The primer sequence should be reported in uppercase letters</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: FWD:GTGCCAGCMGCCGCGGTAA;REV:GGACTACHVGGGTWTCTAAT None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_21">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_mid/">MIMARKS specimen➞mid</a>  <sub>0..1</sub><ul>
<li>Description: Molecular barcodes, called Multiplex Identifiers (MIDs), that are used to specifically tag unique samples in a sequencing run. Sequence should be reported in uppercase letters</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: GTGAATAT None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_22">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_adapters/">MIMARKS specimen➞adapters</a>  <sub>0..1</sub><ul>
<li>Description: Adapters provide priming sequences for both amplification and sequencing of the sample-library fragments. Both adapters should be reported; in uppercase letters</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: AATGATACGGCGACCACCGAGATCTACACGCT;CAAGCAGAAGACGGCATACGAGAT None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_23">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_pcr_cond/">MIMARKS specimen➞pcr_cond</a>  <sub>0..1</sub><ul>
<li>Description: Description of reaction conditions and components of PCR in the form of 'initial denaturation:94degC_1.5min; annealing=...'</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: initial denaturation:94_3;annealing:50_1;elongation:72_1.5;final elongation:72_10;35 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_24">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_seq_meth/">MIMARKS specimen➞seq_meth</a>  <sub>1..1</sub><ul>
<li>Description: Sequencing machine used. Where possible the term should be taken from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 454 Genome Sequencer FLX [OBI:0000702] None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_25">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_seq_quality_check/">MIMARKS specimen➞seq_quality_check</a>  <sub>0..1</sub><ul>
<li>Description: Indicate if the sequence has been called by automatic systems (none) or undergone a manual editing procedure (e.g. by inspecting the raw data or chromatograms). Applied only for sequences that are not submitted to SRA,ENA or DRA</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: none None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_26">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_chimera_check/">MIMARKS specimen➞chimera_check</a>  <sub>0..1</sub><ul>
<li>Description: Tool(s) used for chimera checking, including version number and parameters, to discover and remove chimeric sequences. A chimeric sequence is comprised of two or more phylogenetically distinct parent sequences.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: uchime;v4.1;default parameters None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_27">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_assembly_software/">MIMARKS specimen➞assembly_software</a>  <sub>0..1</sub><ul>
<li>Description: Tool(s) used for assembly, including version number and parameters</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: metaSPAdes;3.11.0;kmer set 21,33,55,77,99,121, default parameters otherwise None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_28">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_associated_resource/">MIMARKS specimen➞associated resource</a>  <sub>0..1</sub><ul>
<li>Description: A related resource that is referenced, cited, or otherwise associated to the sequence.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: http://www.earthmicrobiome.org/ None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimarks-specimen_29">Mixed in from MIMARKS specimen:</h3>
<ul>
<li><a href="../MIMARKS_specimen_sop/">MIMARKS specimen➞sop</a>  <sub>0..1</sub><ul>
<li>Description: Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: http://press.igsb.anl.gov/earthmicrobiome/protocols-and-standards/its/ None</li>
</ul>
</li>
</ul></div>
            </div>
        </div>

        <footer class="col-md-12">
            <hr>
            <p>Documentation built with <a href="https://www.mkdocs.org/">MkDocs</a>.</p>
        </footer>
        <script>
            var base_url = "..",
                shortcuts = {"help": 191, "next": 78, "previous": 80, "search": 83};
        </script>
        <script src="../js/base.js" defer></script>
        <script src="../search/main.js" defer></script>

        <div class="modal" id="mkdocs_search_modal" tabindex="-1" role="dialog" aria-labelledby="searchModalLabel" aria-hidden="true">
    <div class="modal-dialog modal-lg">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="searchModalLabel">Search</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
                <p>From here you can search these documents. Enter your search terms below.</p>
                <form>
                    <div class="form-group">
                        <input type="search" class="form-control" placeholder="Search..." id="mkdocs-search-query" title="Type search term here">
                    </div>
                </form>
                <div id="mkdocs-search-results" data-no-results-text="No results found"></div>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div><div class="modal" id="mkdocs_keyboard_modal" tabindex="-1" role="dialog" aria-labelledby="keyboardModalLabel" aria-hidden="true">
    <div class="modal-dialog">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="keyboardModalLabel">Keyboard Shortcuts</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
              <table class="table">
                <thead>
                  <tr>
                    <th style="width: 20%;">Keys</th>
                    <th>Action</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td class="help shortcut"><kbd>?</kbd></td>
                    <td>Open this help</td>
                  </tr>
                  <tr>
                    <td class="next shortcut"><kbd>n</kbd></td>
                    <td>Next page</td>
                  </tr>
                  <tr>
                    <td class="prev shortcut"><kbd>p</kbd></td>
                    <td>Previous page</td>
                  </tr>
                  <tr>
                    <td class="search shortcut"><kbd>s</kbd></td>
                    <td>Search</td>
                  </tr>
                </tbody>
              </table>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div>

    </body>
</html>
