<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        
        
        <link rel="canonical" href="https://genomicsstandardsconsortium.github.io/mixs/Host-associated/">
        <link rel="shortcut icon" href="../img/favicon.ico">
        <title>Class: host-associated - MIxS</title>
        <link href="../css/bootstrap.min.css" rel="stylesheet">
        <link href="../css/font-awesome.min.css" rel="stylesheet">
        <link href="../css/base.css" rel="stylesheet">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/styles/github.min.css">

        <script src="../js/jquery-1.10.2.min.js" defer></script>
        <script src="../js/bootstrap.min.js" defer></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/highlight.min.js"></script>
        <script>hljs.initHighlightingOnLoad();</script> 
    </head>

    <body>
        <div class="navbar fixed-top navbar-expand-lg navbar-dark bg-dark">
            <div class="container">
                <a class="navbar-brand" href="..">MIxS</a>
                <!-- Expander button -->
                <button type="button" class="navbar-toggler" data-toggle="collapse" data-target="#navbar-collapse">
                    <span class="navbar-toggler-icon"></span>
                </button>

                <!-- Expanded navigation -->
                <div id="navbar-collapse" class="navbar-collapse collapse">
                        <!-- Main navigation -->
                        <ul class="nav navbar-nav">
                            <li class="navitem">
                                <a href=".." class="nav-link">About</a>
                            </li>
                            <li class="dropdown">
                                <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown">Introduction <b class="caret"></b></a>
                                <ul class="dropdown-menu">
                                    
<li>
    <a href="../introduction/background/" class="dropdown-item">Background</a>
</li>
                                    
<li>
    <a href="../introduction/citation/" class="dropdown-item">Cite</a>
</li>
                                    
<li>
    <a href="../introduction/contents/" class="dropdown-item">Contents</a>
</li>
                                </ul>
                            </li>
                        </ul>

                    <ul class="nav navbar-nav ml-auto">
                        <li class="nav-item">
                            <a href="#" class="nav-link" data-toggle="modal" data-target="#mkdocs_search_modal">
                                <i class="fa fa-search"></i> Search
                            </a>
                        </li>
                            <li class="nav-item">
                                <a href="https://github.com/GenomicsStandardsConsortium/mixs/edit/master/docs/Host-associated.md" class="nav-link"><i class="fa fa-github"></i> Edit on GitHub</a>
                            </li>
                    </ul>
                </div>
            </div>
        </div>

        <div class="container">
            <div class="row">
                    <div class="col-md-3"><div class="navbar-light navbar-expand-md bs-sidebar hidden-print affix" role="complementary">
    <div class="navbar-header">
        <button type="button" class="navbar-toggler collapsed" data-toggle="collapse" data-target="#toc-collapse" title="Table of Contents">
            <span class="fa fa-angle-down"></span>
        </button>
    </div>

    
    <div id="toc-collapse" class="navbar-collapse collapse card bg-secondary">
        <ul class="nav flex-column">
            
            <li class="nav-item" data-level="1"><a href="#class-host-associated" class="nav-link">Class: host-associated</a>
              <ul class="nav flex-column">
            <li class="nav-item" data-level="2"><a href="#children" class="nav-link">Children</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#referenced-by-class" class="nav-link">Referenced by Class</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#attributes" class="nav-link">Attributes</a>
              <ul class="nav flex-column">
              </ul>
            </li>
              </ul>
            </li>
        </ul>
    </div>
</div></div>
                    <div class="col-md-9" role="main">

<h1 id="class-host-associated">Class: host-associated</h1>
<p>host-associated</p>
<p>URI: <a href="https://w3id.org/mixs/vocab/Host-associated">mixs.vocab:Host-associated</a></p>
<p><a href="https://yuml.me/diagram/nofunky;dir:TB/class/[QuantityValue],[Host-associatedMIUVIG],[Host-associatedMISAG],[Host-associatedMIMS],[Host-associatedMIMARKSSurvey],[Host-associatedMIMARKSSpecimen],[Host-associatedMIMAG],[Host-associatedMIGSVirus],[Host-associatedMIGSPlant],[Host-associatedMIGSOrg],[Host-associatedMIGSEukaryote],[Host-associatedMIGSBacteria],[QuantityValue]&lt;samp_vol_we_dna_ext%200..1-++[Host-associated&#124;lat_lon:string%20%3F;geo_loc_name:string%20%3F;collection_date:date%20%3F;env_broad_scale:string%20%3F;env_local_scale:string%20%3F;env_medium:string%20%3F;ances_data:string%20%3F;biol_stat:biol_stat_enum%20%3F;genetic_mod:string%20%3F;host_common_name:string%20%3F;samp_capt_status:samp_capt_status_enum%20%3F;samp_dis_stage:samp_dis_stage_enum%20%3F;host_taxid:string%20%3F;host_subject_id:string%20%3F;host_life_stage:string%20%3F;host_sex:host_sex_enum%20%3F;chem_administration:string%20*;host_body_habitat:string%20%3F;host_body_site:string%20%3F;host_body_product:string%20%3F;host_diet:string%20*;host_last_meal:string%20*;host_growth_cond:string%20%3F;host_substrate:string%20%3F;host_family_relation:string%20*;host_subspecf_genlin:string%20*;host_genotype:string%20%3F;host_phenotype:string%20%3F;host_color:string%20%3F;host_shape:string%20%3F;gravidity:string%20%3F;perturbation:string%20*;oxy_stat_samp:oxy_stat_samp_enum%20%3F;organism_count:organism_count_enum%20*;samp_store_dur:string%20%3F;samp_store_loc:string%20%3F;host_symbiont:string%20*;misc_param:string%20*;samp_name:string;project_name:string;host_disease_stat:string%20%3F],[QuantityValue]&lt;samp_store_temp%200..1-++[Host-associated],[QuantityValue]&lt;salinity%200..1-++[Host-associated],[QuantityValue]&lt;blood_press_syst%200..1-++[Host-associated],[QuantityValue]&lt;blood_press_diast%200..1-++[Host-associated],[QuantityValue]&lt;host_dry_mass%200..1-++[Host-associated],[QuantityValue]&lt;host_body_temp%200..1-++[Host-associated],[QuantityValue]&lt;host_length%200..1-++[Host-associated],[QuantityValue]&lt;host_height%200..1-++[Host-associated],[QuantityValue]&lt;host_tot_mass%200..1-++[Host-associated],[QuantityValue]&lt;host_age%200..1-++[Host-associated],[QuantityValue]&lt;temp%200..1-++[Host-associated],[QuantityValue]&lt;elev%200..1-++[Host-associated],[QuantityValue]&lt;alt%200..1-++[Host-associated],[QuantityValue]&lt;depth%200..1-++[Host-associated],[Host-associated]^-[Host-associatedMIUVIG],[Host-associated]^-[Host-associatedMISAG],[Host-associated]^-[Host-associatedMIMS],[Host-associated]^-[Host-associatedMIMARKSSurvey],[Host-associated]^-[Host-associatedMIMARKSSpecimen],[Host-associated]^-[Host-associatedMIMAG],[Host-associated]^-[Host-associatedMIGSVirus],[Host-associated]^-[Host-associatedMIGSPlant],[Host-associated]^-[Host-associatedMIGSOrg],[Host-associated]^-[Host-associatedMIGSEukaryote],[Host-associated]^-[Host-associatedMIGSBacteria]"><img alt="img" src="https://yuml.me/diagram/nofunky;dir:TB/class/[QuantityValue],[Host-associatedMIUVIG],[Host-associatedMISAG],[Host-associatedMIMS],[Host-associatedMIMARKSSurvey],[Host-associatedMIMARKSSpecimen],[Host-associatedMIMAG],[Host-associatedMIGSVirus],[Host-associatedMIGSPlant],[Host-associatedMIGSOrg],[Host-associatedMIGSEukaryote],[Host-associatedMIGSBacteria],[QuantityValue]&lt;samp_vol_we_dna_ext%200..1-++[Host-associated&#124;lat_lon:string%20%3F;geo_loc_name:string%20%3F;collection_date:date%20%3F;env_broad_scale:string%20%3F;env_local_scale:string%20%3F;env_medium:string%20%3F;ances_data:string%20%3F;biol_stat:biol_stat_enum%20%3F;genetic_mod:string%20%3F;host_common_name:string%20%3F;samp_capt_status:samp_capt_status_enum%20%3F;samp_dis_stage:samp_dis_stage_enum%20%3F;host_taxid:string%20%3F;host_subject_id:string%20%3F;host_life_stage:string%20%3F;host_sex:host_sex_enum%20%3F;chem_administration:string%20*;host_body_habitat:string%20%3F;host_body_site:string%20%3F;host_body_product:string%20%3F;host_diet:string%20*;host_last_meal:string%20*;host_growth_cond:string%20%3F;host_substrate:string%20%3F;host_family_relation:string%20*;host_subspecf_genlin:string%20*;host_genotype:string%20%3F;host_phenotype:string%20%3F;host_color:string%20%3F;host_shape:string%20%3F;gravidity:string%20%3F;perturbation:string%20*;oxy_stat_samp:oxy_stat_samp_enum%20%3F;organism_count:organism_count_enum%20*;samp_store_dur:string%20%3F;samp_store_loc:string%20%3F;host_symbiont:string%20*;misc_param:string%20*;samp_name:string;project_name:string;host_disease_stat:string%20%3F],[QuantityValue]&lt;samp_store_temp%200..1-++[Host-associated],[QuantityValue]&lt;salinity%200..1-++[Host-associated],[QuantityValue]&lt;blood_press_syst%200..1-++[Host-associated],[QuantityValue]&lt;blood_press_diast%200..1-++[Host-associated],[QuantityValue]&lt;host_dry_mass%200..1-++[Host-associated],[QuantityValue]&lt;host_body_temp%200..1-++[Host-associated],[QuantityValue]&lt;host_length%200..1-++[Host-associated],[QuantityValue]&lt;host_height%200..1-++[Host-associated],[QuantityValue]&lt;host_tot_mass%200..1-++[Host-associated],[QuantityValue]&lt;host_age%200..1-++[Host-associated],[QuantityValue]&lt;temp%200..1-++[Host-associated],[QuantityValue]&lt;elev%200..1-++[Host-associated],[QuantityValue]&lt;alt%200..1-++[Host-associated],[QuantityValue]&lt;depth%200..1-++[Host-associated],[Host-associated]^-[Host-associatedMIUVIG],[Host-associated]^-[Host-associatedMISAG],[Host-associated]^-[Host-associatedMIMS],[Host-associated]^-[Host-associatedMIMARKSSurvey],[Host-associated]^-[Host-associatedMIMARKSSpecimen],[Host-associated]^-[Host-associatedMIMAG],[Host-associated]^-[Host-associatedMIGSVirus],[Host-associated]^-[Host-associatedMIGSPlant],[Host-associated]^-[Host-associatedMIGSOrg],[Host-associated]^-[Host-associatedMIGSEukaryote],[Host-associated]^-[Host-associatedMIGSBacteria]" /></a></p>
<h2 id="children">Children</h2>
<ul>
<li><a href="../Host-associatedMIGSBacteria/">Host-associatedMIGSBacteria</a> - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package host-associated</li>
<li><a href="../Host-associatedMIGSEukaryote/">Host-associatedMIGSEukaryote</a> - Combinatorial checklist Minimal Information about a Genome Sequence: eukaryote with environmental package host-associated</li>
<li><a href="../Host-associatedMIGSOrg/">Host-associatedMIGSOrg</a> - Combinatorial checklist Minimal Information about a Genome Sequence: org with environmental package host-associated</li>
<li><a href="../Host-associatedMIGSPlant/">Host-associatedMIGSPlant</a> - Combinatorial checklist Minimal Information about a Genome Sequence: plant with environmental package host-associated</li>
<li><a href="../Host-associatedMIGSVirus/">Host-associatedMIGSVirus</a> - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package host-associated</li>
<li><a href="../Host-associatedMIMAG/">Host-associatedMIMAG</a> - Combinatorial checklist Minimum Information About a Metagenome-Assembled Genome with environmental package host-associated</li>
<li><a href="../Host-associatedMIMARKSSpecimen/">Host-associatedMIMARKSSpecimen</a> - Combinatorial checklist Minimal Information about a Marker Specimen: specimen with environmental package host-associated</li>
<li><a href="../Host-associatedMIMARKSSurvey/">Host-associatedMIMARKSSurvey</a> - Combinatorial checklist Minimal Information about a Marker Specimen: survey with environmental package host-associated</li>
<li><a href="../Host-associatedMIMS/">Host-associatedMIMS</a> - Combinatorial checklist Metagenome or Environmental with environmental package host-associated</li>
<li><a href="../Host-associatedMISAG/">Host-associatedMISAG</a> - Combinatorial checklist Minimum Information About a Single Amplified Genome with environmental package host-associated</li>
<li><a href="../Host-associatedMIUVIG/">Host-associatedMIUVIG</a> - Combinatorial checklist Minimum Information About an Uncultivated Virus Genome with environmental package host-associated</li>
</ul>
<h2 id="referenced-by-class">Referenced by Class</h2>
<h2 id="attributes">Attributes</h2>
<h3 id="own">Own</h3>
<ul>
<li><a href="../lat_lon/">lat_lon</a>  <sub>0..1</sub><ul>
<li>Description: The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 50.586825 6.408977 None</li>
</ul>
</li>
<li><a href="../host_associated_depth/">host-associated➞depth</a>  <sub>0..1</sub><ul>
<li>Description: The vertical distance below local surface, e.g. for sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 10 meter None</li>
</ul>
</li>
<li><a href="../host_associated_alt/">host-associated➞alt</a>  <sub>0..1</sub><ul>
<li>Description: Altitude is a term used to identify heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth's surface. In this context, the altitude measurement is the vertical distance between the earth's surface above sea level and the sampled position in the air</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 100 meter None</li>
</ul>
</li>
<li><a href="../host_associated_elev/">host-associated➞elev</a>  <sub>0..1</sub><ul>
<li>Description: Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth's surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 100 meter None</li>
</ul>
</li>
<li><a href="../host_associated_temp/">host-associated➞temp</a>  <sub>0..1</sub><ul>
<li>Description: Temperature of the sample at the time of sampling.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 25 degree Celsius None</li>
</ul>
</li>
<li><a href="../geo_loc_name/">geo_loc_name</a>  <sub>0..1</sub><ul>
<li>Description: The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: USA: Maryland, Bethesda None</li>
</ul>
</li>
<li><a href="../collection_date/">collection_date</a>  <sub>0..1</sub><ul>
<li>Description: The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant</li>
<li>Range: <a href="../types/Date/">Date</a></li>
<li>Example: 2018-05-11T10:00:00+01:00; 2018-05-11 None</li>
</ul>
</li>
<li><a href="../env_broad_scale/">env_broad_scale</a>  <sub>0..1</sub><ul>
<li>Description: Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO’s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: oceanic epipelagic zone biome [ENVO:01000033] for annotating a water sample from the photic zone in middle of the Atlantic Ocean None</li>
</ul>
</li>
<li><a href="../env_local_scale/">env_local_scale</a>  <sub>0..1</sub><ul>
<li>Description: Report the entity or entities which are in the sample or specimen’s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: litter layer [ENVO:01000338]; Annotating a pooled sample taken from various vegetation layers in a forest consider: canopy [ENVO:00000047]|herb and fern layer [ENVO:01000337]|litter layer [ENVO:01000338]|understory [01000335]|shrub layer [ENVO:01000336]. None</li>
</ul>
</li>
<li><a href="../env_medium/">env_medium</a>  <sub>0..1</sub><ul>
<li>Description: Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of 'environmental material' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: soil [ENVO:00001998]; Annotating a fish swimming in the upper 100 m of the Atlantic Ocean, consider: ocean water [ENVO:00002151]. Example: Annotating a duck on a pond consider: pond water [ENVO:00002228]|air [ENVO_00002005] None</li>
</ul>
</li>
<li><a href="../host_associated_ances_data/">host-associated➞ances_data</a>  <sub>0..1</sub><ul>
<li>Description: Information about either pedigree or other ancestral information description (e.g. parental variety in case of mutant or selection), e.g. A/3*B (meaning [(A x B) x B] x B)</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: A/3*B None</li>
</ul>
</li>
<li><a href="../host_associated_biol_stat/">host-associated➞biol_stat</a>  <sub>0..1</sub><ul>
<li>Description: The level of genome modification.</li>
<li>Range: <a href="../biol_stat_enum/">biol_stat_enum</a></li>
<li>Example: natural None</li>
</ul>
</li>
<li><a href="../host_associated_genetic_mod/">host-associated➞genetic_mod</a>  <sub>0..1</sub><ul>
<li>Description: Genetic modifications of the genome of an organism, which may occur naturally by spontaneous mutation, or be introduced by some experimental means, e.g. specification of a transgene or the gene knocked-out or details of transient transfection</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: aox1A transgenic None</li>
</ul>
</li>
<li><a href="../host_associated_host_common_name/">host-associated➞host_common_name</a>  <sub>0..1</sub><ul>
<li>Description: Common name of the host.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: human None</li>
</ul>
</li>
<li><a href="../host_associated_samp_capt_status/">host-associated➞samp_capt_status</a>  <sub>0..1</sub><ul>
<li>Description: Reason for the sample</li>
<li>Range: <a href="../samp_capt_status_enum/">samp_capt_status_enum</a></li>
<li>Example: farm sample None</li>
</ul>
</li>
<li><a href="../host_associated_samp_dis_stage/">host-associated➞samp_dis_stage</a>  <sub>0..1</sub><ul>
<li>Description: Stage of the disease at the time of sample collection, e.g. inoculation, penetration, infection, growth and reproduction, dissemination of pathogen.</li>
<li>Range: <a href="../samp_dis_stage_enum/">samp_dis_stage_enum</a></li>
<li>Example: infection None</li>
</ul>
</li>
<li><a href="../host_associated_host_taxid/">host-associated➞host_taxid</a>  <sub>0..1</sub><ul>
<li>Description: NCBI taxon id of the host, e.g. 9606</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 7955 None</li>
</ul>
</li>
<li><a href="../host_associated_host_subject_id/">host-associated➞host_subject_id</a>  <sub>0..1</sub><ul>
<li>Description: A unique identifier by which each subject can be referred to, de-identified.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: MPI123 None</li>
</ul>
</li>
<li><a href="../host_associated_host_age/">host-associated➞host_age</a>  <sub>0..1</sub><ul>
<li>Description: Age of host at the time of sampling; relevant scale depends on species and study, e.g. Could be seconds for amoebae or centuries for trees</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 10 days None</li>
</ul>
</li>
<li><a href="../host_associated_host_life_stage/">host-associated➞host_life_stage</a>  <sub>0..1</sub><ul>
<li>Description: Description of life stage of host</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: adult None</li>
</ul>
</li>
<li><a href="../host_associated_host_sex/">host-associated➞host_sex</a>  <sub>0..1</sub><ul>
<li>Description: Gender or physical sex of the host.</li>
<li>Range: <a href="../host_sex_enum/">host_sex_enum</a></li>
<li>Example: non-binary None</li>
</ul>
</li>
<li><a href="../host_associated_chem_administration/">host-associated➞chem_administration</a>  <sub>0..*</sub><ul>
<li>Description: List of chemical compounds administered to the host or site where sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can include multiple compounds. For chemical entities of biological interest ontology (chebi) (v 163), http://purl.bioontology.org/ontology/chebi</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: agar [CHEBI:2509];2018-05-11T20:00Z None</li>
</ul>
</li>
<li><a href="../host_associated_host_body_habitat/">host-associated➞host_body_habitat</a>  <sub>0..1</sub><ul>
<li>Description: Original body habitat where the sample was obtained from</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: nasopharynx None</li>
</ul>
</li>
<li><a href="../host_associated_host_body_site/">host-associated➞host_body_site</a>  <sub>0..1</sub><ul>
<li>Description: Name of body site where the sample was obtained from, such as a specific organ or tissue (tongue, lung etc...). For foundational model of anatomy ontology (fma) (v 4.11.0) or Uber-anatomy ontology (UBERON) (v releases/2014-06-15) terms, please see http://purl.bioontology.org/ontology/FMA or http://purl.bioontology.org/ontology/UBERON</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: gill [UBERON:0002535] None</li>
</ul>
</li>
<li><a href="../host_associated_host_body_product/">host-associated➞host_body_product</a>  <sub>0..1</sub><ul>
<li>Description: Substance produced by the body, e.g. Stool, mucus, where the sample was obtained from. For foundational model of anatomy ontology (fma) or Uber-anatomy ontology (UBERON) terms, please see https://www.ebi.ac.uk/ols/ontologies/fma or https://www.ebi.ac.uk/ols/ontologies/uberon</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Portion of mucus [fma66938] None</li>
</ul>
</li>
<li><a href="../host_associated_host_tot_mass/">host-associated➞host_tot_mass</a>  <sub>0..1</sub><ul>
<li>Description: Total mass of the host at collection, the unit depends on host</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 2500 gram None</li>
</ul>
</li>
<li><a href="../host_associated_host_height/">host-associated➞host_height</a>  <sub>0..1</sub><ul>
<li>Description: The height of subject</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 0.1 meter None</li>
</ul>
</li>
<li><a href="../host_associated_host_length/">host-associated➞host_length</a>  <sub>0..1</sub><ul>
<li>Description: The length of subject</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 1 meter None</li>
</ul>
</li>
<li><a href="../host_associated_host_diet/">host-associated➞host_diet</a>  <sub>0..*</sub><ul>
<li>Description: Type of diet depending on the host, for animals omnivore, herbivore etc., for humans high-fat, meditteranean etc.; can include multiple diet types</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: herbivore None</li>
</ul>
</li>
<li><a href="../host_associated_host_last_meal/">host-associated➞host_last_meal</a>  <sub>0..*</sub><ul>
<li>Description: Content of last meal and time since feeding; can include multiple values</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: corn feed;P2H None</li>
</ul>
</li>
<li><a href="../host_associated_host_growth_cond/">host-associated➞host_growth_cond</a>  <sub>0..1</sub><ul>
<li>Description: Literature reference giving growth conditions of the host</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: https://academic.oup.com/icesjms/article/68/2/349/617247 None</li>
</ul>
</li>
<li><a href="../host_associated_host_substrate/">host-associated➞host_substrate</a>  <sub>0..1</sub><ul>
<li>Description: The growth substrate of the host.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: rock None</li>
</ul>
</li>
<li><a href="../host_associated_host_family_relation/">host-associated➞host_family_relation</a>  <sub>0..*</sub><ul>
<li>Description: Familial relationships to other hosts in the same study; can include multiple relationships</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: offspring;Mussel25 None</li>
</ul>
</li>
<li><a href="../host_associated_host_subspecf_genlin/">host-associated➞host_subspecf_genlin</a>  <sub>0..*</sub><ul>
<li>Description: Information about the genetic distinctness of the host organism below the subspecies level e.g., serovar, serotype, biotype, ecotype, variety, cultivar, or any relevant genetic typing schemes like Group I plasmid. Subspecies should not be recorded in this term, but in the NCBI taxonomy. Supply both the lineage name and the lineage rank separated by a colon, e.g., biovar:abc123.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: serovar:Newport, variety:glabrum, cultivar: Red Delicious None</li>
</ul>
</li>
<li><a href="../host_associated_host_genotype/">host-associated➞host_genotype</a>  <sub>0..1</sub><ul>
<li>Description: Observed genotype</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: C57BL/6 None</li>
</ul>
</li>
<li><a href="../host_associated_host_phenotype/">host-associated➞host_phenotype</a>  <sub>0..1</sub><ul>
<li>Description: Phenotype of human or other host. For phenotypic quality ontology (pato) (v 2018-03-27) terms, please see http://purl.bioontology.org/ontology/pato. For Human Phenotype Ontology (HP) (v 2018-06-13) please see http://purl.bioontology.org/ontology/HP</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: elongated [PATO:0001154] None</li>
</ul>
</li>
<li><a href="../host_associated_host_body_temp/">host-associated➞host_body_temp</a>  <sub>0..1</sub><ul>
<li>Description: Core body temperature of the host when sample was collected</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 15 degree Celsius None</li>
</ul>
</li>
<li><a href="../host_associated_host_dry_mass/">host-associated➞host_dry_mass</a>  <sub>0..1</sub><ul>
<li>Description: Measurement of dry mass</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 500 gram None</li>
</ul>
</li>
<li><a href="../host_associated_blood_press_diast/">host-associated➞blood_press_diast</a>  <sub>0..1</sub><ul>
<li>Description: Resting diastolic blood pressure, measured as mm mercury</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../host_associated_blood_press_syst/">host-associated➞blood_press_syst</a>  <sub>0..1</sub><ul>
<li>Description: Resting systolic blood pressure, measured as mm mercury</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../host_associated_host_color/">host-associated➞host_color</a>  <sub>0..1</sub><ul>
<li>Description: The color of host</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../host_associated_host_shape/">host-associated➞host_shape</a>  <sub>0..1</sub><ul>
<li>Description: Morphological shape of host</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: round None</li>
</ul>
</li>
<li><a href="../host_associated_gravidity/">host-associated➞gravidity</a>  <sub>0..1</sub><ul>
<li>Description: Whether or not subject is gravid, and if yes date due or date post-conception, specifying which is used</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: yes;due date:2018-05-11 None</li>
</ul>
</li>
<li><a href="../host_associated_perturbation/">host-associated➞perturbation</a>  <sub>0..*</sub><ul>
<li>Description: Type of perturbation, e.g. chemical administration, physical disturbance, etc., coupled with perturbation regimen including how many times the perturbation was repeated, how long each perturbation lasted, and the start and end time of the entire perturbation period; can include multiple perturbation types</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: antibiotic addition;R2/2018-05-11T14:30Z/2018-05-11T19:30Z/P1H30M None</li>
</ul>
</li>
<li><a href="../host_associated_salinity/">host-associated➞salinity</a>  <sub>0..1</sub><ul>
<li>Description: The total concentration of all dissolved salts in a liquid or solid sample. While salinity can be measured by a complete chemical analysis, this method is difficult and time consuming. More often, it is instead derived from the conductivity measurement. This is known as practical salinity. These derivations compare the specific conductance of the sample to a salinity standard such as seawater.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 25 practical salinity unit None</li>
</ul>
</li>
<li><a href="../host_associated_oxy_stat_samp/">host-associated➞oxy_stat_samp</a>  <sub>0..1</sub><ul>
<li>Description: Oxygenation status of sample</li>
<li>Range: <a href="../oxy_stat_samp_enum/">oxy_stat_samp_enum</a></li>
<li>Example: aerobic None</li>
</ul>
</li>
<li><a href="../host_associated_organism_count/">host-associated➞organism_count</a>  <sub>0..*</sub><ul>
<li>Description: Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)</li>
<li>Range: <a href="../organism_count_enum/">organism_count_enum</a></li>
<li>Example: total prokaryotes;3.5e7 cells per milliliter;qPCR None</li>
</ul>
</li>
<li><a href="../host_associated_samp_store_temp/">host-associated➞samp_store_temp</a>  <sub>0..1</sub><ul>
<li>Description: Temperature at which sample was stored, e.g. -80 degree Celsius</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: -80 degree Celsius None</li>
</ul>
</li>
<li><a href="../host_associated_samp_store_dur/">host-associated➞samp_store_dur</a>  <sub>0..1</sub><ul>
<li>Description: Duration for which the sample was stored</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: P1Y6M None</li>
</ul>
</li>
<li><a href="../host_associated_samp_store_loc/">host-associated➞samp_store_loc</a>  <sub>0..1</sub><ul>
<li>Description: Location at which sample was stored, usually name of a specific freezer/room</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Freezer no:5 None</li>
</ul>
</li>
<li><a href="../host_associated_host_symbiont/">host-associated➞host_symbiont</a>  <sub>0..*</sub><ul>
<li>Description: The taxonomic name of the organism(s) found living in mutualistic, commensalistic, or parasitic symbiosis with the specific host.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: flukeworms None</li>
</ul>
</li>
<li><a href="../host_associated_misc_param/">host-associated➞misc_param</a>  <sub>0..*</sub><ul>
<li>Description: Any other measurement performed or parameter collected, that is not listed here</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Bicarbonate ion concentration;2075 micromole per kilogram None</li>
</ul>
</li>
<li><a href="../host_associated_samp_name/">host-associated➞samp_name</a>  <sub>1..1</sub><ul>
<li>Description: A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: ISDsoil1 None</li>
</ul>
</li>
<li><a href="../host_associated_project_name/">host-associated➞project_name</a>  <sub>1..1</sub><ul>
<li>Description: Name of the project within which the sequencing was organized</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Forest soil metagenome None</li>
</ul>
</li>
<li><a href="../host_associated_host_disease_stat/">host-associated➞host_disease_stat</a>  <sub>0..1</sub><ul>
<li>Description: List of diseases with which the host has been diagnosed; can include multiple diagnoses. The value of the field depends on host; for humans the terms should be chosen from the DO (Human Disease Ontology) at https://www.disease-ontology.org, non-human host diseases are free text</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: rabies [DOID:11260] None</li>
</ul>
</li>
<li><a href="../host_associated_samp_vol_we_dna_ext/">host-associated➞samp_vol_we_dna_ext</a>  <sub>0..1</sub><ul>
<li>Description: Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001).</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 1500 milliliter None</li>
</ul>
</li>
</ul></div>
            </div>
        </div>

        <footer class="col-md-12">
            <hr>
            <p>Documentation built with <a href="https://www.mkdocs.org/">MkDocs</a>.</p>
        </footer>
        <script>
            var base_url = "..",
                shortcuts = {"help": 191, "next": 78, "previous": 80, "search": 83};
        </script>
        <script src="../js/base.js" defer></script>
        <script src="../search/main.js" defer></script>

        <div class="modal" id="mkdocs_search_modal" tabindex="-1" role="dialog" aria-labelledby="searchModalLabel" aria-hidden="true">
    <div class="modal-dialog modal-lg">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="searchModalLabel">Search</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
                <p>From here you can search these documents. Enter your search terms below.</p>
                <form>
                    <div class="form-group">
                        <input type="search" class="form-control" placeholder="Search..." id="mkdocs-search-query" title="Type search term here">
                    </div>
                </form>
                <div id="mkdocs-search-results" data-no-results-text="No results found"></div>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div><div class="modal" id="mkdocs_keyboard_modal" tabindex="-1" role="dialog" aria-labelledby="keyboardModalLabel" aria-hidden="true">
    <div class="modal-dialog">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="keyboardModalLabel">Keyboard Shortcuts</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
              <table class="table">
                <thead>
                  <tr>
                    <th style="width: 20%;">Keys</th>
                    <th>Action</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td class="help shortcut"><kbd>?</kbd></td>
                    <td>Open this help</td>
                  </tr>
                  <tr>
                    <td class="next shortcut"><kbd>n</kbd></td>
                    <td>Next page</td>
                  </tr>
                  <tr>
                    <td class="prev shortcut"><kbd>p</kbd></td>
                    <td>Previous page</td>
                  </tr>
                  <tr>
                    <td class="search shortcut"><kbd>s</kbd></td>
                    <td>Search</td>
                  </tr>
                </tbody>
              </table>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div>

    </body>
</html>
