<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  
  
  <link rel="shortcut icon" href="../img/favicon.ico">
  <title>Class: MIGS virus - MIxS</title>
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:400,700|Roboto+Slab:400,700|Inconsolata:400,700" />

  <link rel="stylesheet" href="../css/theme.css" />
  <link rel="stylesheet" href="../css/theme_extra.css" />
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/styles/github.min.css" />
  
  <script>
    // Current page data
    var mkdocs_page_name = "Class: MIGS virus";
    var mkdocs_page_input_path = "MIGSVirus.md";
    var mkdocs_page_url = null;
  </script>
  
  <script src="../js/jquery-2.1.1.min.js" defer></script>
  <script src="../js/modernizr-2.8.3.min.js" defer></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/highlight.min.js"></script>
  <script>hljs.initHighlightingOnLoad();</script> 
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
    <div class="wy-side-scroll">
      <div class="wy-side-nav-search">
        <a href=".." class="icon icon-home"> MIxS</a>
        <div role="search">
  <form id ="rtd-search-form" class="wy-form" action="../search.html" method="get">
      <input type="text" name="q" placeholder="Search docs" title="Type search term here" />
  </form>
</div>
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
                <ul>
                    <li class="toctree-l1"><a class="" href="../index.md">Index</a>
                    </li>
                </ul>
      </div>
    </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" role="navigation" aria-label="top navigation">
        <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
        <a href="..">MIxS</a>
      </nav>

      
      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href="..">Docs</a> &raquo;</li>
    
      
    
    <li>Class: MIGS virus</li>
    <li class="wy-breadcrumbs-aside">
      
    </li>
  </ul>
  
  <hr/>
</div>

          <div role="main">
            <div class="section">
              
                <h1 id="class-migs-virus">Class: MIGS virus</h1>
<p>Minimal Information about a Genome Sequence: cultured bacteria/archaea</p>
<p>URI: <a href="https://w3id.org/mixs/vocab/MIGSVirus">mixs.vocab:MIGSVirus</a></p>
<p><a href="https://yuml.me/diagram/nofunky;dir:TB/class/[QuantityValue],[QuantityValue]&lt;samp_vol_we_dna_ext%200..1-++[MIGSVirus&#124;samp_name:string;samp_taxon_id:string;project_name:string;experimental_factor:string%20%3F;lat_lon:string;geo_loc_name:string;collection_date:date;neg_cont_type:neg_cont_type_enum%20%3F;pos_cont_type:string%20%3F;env_broad_scale:string;env_local_scale:string;env_medium:string;subspecf_gen_lin:string%20%3F;num_replicons:integer%20%3F;estimated_size:string%20%3F;ref_biomaterial:string%20%3F;source_mat_id:string%20%3F;pathogenicity:string%20%3F;biotic_relationship:biotic_relationship_enum%20%3F;specific_host:string%20%3F;host_spec_range:integer%20%3F;host_disease_stat:string%20%3F;propagation:string;encoded_traits:string%20%3F;isol_growth_condt:string;samp_collec_device:string%20%3F;samp_collec_method:string%20%3F;samp_mat_process:string%20%3F;virus_enrich_appr:virus_enrich_appr_enum%20%3F;nucl_acid_ext:string%20%3F;nucl_acid_amp:string%20%3F;lib_size:integer%20%3F;lib_reads_seqd:integer%20%3F;lib_layout:lib_layout_enum%20%3F;lib_vector:string%20%3F;lib_screen:string%20%3F;adapters:string%20%3F;seq_meth:string;tax_ident:tax_ident_enum%20%3F;assembly_qual:assembly_qual_enum%20%3F;assembly_name:string%20%3F;assembly_software:string;annot:string%20%3F;number_contig:integer%20%3F;feat_pred:string%20%3F;ref_db:string%20%3F;sim_search_meth:string%20%3F;tax_class:string%20%3F;compl_score:compl_score_enum%20%3F;compl_software:string%20%3F;associated_resource:string%20%3F;sop:string%20%3F],[QuantityValue]&lt;samp_size%200..1-++[MIGSVirus],[WaterMIGSVirus]uses%20-.-&gt;[MIGSVirus],[WastewaterSludgeMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Symbiont-associatedMIGSVirus]uses%20-.-&gt;[MIGSVirus],[SoilMIGSVirus]uses%20-.-&gt;[MIGSVirus],[SedimentMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Plant-associatedMIGSVirus]uses%20-.-&gt;[MIGSVirus],[MiscellaneousNaturalOrArtificialEnvironmentMIGSVirus]uses%20-.-&gt;[MIGSVirus],[MicrobialMatBiofilmMIGSVirus]uses%20-.-&gt;[MIGSVirus],[HydrocarbonResources-fluidsSwabsMIGSVirus]uses%20-.-&gt;[MIGSVirus],[HydrocarbonResources-coresMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Human-vaginalMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Human-skinMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Human-oralMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Human-gutMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Human-associatedMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Host-associatedMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Food-humanFoodsMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Food-foodProductionFacilityMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Food-farmEnvironmentMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Food-animalAndAnimalFeedMIGSVirus]uses%20-.-&gt;[MIGSVirus],[BuiltEnvironmentMIGSVirus]uses%20-.-&gt;[MIGSVirus],[AirMIGSVirus]uses%20-.-&gt;[MIGSVirus],[AgricultureMIGSVirus]uses%20-.-&gt;[MIGSVirus],[WaterMIGSVirus],[WastewaterSludgeMIGSVirus],[Symbiont-associatedMIGSVirus],[SoilMIGSVirus],[SedimentMIGSVirus],[Plant-associatedMIGSVirus],[MiscellaneousNaturalOrArtificialEnvironmentMIGSVirus],[MicrobialMatBiofilmMIGSVirus],[HydrocarbonResources-fluidsSwabsMIGSVirus],[HydrocarbonResources-coresMIGSVirus],[Human-vaginalMIGSVirus],[Human-skinMIGSVirus],[Human-oralMIGSVirus],[Human-gutMIGSVirus],[Human-associatedMIGSVirus],[Host-associatedMIGSVirus],[Food-humanFoodsMIGSVirus],[Food-foodProductionFacilityMIGSVirus],[Food-farmEnvironmentMIGSVirus],[Food-animalAndAnimalFeedMIGSVirus],[BuiltEnvironmentMIGSVirus],[AirMIGSVirus],[AgricultureMIGSVirus]"><img alt="img" src="https://yuml.me/diagram/nofunky;dir:TB/class/[QuantityValue],[QuantityValue]&lt;samp_vol_we_dna_ext%200..1-++[MIGSVirus&#124;samp_name:string;samp_taxon_id:string;project_name:string;experimental_factor:string%20%3F;lat_lon:string;geo_loc_name:string;collection_date:date;neg_cont_type:neg_cont_type_enum%20%3F;pos_cont_type:string%20%3F;env_broad_scale:string;env_local_scale:string;env_medium:string;subspecf_gen_lin:string%20%3F;num_replicons:integer%20%3F;estimated_size:string%20%3F;ref_biomaterial:string%20%3F;source_mat_id:string%20%3F;pathogenicity:string%20%3F;biotic_relationship:biotic_relationship_enum%20%3F;specific_host:string%20%3F;host_spec_range:integer%20%3F;host_disease_stat:string%20%3F;propagation:string;encoded_traits:string%20%3F;isol_growth_condt:string;samp_collec_device:string%20%3F;samp_collec_method:string%20%3F;samp_mat_process:string%20%3F;virus_enrich_appr:virus_enrich_appr_enum%20%3F;nucl_acid_ext:string%20%3F;nucl_acid_amp:string%20%3F;lib_size:integer%20%3F;lib_reads_seqd:integer%20%3F;lib_layout:lib_layout_enum%20%3F;lib_vector:string%20%3F;lib_screen:string%20%3F;adapters:string%20%3F;seq_meth:string;tax_ident:tax_ident_enum%20%3F;assembly_qual:assembly_qual_enum%20%3F;assembly_name:string%20%3F;assembly_software:string;annot:string%20%3F;number_contig:integer%20%3F;feat_pred:string%20%3F;ref_db:string%20%3F;sim_search_meth:string%20%3F;tax_class:string%20%3F;compl_score:compl_score_enum%20%3F;compl_software:string%20%3F;associated_resource:string%20%3F;sop:string%20%3F],[QuantityValue]&lt;samp_size%200..1-++[MIGSVirus],[WaterMIGSVirus]uses%20-.-&gt;[MIGSVirus],[WastewaterSludgeMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Symbiont-associatedMIGSVirus]uses%20-.-&gt;[MIGSVirus],[SoilMIGSVirus]uses%20-.-&gt;[MIGSVirus],[SedimentMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Plant-associatedMIGSVirus]uses%20-.-&gt;[MIGSVirus],[MiscellaneousNaturalOrArtificialEnvironmentMIGSVirus]uses%20-.-&gt;[MIGSVirus],[MicrobialMatBiofilmMIGSVirus]uses%20-.-&gt;[MIGSVirus],[HydrocarbonResources-fluidsSwabsMIGSVirus]uses%20-.-&gt;[MIGSVirus],[HydrocarbonResources-coresMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Human-vaginalMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Human-skinMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Human-oralMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Human-gutMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Human-associatedMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Host-associatedMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Food-humanFoodsMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Food-foodProductionFacilityMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Food-farmEnvironmentMIGSVirus]uses%20-.-&gt;[MIGSVirus],[Food-animalAndAnimalFeedMIGSVirus]uses%20-.-&gt;[MIGSVirus],[BuiltEnvironmentMIGSVirus]uses%20-.-&gt;[MIGSVirus],[AirMIGSVirus]uses%20-.-&gt;[MIGSVirus],[AgricultureMIGSVirus]uses%20-.-&gt;[MIGSVirus],[WaterMIGSVirus],[WastewaterSludgeMIGSVirus],[Symbiont-associatedMIGSVirus],[SoilMIGSVirus],[SedimentMIGSVirus],[Plant-associatedMIGSVirus],[MiscellaneousNaturalOrArtificialEnvironmentMIGSVirus],[MicrobialMatBiofilmMIGSVirus],[HydrocarbonResources-fluidsSwabsMIGSVirus],[HydrocarbonResources-coresMIGSVirus],[Human-vaginalMIGSVirus],[Human-skinMIGSVirus],[Human-oralMIGSVirus],[Human-gutMIGSVirus],[Human-associatedMIGSVirus],[Host-associatedMIGSVirus],[Food-humanFoodsMIGSVirus],[Food-foodProductionFacilityMIGSVirus],[Food-farmEnvironmentMIGSVirus],[Food-animalAndAnimalFeedMIGSVirus],[BuiltEnvironmentMIGSVirus],[AirMIGSVirus],[AgricultureMIGSVirus]" /></a></p>
<h2 id="mixin-for">Mixin for</h2>
<ul>
<li><a href="../AgricultureMIGSVirus/">AgricultureMIGSVirus</a> (mixin)  - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package agriculture</li>
<li><a href="../AirMIGSVirus/">AirMIGSVirus</a> (mixin)  - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package air</li>
<li><a href="../BuiltEnvironmentMIGSVirus/">BuiltEnvironmentMIGSVirus</a> (mixin)  - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package built environment</li>
<li><a href="../Food-animalAndAnimalFeedMIGSVirus/">Food-animalAndAnimalFeedMIGSVirus</a> (mixin)  - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package food-animal and animal feed</li>
<li><a href="../Food-farmEnvironmentMIGSVirus/">Food-farmEnvironmentMIGSVirus</a> (mixin)  - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package food-farm environment</li>
<li><a href="../Food-foodProductionFacilityMIGSVirus/">Food-foodProductionFacilityMIGSVirus</a> (mixin)  - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package food-food production facility</li>
<li><a href="../Food-humanFoodsMIGSVirus/">Food-humanFoodsMIGSVirus</a> (mixin)  - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package food-human foods</li>
<li><a href="../Host-associatedMIGSVirus/">Host-associatedMIGSVirus</a> (mixin)  - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package host-associated</li>
<li><a href="../Human-associatedMIGSVirus/">Human-associatedMIGSVirus</a> (mixin)  - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package human-associated</li>
<li><a href="../Human-gutMIGSVirus/">Human-gutMIGSVirus</a> (mixin)  - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package human-gut</li>
<li><a href="../Human-oralMIGSVirus/">Human-oralMIGSVirus</a> (mixin)  - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package human-oral</li>
<li><a href="../Human-skinMIGSVirus/">Human-skinMIGSVirus</a> (mixin)  - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package human-skin</li>
<li><a href="../Human-vaginalMIGSVirus/">Human-vaginalMIGSVirus</a> (mixin)  - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package human-vaginal</li>
<li><a href="../HydrocarbonResources-coresMIGSVirus/">HydrocarbonResources-coresMIGSVirus</a> (mixin)  - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package hydrocarbon resources-cores</li>
<li><a href="../HydrocarbonResources-fluidsSwabsMIGSVirus/">HydrocarbonResources-fluidsSwabsMIGSVirus</a> (mixin)  - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package hydrocarbon resources-fluids_swabs</li>
<li><a href="../MicrobialMatBiofilmMIGSVirus/">MicrobialMatBiofilmMIGSVirus</a> (mixin)  - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package microbial mat_biofilm</li>
<li><a href="../MiscellaneousNaturalOrArtificialEnvironmentMIGSVirus/">MiscellaneousNaturalOrArtificialEnvironmentMIGSVirus</a> (mixin)  - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package miscellaneous natural or artificial environment</li>
<li><a href="../Plant-associatedMIGSVirus/">Plant-associatedMIGSVirus</a> (mixin)  - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package plant-associated</li>
<li><a href="../SedimentMIGSVirus/">SedimentMIGSVirus</a> (mixin)  - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package sediment</li>
<li><a href="../SoilMIGSVirus/">SoilMIGSVirus</a> (mixin)  - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package soil</li>
<li><a href="../Symbiont-associatedMIGSVirus/">Symbiont-associatedMIGSVirus</a> (mixin)  - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package symbiont-associated</li>
<li><a href="../WastewaterSludgeMIGSVirus/">WastewaterSludgeMIGSVirus</a> (mixin)  - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package wastewater_sludge</li>
<li><a href="../WaterMIGSVirus/">WaterMIGSVirus</a> (mixin)  - Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package water</li>
</ul>
<h2 id="referenced-by-class">Referenced by Class</h2>
<h2 id="attributes">Attributes</h2>
<h3 id="own">Own</h3>
<ul>
<li><a href="../MIGS_virus_samp_name/">MIGS virus➞samp_name</a>  <sub>1..1</sub><ul>
<li>Description: A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: ISDsoil1 None</li>
</ul>
</li>
<li><a href="../MIGS_virus_samp_taxon_id/">MIGS virus➞samp_taxon_id</a>  <sub>1..1</sub><ul>
<li>Description: NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa sample. Use 'synthetic metagenome’ for mock community/positive controls, or 'blank sample' for negative controls.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Gut Metagenome [NCBI:txid749906] None</li>
</ul>
</li>
<li><a href="../MIGS_virus_project_name/">MIGS virus➞project_name</a>  <sub>1..1</sub><ul>
<li>Description: Name of the project within which the sequencing was organized</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Forest soil metagenome None</li>
</ul>
</li>
<li><a href="../MIGS_virus_experimental_factor/">MIGS virus➞experimental_factor</a>  <sub>0..1</sub><ul>
<li>Description: Experimental factors are essentially the variable aspects of an experiment design which can be used to describe an experiment, or set of experiments, in an increasingly detailed manner. This field accepts ontology terms from Experimental Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI). For a browser of EFO (v 2.95) terms, please see http://purl.bioontology.org/ontology/EFO; for a browser of OBI (v 2018-02-12) terms please see http://purl.bioontology.org/ontology/OBI</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: time series design [EFO:EFO_0001779] None</li>
</ul>
</li>
<li><a href="../MIGS_virus_lat_lon/">MIGS virus➞lat_lon</a>  <sub>1..1</sub><ul>
<li>Description: The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 50.586825 6.408977 None</li>
</ul>
</li>
<li><a href="../MIGS_virus_geo_loc_name/">MIGS virus➞geo_loc_name</a>  <sub>1..1</sub><ul>
<li>Description: The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: USA: Maryland, Bethesda None</li>
</ul>
</li>
<li><a href="../MIGS_virus_collection_date/">MIGS virus➞collection_date</a>  <sub>1..1</sub><ul>
<li>Description: The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant</li>
<li>Range: <a href="../types/Date/">Date</a></li>
<li>Example: 2018-05-11T10:00:00+01:00; 2018-05-11 None</li>
</ul>
</li>
<li><a href="../MIGS_virus_neg_cont_type/">MIGS virus➞neg_cont_type</a>  <sub>0..1</sub><ul>
<li>Description: The substance or equipment used as a negative control in an investigation</li>
<li>Range: <a href="../neg_cont_type_enum/">neg_cont_type_enum</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../MIGS_virus_pos_cont_type/">MIGS virus➞pos_cont_type</a>  <sub>0..1</sub><ul>
<li>Description: The substance, mixture, product, or apparatus used to verify that a process which is part of an investigation delivers a true positive.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../MIGS_virus_env_broad_scale/">MIGS virus➞env_broad_scale</a>  <sub>1..1</sub><ul>
<li>Description: Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO’s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: oceanic epipelagic zone biome [ENVO:01000033] for annotating a water sample from the photic zone in middle of the Atlantic Ocean None</li>
</ul>
</li>
<li><a href="../MIGS_virus_env_local_scale/">MIGS virus➞env_local_scale</a>  <sub>1..1</sub><ul>
<li>Description: Report the entity or entities which are in the sample or specimen’s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: litter layer [ENVO:01000338]; Annotating a pooled sample taken from various vegetation layers in a forest consider: canopy [ENVO:00000047]|herb and fern layer [ENVO:01000337]|litter layer [ENVO:01000338]|understory [01000335]|shrub layer [ENVO:01000336]. None</li>
</ul>
</li>
<li><a href="../MIGS_virus_env_medium/">MIGS virus➞env_medium</a>  <sub>1..1</sub><ul>
<li>Description: Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of 'environmental material' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: soil [ENVO:00001998]; Annotating a fish swimming in the upper 100 m of the Atlantic Ocean, consider: ocean water [ENVO:00002151]. Example: Annotating a duck on a pond consider: pond water [ENVO:00002228]|air [ENVO_00002005] None</li>
</ul>
</li>
<li><a href="../MIGS_virus_subspecf_gen_lin/">MIGS virus➞subspecf_gen_lin</a>  <sub>0..1</sub><ul>
<li>Description: Information about the genetic distinctness of the sequenced organism below the subspecies level, e.g., serovar, serotype, biotype, ecotype, or any relevant genetic typing schemes like Group I plasmid. Subspecies should not be recorded in this term, but in the NCBI taxonomy. Supply both the lineage name and the lineage rank separated by a colon, e.g., biovar:abc123.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: serovar:Newport None</li>
</ul>
</li>
<li><a href="../MIGS_virus_num_replicons/">MIGS virus➞num_replicons</a>  <sub>0..1</sub><ul>
<li>Description: Reports the number of replicons in a nuclear genome of eukaryotes, in the genome of a bacterium or archaea or the number of segments in a segmented virus. Always applied to the haploid chromosome count of a eukaryote</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 2 None</li>
</ul>
</li>
<li><a href="../MIGS_virus_estimated_size/">MIGS virus➞estimated_size</a>  <sub>0..1</sub><ul>
<li>Description: The estimated size of the genome prior to sequencing. Of particular importance in the sequencing of (eukaryotic) genome which could remain in draft form for a long or unspecified period.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 300000 bp None</li>
</ul>
</li>
<li><a href="../MIGS_virus_ref_biomaterial/">MIGS virus➞ref_biomaterial</a>  <sub>0..1</sub><ul>
<li>Description: Primary publication if isolated before genome publication; otherwise, primary genome report.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: doi:10.1016/j.syapm.2018.01.009 None</li>
</ul>
</li>
<li><a href="../MIGS_virus_source_mat_id/">MIGS virus➞source_mat_id</a>  <sub>0..1</sub><ul>
<li>Description: A unique identifier assigned to a material sample (as defined by http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular digital record of a material sample) used for extracting nucleic acids, and subsequent sequencing. The identifier can refer either to the original material collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher, /bio_material, or /culture_collection may or may not share the same value as the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id may both contain 'UAM:Herps:14' , referring to both the specimen voucher and sampled tissue with the same identifier. However, the /culture_collection qualifier may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id would refer to an identifier from some derived culture from which the nucleic acids were extracted (e.g. xatc123 or ark:/2154/R2).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: MPI012345 None</li>
</ul>
</li>
<li><a href="../MIGS_virus_pathogenicity/">MIGS virus➞pathogenicity</a>  <sub>0..1</sub><ul>
<li>Description: To what is the entity pathogenic</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: human, animal, plant, fungi, bacteria None</li>
</ul>
</li>
<li><a href="../MIGS_virus_biotic_relationship/">MIGS virus➞biotic_relationship</a>  <sub>0..1</sub><ul>
<li>Description: Description of relationship(s) between the subject organism and other organism(s) it is associated with. E.g., parasite on species X; mutualist with species Y. The target organism is the subject of the relationship, and the other organism(s) is the object</li>
<li>Range: <a href="../biotic_relationship_enum/">biotic_relationship_enum</a></li>
<li>Example: free living None</li>
</ul>
</li>
<li><a href="../MIGS_virus_specific_host/">MIGS virus➞specific_host</a>  <sub>0..1</sub><ul>
<li>Description: Report the host's taxonomic name and/or NCBI taxonomy ID.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Homo sapiens and/or 9606 None</li>
</ul>
</li>
<li><a href="../MIGS_virus_host_spec_range/">MIGS virus➞host_spec_range</a>  <sub>0..1</sub><ul>
<li>Description: The range and diversity of host species that an organism is capable of infecting, defined by NCBI taxonomy identifier.</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 9606 None</li>
</ul>
</li>
<li><a href="../MIGS_virus_host_disease_stat/">MIGS virus➞host_disease_stat</a>  <sub>0..1</sub><ul>
<li>Description: List of diseases with which the host has been diagnosed; can include multiple diagnoses. The value of the field depends on host; for humans the terms should be chosen from the DO (Human Disease Ontology) at https://www.disease-ontology.org, non-human host diseases are free text</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: rabies [DOID:11260] None</li>
</ul>
</li>
<li><a href="../MIGS_virus_propagation/">MIGS virus➞propagation</a>  <sub>1..1</sub><ul>
<li>Description: The type of reproduction from the parent stock. Values for this field is specific to different taxa. For phage or virus: lytic/lysogenic/temperate/obligately lytic. For plasmids: incompatibility group. For eukaryotes: sexual/asexual.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: lytic None</li>
</ul>
</li>
<li><a href="../MIGS_virus_encoded_traits/">MIGS virus➞encoded_traits</a>  <sub>0..1</sub><ul>
<li>Description: Should include key traits like antibiotic resistance or xenobiotic degradation phenotypes for plasmids, converting genes for phage</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: beta-lactamase class A None</li>
</ul>
</li>
<li><a href="../MIGS_virus_isol_growth_condt/">MIGS virus➞isol_growth_condt</a>  <sub>1..1</sub><ul>
<li>Description: Publication reference in the form of pubmed ID (pmid), digital object identifier (doi) or url for isolation and growth condition specifications of the organism/material</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: doi: 10.1016/j.syapm.2018.01.009 None</li>
</ul>
</li>
<li><a href="../MIGS_virus_samp_collec_device/">MIGS virus➞samp_collec_device</a>  <sub>0..1</sub><ul>
<li>Description: The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: swab, biopsy, niskin bottle, push core, drag swab [GENEPIO:0002713] None</li>
</ul>
</li>
<li><a href="../MIGS_virus_samp_collec_method/">MIGS virus➞samp_collec_method</a>  <sub>0..1</sub><ul>
<li>Description: The method employed for collecting the sample.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: swabbing None</li>
</ul>
</li>
<li><a href="../MIGS_virus_samp_mat_process/">MIGS virus➞samp_mat_process</a>  <sub>0..1</sub><ul>
<li>Description: A brief description of any processing applied to the sample during or after retrieving the sample from environment, or a link to the relevant protocol(s) performed.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: filtering of seawater, storing samples in ethanol None</li>
</ul>
</li>
<li><a href="../MIGS_virus_samp_size/">MIGS virus➞samp_size</a>  <sub>0..1</sub><ul>
<li>Description: The total amount or size (volume (ml), mass (g) or area (m2) ) of sample collected.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 5 liter None</li>
</ul>
</li>
<li><a href="../MIGS_virus_samp_vol_we_dna_ext/">MIGS virus➞samp_vol_we_dna_ext</a>  <sub>0..1</sub><ul>
<li>Description: Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001).</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 1500 milliliter None</li>
</ul>
</li>
<li><a href="../MIGS_virus_virus_enrich_appr/">MIGS virus➞virus_enrich_appr</a>  <sub>0..1</sub><ul>
<li>Description: List of approaches used to enrich the sample for viruses, if any</li>
<li>Range: <a href="../virus_enrich_appr_enum/">virus_enrich_appr_enum</a></li>
<li>Example: filtration + FeCl Precipitation + ultracentrifugation + DNAse None</li>
</ul>
</li>
<li><a href="../MIGS_virus_nucl_acid_ext/">MIGS virus➞nucl_acid_ext</a>  <sub>0..1</sub><ul>
<li>Description: A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the material separation to recover the nucleic acid fraction from a sample</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: https://mobio.com/media/wysiwyg/pdfs/protocols/12888.pdf None</li>
</ul>
</li>
<li><a href="../MIGS_virus_nucl_acid_amp/">MIGS virus➞nucl_acid_amp</a>  <sub>0..1</sub><ul>
<li>Description: A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the enzymatic amplification (PCR, TMA, NASBA) of specific nucleic acids</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: https://phylogenomics.me/protocols/16s-pcr-protocol/ None</li>
</ul>
</li>
<li><a href="../MIGS_virus_lib_size/">MIGS virus➞lib_size</a>  <sub>0..1</sub><ul>
<li>Description: Total number of clones in the library prepared for the project</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 50 None</li>
</ul>
</li>
<li><a href="../MIGS_virus_lib_reads_seqd/">MIGS virus➞lib_reads_seqd</a>  <sub>0..1</sub><ul>
<li>Description: Total number of clones sequenced from the library</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 20 None</li>
</ul>
</li>
<li><a href="../MIGS_virus_lib_layout/">MIGS virus➞lib_layout</a>  <sub>0..1</sub><ul>
<li>Description: Specify whether to expect single, paired, or other configuration of reads</li>
<li>Range: <a href="../lib_layout_enum/">lib_layout_enum</a></li>
<li>Example: paired None</li>
</ul>
</li>
<li><a href="../MIGS_virus_lib_vector/">MIGS virus➞lib_vector</a>  <sub>0..1</sub><ul>
<li>Description: Cloning vector type(s) used in construction of libraries</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Bacteriophage P1 None</li>
</ul>
</li>
<li><a href="../MIGS_virus_lib_screen/">MIGS virus➞lib_screen</a>  <sub>0..1</sub><ul>
<li>Description: Specific enrichment or screening methods applied before and/or after creating libraries</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: enriched, screened, normalized None</li>
</ul>
</li>
<li><a href="../MIGS_virus_adapters/">MIGS virus➞adapters</a>  <sub>0..1</sub><ul>
<li>Description: Adapters provide priming sequences for both amplification and sequencing of the sample-library fragments. Both adapters should be reported; in uppercase letters</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: AATGATACGGCGACCACCGAGATCTACACGCT;CAAGCAGAAGACGGCATACGAGAT None</li>
</ul>
</li>
<li><a href="../MIGS_virus_seq_meth/">MIGS virus➞seq_meth</a>  <sub>1..1</sub><ul>
<li>Description: Sequencing machine used. Where possible the term should be taken from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 454 Genome Sequencer FLX [OBI:0000702] None</li>
</ul>
</li>
<li><a href="../MIGS_virus_tax_ident/">MIGS virus➞tax_ident</a>  <sub>0..1</sub><ul>
<li>Description: The phylogenetic marker(s) used to assign an organism name to the SAG or MAG</li>
<li>Range: <a href="../tax_ident_enum/">tax_ident_enum</a></li>
<li>Example: other: rpoB gene None</li>
</ul>
</li>
<li><a href="../MIGS_virus_assembly_qual/">MIGS virus➞assembly_qual</a>  <sub>0..1</sub><ul>
<li>Description: The assembly quality category is based on sets of criteria outlined for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities with a consensus error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments where gaps span repetitive regions. Presence of the 23S, 16S and 5S rRNA genes and at least 18 tRNAs. Medium Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Low Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Assembly statistics include, but are not limited to total assembly size, number of contigs, contig N50/L50, and maximum contig length. For MIUVIG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities, with extensive manual review and editing to annotate putative gene functions and transcriptional units. High-quality draft genome: One or multiple fragments, totaling ≥ 90% of the expected genome or replicon sequence or predicted complete. Genome fragment(s): One or multiple fragments, totalling &lt; 90% of the expected genome or replicon sequence, or for which no genome size could be estimated</li>
<li>Range: <a href="../assembly_qual_enum/">assembly_qual_enum</a></li>
<li>Example: High-quality draft genome None</li>
</ul>
</li>
<li><a href="../MIGS_virus_assembly_name/">MIGS virus➞assembly_name</a>  <sub>0..1</sub><ul>
<li>Description: Name/version of the assembly provided by the submitter that is used in the genome browsers and in the community</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: HuRef, JCVI_ISG_i3_1.0 None</li>
</ul>
</li>
<li><a href="../MIGS_virus_assembly_software/">MIGS virus➞assembly_software</a>  <sub>1..1</sub><ul>
<li>Description: Tool(s) used for assembly, including version number and parameters</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: metaSPAdes;3.11.0;kmer set 21,33,55,77,99,121, default parameters otherwise None</li>
</ul>
</li>
<li><a href="../MIGS_virus_annot/">MIGS virus➞annot</a>  <sub>0..1</sub><ul>
<li>Description: Tool used for annotation, or for cases where annotation was provided by a community jamboree or model organism database rather than by a specific submitter</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: prokka None</li>
</ul>
</li>
<li><a href="../MIGS_virus_number_contig/">MIGS virus➞number_contig</a>  <sub>0..1</sub><ul>
<li>Description: Total number of contigs in the cleaned/submitted assembly that makes up a given genome, SAG, MAG, or UViG</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 40 None</li>
</ul>
</li>
<li><a href="../MIGS_virus_feat_pred/">MIGS virus➞feat_pred</a>  <sub>0..1</sub><ul>
<li>Description: Method used to predict UViGs features such as ORFs, integration site, etc.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Prodigal;2.6.3;default parameters None</li>
</ul>
</li>
<li><a href="../MIGS_virus_ref_db/">MIGS virus➞ref_db</a>  <sub>0..1</sub><ul>
<li>Description: List of database(s) used for ORF annotation, along with version number and reference to website or publication</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: pVOGs;5;http://dmk-brain.ecn.uiowa.edu/pVOGs/ Grazziotin et al. 2017 doi:10.1093/nar/gkw975 None</li>
</ul>
</li>
<li><a href="../MIGS_virus_sim_search_meth/">MIGS virus➞sim_search_meth</a>  <sub>0..1</sub><ul>
<li>Description: Tool used to compare ORFs with database, along with version and cutoffs used</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: HMMER3;3.1b2;hmmsearch, cutoff of 50 on score None</li>
</ul>
</li>
<li><a href="../MIGS_virus_tax_class/">MIGS virus➞tax_class</a>  <sub>0..1</sub><ul>
<li>Description: Method used for taxonomic classification, along with reference database used, classification rank, and thresholds used to classify new genomes</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: vConTACT vContact2 (references from NCBI RefSeq v83, genus rank classification, default parameters) None</li>
</ul>
</li>
<li><a href="../MIGS_virus_compl_score/">MIGS virus➞compl_score</a>  <sub>0..1</sub><ul>
<li>Description: Completeness score is typically based on either the fraction of markers found as compared to a database or the percent of a genome found as compared to a closely related reference genome. High Quality Draft: &gt;90%, Medium Quality Draft: &gt;50%, and Low Quality Draft: &lt; 50% should have the indicated completeness scores</li>
<li>Range: <a href="../compl_score_enum/">compl_score_enum</a></li>
<li>Example: med;60% None</li>
</ul>
</li>
<li><a href="../MIGS_virus_compl_software/">MIGS virus➞compl_software</a>  <sub>0..1</sub><ul>
<li>Description: Tools used for completion estimate, i.e. checkm, anvi'o, busco</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: checkm None</li>
</ul>
</li>
<li><a href="../MIGS_virus_associated_resource/">MIGS virus➞associated resource</a>  <sub>0..1</sub><ul>
<li>Description: A related resource that is referenced, cited, or otherwise associated to the sequence.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: http://www.earthmicrobiome.org/ None</li>
</ul>
</li>
<li><a href="../MIGS_virus_sop/">MIGS virus➞sop</a>  <sub>0..1</sub><ul>
<li>Description: Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: http://press.igsb.anl.gov/earthmicrobiome/protocols-and-standards/its/ None</li>
</ul>
</li>
</ul>
<h2 id="other-properties">Other properties</h2>
<table>
<thead>
<tr>
<th></th>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr>
<td><strong>Aliases:</strong></td>
<td></td>
<td>MIGS.ba</td>
</tr>
</tbody>
</table>
              
            </div>
          </div>
          <footer>
  

  <hr/>

  <div role="contentinfo">
    <!-- Copyright etc -->
    
  </div>

  Built with <a href="https://www.mkdocs.org/">MkDocs</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
</footer>
      
        </div>
      </div>

    </section>

  </div>

  <div class="rst-versions" role="note" aria-label="versions">
  <span class="rst-current-version" data-toggle="rst-current-version">
    
    
    
  </span>
</div>
    <script>var base_url = '..';</script>
    <script src="../js/theme_extra.js" defer></script>
    <script src="../js/theme.js" defer></script>
      <script src="../search/main.js" defer></script>
    <script defer>
        window.onload = function () {
            SphinxRtdTheme.Navigation.enable(true);
        };
    </script>

</body>
</html>
