<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  
  
  <link rel="shortcut icon" href="../img/favicon.ico">
  <title>Class: human-oral MIMAG - MIxS</title>
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:400,700|Roboto+Slab:400,700|Inconsolata:400,700" />

  <link rel="stylesheet" href="../css/theme.css" />
  <link rel="stylesheet" href="../css/theme_extra.css" />
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/styles/github.min.css" />
  
  <script>
    // Current page data
    var mkdocs_page_name = "Class: human-oral MIMAG";
    var mkdocs_page_input_path = "Human-oralMIMAG.md";
    var mkdocs_page_url = null;
  </script>
  
  <script src="../js/jquery-2.1.1.min.js" defer></script>
  <script src="../js/modernizr-2.8.3.min.js" defer></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/highlight.min.js"></script>
  <script>hljs.initHighlightingOnLoad();</script> 
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
    <div class="wy-side-scroll">
      <div class="wy-side-nav-search">
        <a href=".." class="icon icon-home"> MIxS</a>
        <div role="search">
  <form id ="rtd-search-form" class="wy-form" action="../search.html" method="get">
      <input type="text" name="q" placeholder="Search docs" title="Type search term here" />
  </form>
</div>
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
                <ul>
                    <li class="toctree-l1"><a class="" href="../index.md">Index</a>
                    </li>
                </ul>
      </div>
    </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" role="navigation" aria-label="top navigation">
        <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
        <a href="..">MIxS</a>
      </nav>

      
      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href="..">Docs</a> &raquo;</li>
    
      
    
    <li>Class: human-oral MIMAG</li>
    <li class="wy-breadcrumbs-aside">
      
    </li>
  </ul>
  
  <hr/>
</div>

          <div role="main">
            <div class="section">
              
                <h1 id="class-human-oral-mimag">Class: human-oral MIMAG</h1>
<p>Combinatorial checklist Minimum Information About a Metagenome-Assembled Genome with environmental package human-oral</p>
<p>URI: <a href="https://w3id.org/mixs/vocab/Human-oralMIMAG">mixs.vocab:Human-oralMIMAG</a></p>
<p><a href="https://yuml.me/diagram/nofunky;dir:TB/class/[QuantityValue],[Human-oralMIMAG&#124;samp_taxon_id:string;experimental_factor:string%20%3F;neg_cont_type:neg_cont_type_enum%20%3F;pos_cont_type:string%20%3F;ref_biomaterial:string%20%3F;source_mat_id:string%20%3F;rel_to_oxygen:rel_to_oxygen_enum%20%3F;samp_collec_device:string%20%3F;samp_collec_method:string%20%3F;samp_mat_process:string%20%3F;size_frac:string%20%3F;nucl_acid_ext:string%20%3F;nucl_acid_amp:string%20%3F;lib_size:integer%20%3F;lib_reads_seqd:integer%20%3F;lib_layout:lib_layout_enum%20%3F;lib_vector:string%20%3F;lib_screen:string%20%3F;mid:string%20%3F;adapters:string%20%3F;seq_meth:string;tax_ident:tax_ident_enum;assembly_qual:assembly_qual_enum;assembly_name:string%20%3F;assembly_software:string;annot:string%20%3F;number_contig:integer%20%3F;feat_pred:string%20%3F;ref_db:string%20%3F;sim_search_meth:string%20%3F;tax_class:string%20%3F;x_16s_recover:string%20%3F;x_16s_recover_software:string%20%3F;trnas:integer%20%3F;trna_ext_software:string%20%3F;compl_score:compl_score_enum;compl_software:string;compl_appr:compl_appr_enum%20%3F;contam_score:string;contam_screen_input:string%20%3F;contam_screen_param:contam_screen_param_enum%20%3F;decontam_software:decontam_software_enum%20%3F;bin_param:bin_param_enum;bin_software:string;reassembly_bin:string%20%3F;mag_cov_software:mag_cov_software_enum%20%3F;associated_resource:string%20%3F;sop:string%20%3F;lat_lon(i):string%20%3F;geo_loc_name(i):string%20%3F;collection_date(i):date%20%3F;env_broad_scale(i):string%20%3F;env_local_scale(i):string%20%3F;env_medium(i):string%20%3F;nose_throat_disord(i):string%20*;time_last_toothbrush(i):string%20%3F;host_subject_id(i):string%20%3F;host_sex(i):host_sex_enum%20%3F;ihmc_medication_code(i):integer%20*;chem_administration(i):string%20*;host_body_site(i):string%20%3F;host_body_product(i):string%20%3F;host_diet(i):string%20*;host_last_meal(i):string%20*;host_family_relation(i):string%20*;host_genotype(i):string%20%3F;host_phenotype(i):string%20%3F;ethnicity(i):string%20*;host_occupation(i):integer%20%3F;medic_hist_perform(i):string%20%3F;perturbation(i):string%20*;oxy_stat_samp(i):oxy_stat_samp_enum%20%3F;organism_count(i):organism_count_enum%20*;samp_store_dur(i):string%20%3F;host_symbiont(i):string%20*;samp_store_loc(i):string%20%3F;misc_param(i):string%20*;samp_name(i):string;project_name(i):string;host_disease_stat(i):string%20%3F]uses%20-.-&gt;[MIMAG],[Human-oral]^-[Human-oralMIMAG],[Human-oral],[MIMAG]"><img alt="img" src="https://yuml.me/diagram/nofunky;dir:TB/class/[QuantityValue],[Human-oralMIMAG&#124;samp_taxon_id:string;experimental_factor:string%20%3F;neg_cont_type:neg_cont_type_enum%20%3F;pos_cont_type:string%20%3F;ref_biomaterial:string%20%3F;source_mat_id:string%20%3F;rel_to_oxygen:rel_to_oxygen_enum%20%3F;samp_collec_device:string%20%3F;samp_collec_method:string%20%3F;samp_mat_process:string%20%3F;size_frac:string%20%3F;nucl_acid_ext:string%20%3F;nucl_acid_amp:string%20%3F;lib_size:integer%20%3F;lib_reads_seqd:integer%20%3F;lib_layout:lib_layout_enum%20%3F;lib_vector:string%20%3F;lib_screen:string%20%3F;mid:string%20%3F;adapters:string%20%3F;seq_meth:string;tax_ident:tax_ident_enum;assembly_qual:assembly_qual_enum;assembly_name:string%20%3F;assembly_software:string;annot:string%20%3F;number_contig:integer%20%3F;feat_pred:string%20%3F;ref_db:string%20%3F;sim_search_meth:string%20%3F;tax_class:string%20%3F;x_16s_recover:string%20%3F;x_16s_recover_software:string%20%3F;trnas:integer%20%3F;trna_ext_software:string%20%3F;compl_score:compl_score_enum;compl_software:string;compl_appr:compl_appr_enum%20%3F;contam_score:string;contam_screen_input:string%20%3F;contam_screen_param:contam_screen_param_enum%20%3F;decontam_software:decontam_software_enum%20%3F;bin_param:bin_param_enum;bin_software:string;reassembly_bin:string%20%3F;mag_cov_software:mag_cov_software_enum%20%3F;associated_resource:string%20%3F;sop:string%20%3F;lat_lon(i):string%20%3F;geo_loc_name(i):string%20%3F;collection_date(i):date%20%3F;env_broad_scale(i):string%20%3F;env_local_scale(i):string%20%3F;env_medium(i):string%20%3F;nose_throat_disord(i):string%20*;time_last_toothbrush(i):string%20%3F;host_subject_id(i):string%20%3F;host_sex(i):host_sex_enum%20%3F;ihmc_medication_code(i):integer%20*;chem_administration(i):string%20*;host_body_site(i):string%20%3F;host_body_product(i):string%20%3F;host_diet(i):string%20*;host_last_meal(i):string%20*;host_family_relation(i):string%20*;host_genotype(i):string%20%3F;host_phenotype(i):string%20%3F;ethnicity(i):string%20*;host_occupation(i):integer%20%3F;medic_hist_perform(i):string%20%3F;perturbation(i):string%20*;oxy_stat_samp(i):oxy_stat_samp_enum%20%3F;organism_count(i):organism_count_enum%20*;samp_store_dur(i):string%20%3F;host_symbiont(i):string%20*;samp_store_loc(i):string%20%3F;misc_param(i):string%20*;samp_name(i):string;project_name(i):string;host_disease_stat(i):string%20%3F]uses%20-.-&gt;[MIMAG],[Human-oral]^-[Human-oralMIMAG],[Human-oral],[MIMAG]" /></a></p>
<h2 id="parents">Parents</h2>
<ul>
<li>is_a: <a href="../Human-oral/">Human-oral</a> - human-oral</li>
</ul>
<h2 id="uses-mixin">Uses Mixin</h2>
<ul>
<li>mixin: <a href="../MIMAG/">MIMAG</a> - Minimum Information About a Metagenome-Assembled Genome</li>
</ul>
<h2 id="attributes">Attributes</h2>
<h3 id="inherited-from-human-oral">Inherited from human-oral:</h3>
<ul>
<li><a href="../lat_lon/">lat_lon</a>  <sub>0..1</sub><ul>
<li>Description: The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 50.586825 6.408977 None</li>
</ul>
</li>
<li><a href="../depth/">depth</a>  <sub>0..1</sub><ul>
<li>Description: The vertical distance below local surface, e.g. for sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 10 meter None</li>
</ul>
</li>
<li><a href="../alt/">alt</a>  <sub>0..1</sub><ul>
<li>Description: Altitude is a term used to identify heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth's surface. In this context, the altitude measurement is the vertical distance between the earth's surface above sea level and the sampled position in the air</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 100 meter None</li>
</ul>
</li>
<li><a href="../elev/">elev</a>  <sub>0..1</sub><ul>
<li>Description: Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth's surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 100 meter None</li>
</ul>
</li>
<li><a href="../human_oral_temp/">human-oral➞temp</a>  <sub>0..1</sub><ul>
<li>Description: Temperature of the sample at the time of sampling.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 25 degree Celsius None</li>
</ul>
</li>
<li><a href="../geo_loc_name/">geo_loc_name</a>  <sub>0..1</sub><ul>
<li>Description: The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: USA: Maryland, Bethesda None</li>
</ul>
</li>
<li><a href="../collection_date/">collection_date</a>  <sub>0..1</sub><ul>
<li>Description: The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant</li>
<li>Range: <a href="../types/Date/">Date</a></li>
<li>Example: 2018-05-11T10:00:00+01:00; 2018-05-11 None</li>
</ul>
</li>
<li><a href="../env_broad_scale/">env_broad_scale</a>  <sub>0..1</sub><ul>
<li>Description: Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO’s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: oceanic epipelagic zone biome [ENVO:01000033] for annotating a water sample from the photic zone in middle of the Atlantic Ocean None</li>
</ul>
</li>
<li><a href="../env_local_scale/">env_local_scale</a>  <sub>0..1</sub><ul>
<li>Description: Report the entity or entities which are in the sample or specimen’s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: litter layer [ENVO:01000338]; Annotating a pooled sample taken from various vegetation layers in a forest consider: canopy [ENVO:00000047]|herb and fern layer [ENVO:01000337]|litter layer [ENVO:01000338]|understory [01000335]|shrub layer [ENVO:01000336]. None</li>
</ul>
</li>
<li><a href="../env_medium/">env_medium</a>  <sub>0..1</sub><ul>
<li>Description: Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of 'environmental material' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: soil [ENVO:00001998]; Annotating a fish swimming in the upper 100 m of the Atlantic Ocean, consider: ocean water [ENVO:00002151]. Example: Annotating a duck on a pond consider: pond water [ENVO:00002228]|air [ENVO_00002005] None</li>
</ul>
</li>
<li><a href="../human_oral_nose_throat_disord/">human-oral➞nose_throat_disord</a>  <sub>0..*</sub><ul>
<li>Description: History of nose-throat disorders; can include multiple disorders,  The terms should be chosen from the DO (Human Disease Ontology) at http://www.disease-ontology.org, lung disease (https://disease-ontology.org/?id=DOID:850), upper respiratory tract disease (https://disease-ontology.org/?id=DOID:974).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../human_oral_time_last_toothbrush/">human-oral➞time_last_toothbrush</a>  <sub>0..1</sub><ul>
<li>Description: Specification of the time since last toothbrushing</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: P2H45M None</li>
</ul>
</li>
<li><a href="../human_oral_host_subject_id/">human-oral➞host_subject_id</a>  <sub>0..1</sub><ul>
<li>Description: A unique identifier by which each subject can be referred to, de-identified.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: MPI123 None</li>
</ul>
</li>
<li><a href="../human_oral_host_age/">human-oral➞host_age</a>  <sub>0..1</sub><ul>
<li>Description: Age of host at the time of sampling; relevant scale depends on species and study, e.g. Could be seconds for amoebae or centuries for trees</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 10 days None</li>
</ul>
</li>
<li><a href="../human_oral_host_sex/">human-oral➞host_sex</a>  <sub>0..1</sub><ul>
<li>Description: Gender or physical sex of the host.</li>
<li>Range: <a href="../host_sex_enum/">host_sex_enum</a></li>
<li>Example: non-binary None</li>
</ul>
</li>
<li><a href="../human_oral_ihmc_medication_code/">human-oral➞ihmc_medication_code</a>  <sub>0..*</sub><ul>
<li>Description: Can include multiple medication codes</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 810 None</li>
</ul>
</li>
<li><a href="../human_oral_chem_administration/">human-oral➞chem_administration</a>  <sub>0..*</sub><ul>
<li>Description: List of chemical compounds administered to the host or site where sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can include multiple compounds. For chemical entities of biological interest ontology (chebi) (v 163), http://purl.bioontology.org/ontology/chebi</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: agar [CHEBI:2509];2018-05-11T20:00Z None</li>
</ul>
</li>
<li><a href="../human_oral_host_body_site/">human-oral➞host_body_site</a>  <sub>0..1</sub><ul>
<li>Description: Name of body site where the sample was obtained from, such as a specific organ or tissue (tongue, lung etc...). For foundational model of anatomy ontology (fma) (v 4.11.0) or Uber-anatomy ontology (UBERON) (v releases/2014-06-15) terms, please see http://purl.bioontology.org/ontology/FMA or http://purl.bioontology.org/ontology/UBERON</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: gill [UBERON:0002535] None</li>
</ul>
</li>
<li><a href="../human_oral_host_body_product/">human-oral➞host_body_product</a>  <sub>0..1</sub><ul>
<li>Description: Substance produced by the body, e.g. Stool, mucus, where the sample was obtained from. For foundational model of anatomy ontology (fma) or Uber-anatomy ontology (UBERON) terms, please see https://www.ebi.ac.uk/ols/ontologies/fma or https://www.ebi.ac.uk/ols/ontologies/uberon</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Portion of mucus [fma66938] None</li>
</ul>
</li>
<li><a href="../human_oral_host_tot_mass/">human-oral➞host_tot_mass</a>  <sub>0..1</sub><ul>
<li>Description: Total mass of the host at collection, the unit depends on host</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 2500 gram None</li>
</ul>
</li>
<li><a href="../human_oral_host_height/">human-oral➞host_height</a>  <sub>0..1</sub><ul>
<li>Description: The height of subject</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 0.1 meter None</li>
</ul>
</li>
<li><a href="../human_oral_host_diet/">human-oral➞host_diet</a>  <sub>0..*</sub><ul>
<li>Description: Type of diet depending on the host, for animals omnivore, herbivore etc., for humans high-fat, meditteranean etc.; can include multiple diet types</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: herbivore None</li>
</ul>
</li>
<li><a href="../human_oral_host_last_meal/">human-oral➞host_last_meal</a>  <sub>0..*</sub><ul>
<li>Description: Content of last meal and time since feeding; can include multiple values</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: corn feed;P2H None</li>
</ul>
</li>
<li><a href="../human_oral_host_family_relation/">human-oral➞host_family_relation</a>  <sub>0..*</sub><ul>
<li>Description: Familial relationships to other hosts in the same study; can include multiple relationships</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: offspring;Mussel25 None</li>
</ul>
</li>
<li><a href="../human_oral_host_genotype/">human-oral➞host_genotype</a>  <sub>0..1</sub><ul>
<li>Description: Observed genotype</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: C57BL/6 None</li>
</ul>
</li>
<li><a href="../human_oral_host_phenotype/">human-oral➞host_phenotype</a>  <sub>0..1</sub><ul>
<li>Description: Phenotype of human or other host. For phenotypic quality ontology (pato) (v 2018-03-27) terms, please see http://purl.bioontology.org/ontology/pato. For Human Phenotype Ontology (HP) (v 2018-06-13) please see http://purl.bioontology.org/ontology/HP</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: elongated [PATO:0001154] None</li>
</ul>
</li>
<li><a href="../human_oral_host_body_temp/">human-oral➞host_body_temp</a>  <sub>0..1</sub><ul>
<li>Description: Core body temperature of the host when sample was collected</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 15 degree Celsius None</li>
</ul>
</li>
<li><a href="../human_oral_host_body_mass_index/">human-oral➞host_body_mass_index</a>  <sub>0..1</sub><ul>
<li>Description: Body mass index, calculated as weight/(height)squared</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 22 kilogram per square meter None</li>
</ul>
</li>
<li><a href="../human_oral_ethnicity/">human-oral➞ethnicity</a>  <sub>0..*</sub><ul>
<li>Description: A category of people who identify with each other, usually on the basis of presumed similarities such as a common language, ancestry, history, society, culture, nation or social treatment within their residing area. https://en.wikipedia.org/wiki/List_of_contemporary_ethnic_groups</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: native american None</li>
</ul>
</li>
<li><a href="../human_oral_host_occupation/">human-oral➞host_occupation</a>  <sub>0..1</sub><ul>
<li>Description: Most frequent job performed by subject</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: veterinary None</li>
</ul>
</li>
<li><a href="../human_oral_medic_hist_perform/">human-oral➞medic_hist_perform</a>  <sub>0..1</sub><ul>
<li>Description: Whether full medical history was collected</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: TRUE None</li>
</ul>
</li>
<li><a href="../human_oral_host_pulse/">human-oral➞host_pulse</a>  <sub>0..1</sub><ul>
<li>Description: Resting pulse, measured as beats per minute</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 65 beats per minute None</li>
</ul>
</li>
<li><a href="../human_oral_perturbation/">human-oral➞perturbation</a>  <sub>0..*</sub><ul>
<li>Description: Type of perturbation, e.g. chemical administration, physical disturbance, etc., coupled with perturbation regimen including how many times the perturbation was repeated, how long each perturbation lasted, and the start and end time of the entire perturbation period; can include multiple perturbation types</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: antibiotic addition;R2/2018-05-11T14:30Z/2018-05-11T19:30Z/P1H30M None</li>
</ul>
</li>
<li><a href="../human_oral_salinity/">human-oral➞salinity</a>  <sub>0..1</sub><ul>
<li>Description: The total concentration of all dissolved salts in a liquid or solid sample. While salinity can be measured by a complete chemical analysis, this method is difficult and time consuming. More often, it is instead derived from the conductivity measurement. This is known as practical salinity. These derivations compare the specific conductance of the sample to a salinity standard such as seawater.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 25 practical salinity unit None</li>
</ul>
</li>
<li><a href="../human_oral_oxy_stat_samp/">human-oral➞oxy_stat_samp</a>  <sub>0..1</sub><ul>
<li>Description: Oxygenation status of sample</li>
<li>Range: <a href="../oxy_stat_samp_enum/">oxy_stat_samp_enum</a></li>
<li>Example: aerobic None</li>
</ul>
</li>
<li><a href="../human_oral_organism_count/">human-oral➞organism_count</a>  <sub>0..*</sub><ul>
<li>Description: Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)</li>
<li>Range: <a href="../organism_count_enum/">organism_count_enum</a></li>
<li>Example: total prokaryotes;3.5e7 cells per milliliter;qPCR None</li>
</ul>
</li>
<li><a href="../human_oral_samp_store_temp/">human-oral➞samp_store_temp</a>  <sub>0..1</sub><ul>
<li>Description: Temperature at which sample was stored, e.g. -80 degree Celsius</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: -80 degree Celsius None</li>
</ul>
</li>
<li><a href="../human_oral_samp_store_dur/">human-oral➞samp_store_dur</a>  <sub>0..1</sub><ul>
<li>Description: Duration for which the sample was stored</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: P1Y6M None</li>
</ul>
</li>
<li><a href="../human_oral_host_symbiont/">human-oral➞host_symbiont</a>  <sub>0..*</sub><ul>
<li>Description: The taxonomic name of the organism(s) found living in mutualistic, commensalistic, or parasitic symbiosis with the specific host.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: flukeworms None</li>
</ul>
</li>
<li><a href="../human_oral_samp_store_loc/">human-oral➞samp_store_loc</a>  <sub>0..1</sub><ul>
<li>Description: Location at which sample was stored, usually name of a specific freezer/room</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Freezer no:5 None</li>
</ul>
</li>
<li><a href="../human_oral_misc_param/">human-oral➞misc_param</a>  <sub>0..*</sub><ul>
<li>Description: Any other measurement performed or parameter collected, that is not listed here</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Bicarbonate ion concentration;2075 micromole per kilogram None</li>
</ul>
</li>
<li><a href="../human_oral_samp_name/">human-oral➞samp_name</a>  <sub>1..1</sub><ul>
<li>Description: A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: ISDsoil1 None</li>
</ul>
</li>
<li><a href="../human_oral_project_name/">human-oral➞project_name</a>  <sub>1..1</sub><ul>
<li>Description: Name of the project within which the sequencing was organized</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Forest soil metagenome None</li>
</ul>
</li>
<li><a href="../human_oral_host_disease_stat/">human-oral➞host_disease_stat</a>  <sub>0..1</sub><ul>
<li>Description: List of diseases with which the host has been diagnosed; can include multiple diagnoses. The value of the field depends on host; for humans the terms should be chosen from the DO (Human Disease Ontology) at https://www.disease-ontology.org, non-human host diseases are free text</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: rabies [DOID:11260] None</li>
</ul>
</li>
<li><a href="../human_oral_samp_vol_we_dna_ext/">human-oral➞samp_vol_we_dna_ext</a>  <sub>0..1</sub><ul>
<li>Description: Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001).</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 1500 milliliter None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_samp_taxon_id/">MIMAG➞samp_taxon_id</a>  <sub>1..1</sub><ul>
<li>Description: NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa sample. Use 'synthetic metagenome’ for mock community/positive controls, or 'blank sample' for negative controls.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Gut Metagenome [NCBI:txid749906] None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_1">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_experimental_factor/">MIMAG➞experimental_factor</a>  <sub>0..1</sub><ul>
<li>Description: Experimental factors are essentially the variable aspects of an experiment design which can be used to describe an experiment, or set of experiments, in an increasingly detailed manner. This field accepts ontology terms from Experimental Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI). For a browser of EFO (v 2.95) terms, please see http://purl.bioontology.org/ontology/EFO; for a browser of OBI (v 2018-02-12) terms please see http://purl.bioontology.org/ontology/OBI</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: time series design [EFO:EFO_0001779] None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_2">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_neg_cont_type/">MIMAG➞neg_cont_type</a>  <sub>0..1</sub><ul>
<li>Description: The substance or equipment used as a negative control in an investigation</li>
<li>Range: <a href="../neg_cont_type_enum/">neg_cont_type_enum</a></li>
<li>Example:  None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_3">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_pos_cont_type/">MIMAG➞pos_cont_type</a>  <sub>0..1</sub><ul>
<li>Description: The substance, mixture, product, or apparatus used to verify that a process which is part of an investigation delivers a true positive.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_4">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_ref_biomaterial/">MIMAG➞ref_biomaterial</a>  <sub>0..1</sub><ul>
<li>Description: Primary publication if isolated before genome publication; otherwise, primary genome report.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: doi:10.1016/j.syapm.2018.01.009 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_5">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_source_mat_id/">MIMAG➞source_mat_id</a>  <sub>0..1</sub><ul>
<li>Description: A unique identifier assigned to a material sample (as defined by http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular digital record of a material sample) used for extracting nucleic acids, and subsequent sequencing. The identifier can refer either to the original material collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher, /bio_material, or /culture_collection may or may not share the same value as the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id may both contain 'UAM:Herps:14' , referring to both the specimen voucher and sampled tissue with the same identifier. However, the /culture_collection qualifier may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id would refer to an identifier from some derived culture from which the nucleic acids were extracted (e.g. xatc123 or ark:/2154/R2).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: MPI012345 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_6">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_rel_to_oxygen/">MIMAG➞rel_to_oxygen</a>  <sub>0..1</sub><ul>
<li>Description: Is this organism an aerobe, anaerobe? Please note that aerobic and anaerobic are valid descriptors for microbial environments</li>
<li>Range: <a href="../rel_to_oxygen_enum/">rel_to_oxygen_enum</a></li>
<li>Example: aerobe None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_7">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_samp_collec_device/">MIMAG➞samp_collec_device</a>  <sub>0..1</sub><ul>
<li>Description: The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: swab, biopsy, niskin bottle, push core, drag swab [GENEPIO:0002713] None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_8">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_samp_collec_method/">MIMAG➞samp_collec_method</a>  <sub>0..1</sub><ul>
<li>Description: The method employed for collecting the sample.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: swabbing None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_9">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_samp_mat_process/">MIMAG➞samp_mat_process</a>  <sub>0..1</sub><ul>
<li>Description: A brief description of any processing applied to the sample during or after retrieving the sample from environment, or a link to the relevant protocol(s) performed.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: filtering of seawater, storing samples in ethanol None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_10">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_size_frac/">MIMAG➞size_frac</a>  <sub>0..1</sub><ul>
<li>Description: Filtering pore size used in sample preparation</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 0-0.22 micrometer None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_11">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_samp_size/">MIMAG➞samp_size</a>  <sub>0..1</sub><ul>
<li>Description: The total amount or size (volume (ml), mass (g) or area (m2) ) of sample collected.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 5 liter None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_12">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_nucl_acid_ext/">MIMAG➞nucl_acid_ext</a>  <sub>0..1</sub><ul>
<li>Description: A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the material separation to recover the nucleic acid fraction from a sample</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: https://mobio.com/media/wysiwyg/pdfs/protocols/12888.pdf None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_13">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_nucl_acid_amp/">MIMAG➞nucl_acid_amp</a>  <sub>0..1</sub><ul>
<li>Description: A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the enzymatic amplification (PCR, TMA, NASBA) of specific nucleic acids</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: https://phylogenomics.me/protocols/16s-pcr-protocol/ None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_14">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_lib_size/">MIMAG➞lib_size</a>  <sub>0..1</sub><ul>
<li>Description: Total number of clones in the library prepared for the project</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 50 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_15">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_lib_reads_seqd/">MIMAG➞lib_reads_seqd</a>  <sub>0..1</sub><ul>
<li>Description: Total number of clones sequenced from the library</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 20 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_16">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_lib_layout/">MIMAG➞lib_layout</a>  <sub>0..1</sub><ul>
<li>Description: Specify whether to expect single, paired, or other configuration of reads</li>
<li>Range: <a href="../lib_layout_enum/">lib_layout_enum</a></li>
<li>Example: paired None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_17">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_lib_vector/">MIMAG➞lib_vector</a>  <sub>0..1</sub><ul>
<li>Description: Cloning vector type(s) used in construction of libraries</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Bacteriophage P1 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_18">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_lib_screen/">MIMAG➞lib_screen</a>  <sub>0..1</sub><ul>
<li>Description: Specific enrichment or screening methods applied before and/or after creating libraries</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: enriched, screened, normalized None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_19">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_mid/">MIMAG➞mid</a>  <sub>0..1</sub><ul>
<li>Description: Molecular barcodes, called Multiplex Identifiers (MIDs), that are used to specifically tag unique samples in a sequencing run. Sequence should be reported in uppercase letters</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: GTGAATAT None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_20">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_adapters/">MIMAG➞adapters</a>  <sub>0..1</sub><ul>
<li>Description: Adapters provide priming sequences for both amplification and sequencing of the sample-library fragments. Both adapters should be reported; in uppercase letters</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: AATGATACGGCGACCACCGAGATCTACACGCT;CAAGCAGAAGACGGCATACGAGAT None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_21">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_seq_meth/">MIMAG➞seq_meth</a>  <sub>1..1</sub><ul>
<li>Description: Sequencing machine used. Where possible the term should be taken from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 454 Genome Sequencer FLX [OBI:0000702] None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_22">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_tax_ident/">MIMAG➞tax_ident</a>  <sub>1..1</sub><ul>
<li>Description: The phylogenetic marker(s) used to assign an organism name to the SAG or MAG</li>
<li>Range: <a href="../tax_ident_enum/">tax_ident_enum</a></li>
<li>Example: other: rpoB gene None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_23">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_assembly_qual/">MIMAG➞assembly_qual</a>  <sub>1..1</sub><ul>
<li>Description: The assembly quality category is based on sets of criteria outlined for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities with a consensus error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments where gaps span repetitive regions. Presence of the 23S, 16S and 5S rRNA genes and at least 18 tRNAs. Medium Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Low Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Assembly statistics include, but are not limited to total assembly size, number of contigs, contig N50/L50, and maximum contig length. For MIUVIG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities, with extensive manual review and editing to annotate putative gene functions and transcriptional units. High-quality draft genome: One or multiple fragments, totaling ≥ 90% of the expected genome or replicon sequence or predicted complete. Genome fragment(s): One or multiple fragments, totalling &lt; 90% of the expected genome or replicon sequence, or for which no genome size could be estimated</li>
<li>Range: <a href="../assembly_qual_enum/">assembly_qual_enum</a></li>
<li>Example: High-quality draft genome None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_24">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_assembly_name/">MIMAG➞assembly_name</a>  <sub>0..1</sub><ul>
<li>Description: Name/version of the assembly provided by the submitter that is used in the genome browsers and in the community</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: HuRef, JCVI_ISG_i3_1.0 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_25">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_assembly_software/">MIMAG➞assembly_software</a>  <sub>1..1</sub><ul>
<li>Description: Tool(s) used for assembly, including version number and parameters</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: metaSPAdes;3.11.0;kmer set 21,33,55,77,99,121, default parameters otherwise None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_26">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_annot/">MIMAG➞annot</a>  <sub>0..1</sub><ul>
<li>Description: Tool used for annotation, or for cases where annotation was provided by a community jamboree or model organism database rather than by a specific submitter</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: prokka None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_27">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_number_contig/">MIMAG➞number_contig</a>  <sub>0..1</sub><ul>
<li>Description: Total number of contigs in the cleaned/submitted assembly that makes up a given genome, SAG, MAG, or UViG</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 40 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_28">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_feat_pred/">MIMAG➞feat_pred</a>  <sub>0..1</sub><ul>
<li>Description: Method used to predict UViGs features such as ORFs, integration site, etc.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Prodigal;2.6.3;default parameters None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_29">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_ref_db/">MIMAG➞ref_db</a>  <sub>0..1</sub><ul>
<li>Description: List of database(s) used for ORF annotation, along with version number and reference to website or publication</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: pVOGs;5;http://dmk-brain.ecn.uiowa.edu/pVOGs/ Grazziotin et al. 2017 doi:10.1093/nar/gkw975 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_30">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_sim_search_meth/">MIMAG➞sim_search_meth</a>  <sub>0..1</sub><ul>
<li>Description: Tool used to compare ORFs with database, along with version and cutoffs used</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: HMMER3;3.1b2;hmmsearch, cutoff of 50 on score None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_31">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_tax_class/">MIMAG➞tax_class</a>  <sub>0..1</sub><ul>
<li>Description: Method used for taxonomic classification, along with reference database used, classification rank, and thresholds used to classify new genomes</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: vConTACT vContact2 (references from NCBI RefSeq v83, genus rank classification, default parameters) None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_32">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_x_16s_recover/">MIMAG➞x_16s_recover</a>  <sub>0..1</sub><ul>
<li>Description: Can a 16S gene be recovered from the submitted SAG or MAG?</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: yes None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_33">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_x_16s_recover_software/">MIMAG➞x_16s_recover_software</a>  <sub>0..1</sub><ul>
<li>Description: Tools used for 16S rRNA gene extraction</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: rambl;v2;default parameters None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_34">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_trnas/">MIMAG➞trnas</a>  <sub>0..1</sub><ul>
<li>Description: The total number of tRNAs identified from the SAG or MAG</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 18 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_35">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_trna_ext_software/">MIMAG➞trna_ext_software</a>  <sub>0..1</sub><ul>
<li>Description: Tools used for tRNA identification</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: infernal;v2;default parameters None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_36">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_compl_score/">MIMAG➞compl_score</a>  <sub>1..1</sub><ul>
<li>Description: Completeness score is typically based on either the fraction of markers found as compared to a database or the percent of a genome found as compared to a closely related reference genome. High Quality Draft: &gt;90%, Medium Quality Draft: &gt;50%, and Low Quality Draft: &lt; 50% should have the indicated completeness scores</li>
<li>Range: <a href="../compl_score_enum/">compl_score_enum</a></li>
<li>Example: med;60% None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_37">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_compl_software/">MIMAG➞compl_software</a>  <sub>1..1</sub><ul>
<li>Description: Tools used for completion estimate, i.e. checkm, anvi'o, busco</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: checkm None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_38">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_compl_appr/">MIMAG➞compl_appr</a>  <sub>0..1</sub><ul>
<li>Description: The approach used to determine the completeness of a given genomic assembly, which would typically make use of a set of conserved marker genes or a closely related reference genome. For UViG completeness, include reference genome or group used, and contig feature suggesting a complete genome</li>
<li>Range: <a href="../compl_appr_enum/">compl_appr_enum</a></li>
<li>Example: other: UViG length compared to the average length of reference genomes from the P22virus genus (NCBI RefSeq v83) None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_39">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_contam_score/">MIMAG➞contam_score</a>  <sub>1..1</sub><ul>
<li>Description: The contamination score is based on the fraction of single-copy genes that are observed more than once in a query genome. The following scores are acceptable for; High Quality Draft: &lt; 5%, Medium Quality Draft: &lt; 10%, Low Quality Draft: &lt; 10%. Contamination must be below 5% for a SAG or MAG to be deposited into any of the public databases</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 1% None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_40">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_contam_screen_input/">MIMAG➞contam_screen_input</a>  <sub>0..1</sub><ul>
<li>Description: The type of sequence data used as input</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: contigs None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_41">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_contam_screen_param/">MIMAG➞contam_screen_param</a>  <sub>0..1</sub><ul>
<li>Description: Specific parameters used in the decontamination sofware, such as reference database, coverage, and kmers. Combinations of these parameters may also be used, i.e. kmer and coverage, or reference database and kmer</li>
<li>Range: <a href="../contam_screen_param_enum/">contam_screen_param_enum</a></li>
<li>Example: kmer None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_42">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_decontam_software/">MIMAG➞decontam_software</a>  <sub>0..1</sub><ul>
<li>Description: Tool(s) used in contamination screening</li>
<li>Range: <a href="../decontam_software_enum/">decontam_software_enum</a></li>
<li>Example: anvi'o None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_43">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_bin_param/">MIMAG➞bin_param</a>  <sub>1..1</sub><ul>
<li>Description: The parameters that have been applied during the extraction of genomes from metagenomic datasets</li>
<li>Range: <a href="../bin_param_enum/">bin_param_enum</a></li>
<li>Example: coverage and kmer None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_44">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_bin_software/">MIMAG➞bin_software</a>  <sub>1..1</sub><ul>
<li>Description: Tool(s) used for the extraction of genomes from metagenomic datasets, where possible include a product ID (PID) of the tool(s) used.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: MetaCluster-TA (RRID:SCR_004599), MaxBin (biotools:maxbin) None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_45">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_reassembly_bin/">MIMAG➞reassembly_bin</a>  <sub>0..1</sub><ul>
<li>Description: Has an assembly been performed on a genome bin extracted from a metagenomic assembly?</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: no None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_46">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_mag_cov_software/">MIMAG➞mag_cov_software</a>  <sub>0..1</sub><ul>
<li>Description: Tool(s) used to determine the genome coverage if coverage is used as a binning parameter in the extraction of genomes from metagenomic datasets</li>
<li>Range: <a href="../mag_cov_software_enum/">mag_cov_software_enum</a></li>
<li>Example: bbmap None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_47">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_associated_resource/">MIMAG➞associated resource</a>  <sub>0..1</sub><ul>
<li>Description: A related resource that is referenced, cited, or otherwise associated to the sequence.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: http://www.earthmicrobiome.org/ None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-mimag_48">Mixed in from MIMAG:</h3>
<ul>
<li><a href="../MIMAG_sop/">MIMAG➞sop</a>  <sub>0..1</sub><ul>
<li>Description: Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: http://press.igsb.anl.gov/earthmicrobiome/protocols-and-standards/its/ None</li>
</ul>
</li>
</ul>
              
            </div>
          </div>
          <footer>
  

  <hr/>

  <div role="contentinfo">
    <!-- Copyright etc -->
    
  </div>

  Built with <a href="https://www.mkdocs.org/">MkDocs</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
</footer>
      
        </div>
      </div>

    </section>

  </div>

  <div class="rst-versions" role="note" aria-label="versions">
  <span class="rst-current-version" data-toggle="rst-current-version">
    
    
    
  </span>
</div>
    <script>var base_url = '..';</script>
    <script src="../js/theme_extra.js" defer></script>
    <script src="../js/theme.js" defer></script>
      <script src="../search/main.js" defer></script>
    <script defer>
        window.onload = function () {
            SphinxRtdTheme.Navigation.enable(true);
        };
    </script>

</body>
</html>
