<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  
  
  <link rel="shortcut icon" href="../img/favicon.ico">
  <title>Class: host-associated MIGS eukaryote - MIxS</title>
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:400,700|Roboto+Slab:400,700|Inconsolata:400,700" />

  <link rel="stylesheet" href="../css/theme.css" />
  <link rel="stylesheet" href="../css/theme_extra.css" />
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/styles/github.min.css" />
  
  <script>
    // Current page data
    var mkdocs_page_name = "Class: host-associated MIGS eukaryote";
    var mkdocs_page_input_path = "Host-associatedMIGSEukaryote.md";
    var mkdocs_page_url = null;
  </script>
  
  <script src="../js/jquery-2.1.1.min.js" defer></script>
  <script src="../js/modernizr-2.8.3.min.js" defer></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/highlight.min.js"></script>
  <script>hljs.initHighlightingOnLoad();</script> 
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
    <div class="wy-side-scroll">
      <div class="wy-side-nav-search">
        <a href=".." class="icon icon-home"> MIxS</a>
        <div role="search">
  <form id ="rtd-search-form" class="wy-form" action="../search.html" method="get">
      <input type="text" name="q" placeholder="Search docs" title="Type search term here" />
  </form>
</div>
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
                <ul>
                    <li class="toctree-l1"><a class="" href="../index.md">Index</a>
                    </li>
                </ul>
      </div>
    </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" role="navigation" aria-label="top navigation">
        <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
        <a href="..">MIxS</a>
      </nav>

      
      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href="..">Docs</a> &raquo;</li>
    
      
    
    <li>Class: host-associated MIGS eukaryote</li>
    <li class="wy-breadcrumbs-aside">
      
    </li>
  </ul>
  
  <hr/>
</div>

          <div role="main">
            <div class="section">
              
                <h1 id="class-host-associated-migs-eukaryote">Class: host-associated MIGS eukaryote</h1>
<p>Combinatorial checklist Minimal Information about a Genome Sequence: eukaryote with environmental package host-associated</p>
<p>URI: <a href="https://w3id.org/mixs/vocab/Host-associatedMIGSEukaryote">mixs.vocab:Host-associatedMIGSEukaryote</a></p>
<p><a href="https://yuml.me/diagram/nofunky;dir:TB/class/[QuantityValue],[Host-associatedMIGSEukaryote&#124;samp_taxon_id:string;experimental_factor:string%20%3F;neg_cont_type:neg_cont_type_enum%20%3F;pos_cont_type:string%20%3F;subspecf_gen_lin:string%20%3F;ploidy:string%20%3F;num_replicons:integer%20%3F;extrachrom_elements:integer%20%3F;estimated_size:string%20%3F;ref_biomaterial:string%20%3F;source_mat_id:string%20%3F;pathogenicity:string%20%3F;biotic_relationship:biotic_relationship_enum%20%3F;specific_host:string%20%3F;host_spec_range:integer%20%3F;trophic_level:trophic_level_enum%20%3F;propagation:string%20%3F;isol_growth_condt:string;samp_collec_device:string%20%3F;samp_collec_method:string%20%3F;samp_mat_process:string%20%3F;nucl_acid_ext:string%20%3F;nucl_acid_amp:string%20%3F;lib_size:integer%20%3F;lib_reads_seqd:integer%20%3F;lib_layout:lib_layout_enum%20%3F;lib_vector:string%20%3F;lib_screen:string%20%3F;adapters:string%20%3F;seq_meth:string;tax_ident:tax_ident_enum%20%3F;assembly_qual:assembly_qual_enum;assembly_name:string%20%3F;assembly_software:string;annot:string%20%3F;number_contig:integer;feat_pred:string%20%3F;ref_db:string%20%3F;sim_search_meth:string%20%3F;tax_class:string%20%3F;compl_score:compl_score_enum%20%3F;compl_software:string%20%3F;associated_resource:string%20%3F;sop:string%20%3F;lat_lon(i):string%20%3F;geo_loc_name(i):string%20%3F;collection_date(i):date%20%3F;env_broad_scale(i):string%20%3F;env_local_scale(i):string%20%3F;env_medium(i):string%20%3F;ances_data(i):string%20%3F;biol_stat(i):biol_stat_enum%20%3F;genetic_mod(i):string%20%3F;host_common_name(i):string%20%3F;samp_capt_status(i):samp_capt_status_enum%20%3F;samp_dis_stage(i):samp_dis_stage_enum%20%3F;host_taxid(i):string%20%3F;host_subject_id(i):string%20%3F;host_life_stage(i):string%20%3F;host_sex(i):host_sex_enum%20%3F;chem_administration(i):string%20*;host_body_habitat(i):string%20%3F;host_body_site(i):string%20%3F;host_body_product(i):string%20%3F;host_diet(i):string%20*;host_last_meal(i):string%20*;host_growth_cond(i):string%20%3F;host_substrate(i):string%20%3F;host_family_relation(i):string%20*;host_subspecf_genlin(i):string%20*;host_genotype(i):string%20%3F;host_phenotype(i):string%20%3F;host_color(i):string%20%3F;host_shape(i):string%20%3F;gravidity(i):string%20%3F;perturbation(i):string%20*;oxy_stat_samp(i):oxy_stat_samp_enum%20%3F;organism_count(i):organism_count_enum%20*;samp_store_dur(i):string%20%3F;samp_store_loc(i):string%20%3F;host_symbiont(i):string%20*;misc_param(i):string%20*;samp_name(i):string;project_name(i):string;host_disease_stat(i):string%20%3F]uses%20-.-&gt;[MIGSEukaryote],[Host-associated]^-[Host-associatedMIGSEukaryote],[Host-associated],[MIGSEukaryote]"><img alt="img" src="https://yuml.me/diagram/nofunky;dir:TB/class/[QuantityValue],[Host-associatedMIGSEukaryote&#124;samp_taxon_id:string;experimental_factor:string%20%3F;neg_cont_type:neg_cont_type_enum%20%3F;pos_cont_type:string%20%3F;subspecf_gen_lin:string%20%3F;ploidy:string%20%3F;num_replicons:integer%20%3F;extrachrom_elements:integer%20%3F;estimated_size:string%20%3F;ref_biomaterial:string%20%3F;source_mat_id:string%20%3F;pathogenicity:string%20%3F;biotic_relationship:biotic_relationship_enum%20%3F;specific_host:string%20%3F;host_spec_range:integer%20%3F;trophic_level:trophic_level_enum%20%3F;propagation:string%20%3F;isol_growth_condt:string;samp_collec_device:string%20%3F;samp_collec_method:string%20%3F;samp_mat_process:string%20%3F;nucl_acid_ext:string%20%3F;nucl_acid_amp:string%20%3F;lib_size:integer%20%3F;lib_reads_seqd:integer%20%3F;lib_layout:lib_layout_enum%20%3F;lib_vector:string%20%3F;lib_screen:string%20%3F;adapters:string%20%3F;seq_meth:string;tax_ident:tax_ident_enum%20%3F;assembly_qual:assembly_qual_enum;assembly_name:string%20%3F;assembly_software:string;annot:string%20%3F;number_contig:integer;feat_pred:string%20%3F;ref_db:string%20%3F;sim_search_meth:string%20%3F;tax_class:string%20%3F;compl_score:compl_score_enum%20%3F;compl_software:string%20%3F;associated_resource:string%20%3F;sop:string%20%3F;lat_lon(i):string%20%3F;geo_loc_name(i):string%20%3F;collection_date(i):date%20%3F;env_broad_scale(i):string%20%3F;env_local_scale(i):string%20%3F;env_medium(i):string%20%3F;ances_data(i):string%20%3F;biol_stat(i):biol_stat_enum%20%3F;genetic_mod(i):string%20%3F;host_common_name(i):string%20%3F;samp_capt_status(i):samp_capt_status_enum%20%3F;samp_dis_stage(i):samp_dis_stage_enum%20%3F;host_taxid(i):string%20%3F;host_subject_id(i):string%20%3F;host_life_stage(i):string%20%3F;host_sex(i):host_sex_enum%20%3F;chem_administration(i):string%20*;host_body_habitat(i):string%20%3F;host_body_site(i):string%20%3F;host_body_product(i):string%20%3F;host_diet(i):string%20*;host_last_meal(i):string%20*;host_growth_cond(i):string%20%3F;host_substrate(i):string%20%3F;host_family_relation(i):string%20*;host_subspecf_genlin(i):string%20*;host_genotype(i):string%20%3F;host_phenotype(i):string%20%3F;host_color(i):string%20%3F;host_shape(i):string%20%3F;gravidity(i):string%20%3F;perturbation(i):string%20*;oxy_stat_samp(i):oxy_stat_samp_enum%20%3F;organism_count(i):organism_count_enum%20*;samp_store_dur(i):string%20%3F;samp_store_loc(i):string%20%3F;host_symbiont(i):string%20*;misc_param(i):string%20*;samp_name(i):string;project_name(i):string;host_disease_stat(i):string%20%3F]uses%20-.-&gt;[MIGSEukaryote],[Host-associated]^-[Host-associatedMIGSEukaryote],[Host-associated],[MIGSEukaryote]" /></a></p>
<h2 id="parents">Parents</h2>
<ul>
<li>is_a: <a href="../Host-associated/">Host-associated</a> - host-associated</li>
</ul>
<h2 id="uses-mixin">Uses Mixin</h2>
<ul>
<li>mixin: <a href="../MIGSEukaryote/">MIGSEukaryote</a> - Minimal Information about a Genome Sequence: eukaryote</li>
</ul>
<h2 id="attributes">Attributes</h2>
<h3 id="inherited-from-host-associated">Inherited from host-associated:</h3>
<ul>
<li><a href="../lat_lon/">lat_lon</a>  <sub>0..1</sub><ul>
<li>Description: The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 50.586825 6.408977 None</li>
</ul>
</li>
<li><a href="../host_associated_depth/">host-associated➞depth</a>  <sub>0..1</sub><ul>
<li>Description: The vertical distance below local surface, e.g. for sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 10 meter None</li>
</ul>
</li>
<li><a href="../host_associated_alt/">host-associated➞alt</a>  <sub>0..1</sub><ul>
<li>Description: Altitude is a term used to identify heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth's surface. In this context, the altitude measurement is the vertical distance between the earth's surface above sea level and the sampled position in the air</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 100 meter None</li>
</ul>
</li>
<li><a href="../host_associated_elev/">host-associated➞elev</a>  <sub>0..1</sub><ul>
<li>Description: Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth's surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 100 meter None</li>
</ul>
</li>
<li><a href="../host_associated_temp/">host-associated➞temp</a>  <sub>0..1</sub><ul>
<li>Description: Temperature of the sample at the time of sampling.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 25 degree Celsius None</li>
</ul>
</li>
<li><a href="../geo_loc_name/">geo_loc_name</a>  <sub>0..1</sub><ul>
<li>Description: The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: USA: Maryland, Bethesda None</li>
</ul>
</li>
<li><a href="../collection_date/">collection_date</a>  <sub>0..1</sub><ul>
<li>Description: The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant</li>
<li>Range: <a href="../types/Date/">Date</a></li>
<li>Example: 2018-05-11T10:00:00+01:00; 2018-05-11 None</li>
</ul>
</li>
<li><a href="../env_broad_scale/">env_broad_scale</a>  <sub>0..1</sub><ul>
<li>Description: Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO’s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: oceanic epipelagic zone biome [ENVO:01000033] for annotating a water sample from the photic zone in middle of the Atlantic Ocean None</li>
</ul>
</li>
<li><a href="../env_local_scale/">env_local_scale</a>  <sub>0..1</sub><ul>
<li>Description: Report the entity or entities which are in the sample or specimen’s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: litter layer [ENVO:01000338]; Annotating a pooled sample taken from various vegetation layers in a forest consider: canopy [ENVO:00000047]|herb and fern layer [ENVO:01000337]|litter layer [ENVO:01000338]|understory [01000335]|shrub layer [ENVO:01000336]. None</li>
</ul>
</li>
<li><a href="../env_medium/">env_medium</a>  <sub>0..1</sub><ul>
<li>Description: Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of 'environmental material' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: soil [ENVO:00001998]; Annotating a fish swimming in the upper 100 m of the Atlantic Ocean, consider: ocean water [ENVO:00002151]. Example: Annotating a duck on a pond consider: pond water [ENVO:00002228]|air [ENVO_00002005] None</li>
</ul>
</li>
<li><a href="../host_associated_ances_data/">host-associated➞ances_data</a>  <sub>0..1</sub><ul>
<li>Description: Information about either pedigree or other ancestral information description (e.g. parental variety in case of mutant or selection), e.g. A/3*B (meaning [(A x B) x B] x B)</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: A/3*B None</li>
</ul>
</li>
<li><a href="../host_associated_biol_stat/">host-associated➞biol_stat</a>  <sub>0..1</sub><ul>
<li>Description: The level of genome modification.</li>
<li>Range: <a href="../biol_stat_enum/">biol_stat_enum</a></li>
<li>Example: natural None</li>
</ul>
</li>
<li><a href="../host_associated_genetic_mod/">host-associated➞genetic_mod</a>  <sub>0..1</sub><ul>
<li>Description: Genetic modifications of the genome of an organism, which may occur naturally by spontaneous mutation, or be introduced by some experimental means, e.g. specification of a transgene or the gene knocked-out or details of transient transfection</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: aox1A transgenic None</li>
</ul>
</li>
<li><a href="../host_associated_host_common_name/">host-associated➞host_common_name</a>  <sub>0..1</sub><ul>
<li>Description: Common name of the host.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: human None</li>
</ul>
</li>
<li><a href="../host_associated_samp_capt_status/">host-associated➞samp_capt_status</a>  <sub>0..1</sub><ul>
<li>Description: Reason for the sample</li>
<li>Range: <a href="../samp_capt_status_enum/">samp_capt_status_enum</a></li>
<li>Example: farm sample None</li>
</ul>
</li>
<li><a href="../host_associated_samp_dis_stage/">host-associated➞samp_dis_stage</a>  <sub>0..1</sub><ul>
<li>Description: Stage of the disease at the time of sample collection, e.g. inoculation, penetration, infection, growth and reproduction, dissemination of pathogen.</li>
<li>Range: <a href="../samp_dis_stage_enum/">samp_dis_stage_enum</a></li>
<li>Example: infection None</li>
</ul>
</li>
<li><a href="../host_associated_host_taxid/">host-associated➞host_taxid</a>  <sub>0..1</sub><ul>
<li>Description: NCBI taxon id of the host, e.g. 9606</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 7955 None</li>
</ul>
</li>
<li><a href="../host_associated_host_subject_id/">host-associated➞host_subject_id</a>  <sub>0..1</sub><ul>
<li>Description: A unique identifier by which each subject can be referred to, de-identified.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: MPI123 None</li>
</ul>
</li>
<li><a href="../host_associated_host_age/">host-associated➞host_age</a>  <sub>0..1</sub><ul>
<li>Description: Age of host at the time of sampling; relevant scale depends on species and study, e.g. Could be seconds for amoebae or centuries for trees</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 10 days None</li>
</ul>
</li>
<li><a href="../host_associated_host_life_stage/">host-associated➞host_life_stage</a>  <sub>0..1</sub><ul>
<li>Description: Description of life stage of host</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: adult None</li>
</ul>
</li>
<li><a href="../host_associated_host_sex/">host-associated➞host_sex</a>  <sub>0..1</sub><ul>
<li>Description: Gender or physical sex of the host.</li>
<li>Range: <a href="../host_sex_enum/">host_sex_enum</a></li>
<li>Example: non-binary None</li>
</ul>
</li>
<li><a href="../host_associated_chem_administration/">host-associated➞chem_administration</a>  <sub>0..*</sub><ul>
<li>Description: List of chemical compounds administered to the host or site where sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can include multiple compounds. For chemical entities of biological interest ontology (chebi) (v 163), http://purl.bioontology.org/ontology/chebi</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: agar [CHEBI:2509];2018-05-11T20:00Z None</li>
</ul>
</li>
<li><a href="../host_associated_host_body_habitat/">host-associated➞host_body_habitat</a>  <sub>0..1</sub><ul>
<li>Description: Original body habitat where the sample was obtained from</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: nasopharynx None</li>
</ul>
</li>
<li><a href="../host_associated_host_body_site/">host-associated➞host_body_site</a>  <sub>0..1</sub><ul>
<li>Description: Name of body site where the sample was obtained from, such as a specific organ or tissue (tongue, lung etc...). For foundational model of anatomy ontology (fma) (v 4.11.0) or Uber-anatomy ontology (UBERON) (v releases/2014-06-15) terms, please see http://purl.bioontology.org/ontology/FMA or http://purl.bioontology.org/ontology/UBERON</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: gill [UBERON:0002535] None</li>
</ul>
</li>
<li><a href="../host_associated_host_body_product/">host-associated➞host_body_product</a>  <sub>0..1</sub><ul>
<li>Description: Substance produced by the body, e.g. Stool, mucus, where the sample was obtained from. For foundational model of anatomy ontology (fma) or Uber-anatomy ontology (UBERON) terms, please see https://www.ebi.ac.uk/ols/ontologies/fma or https://www.ebi.ac.uk/ols/ontologies/uberon</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Portion of mucus [fma66938] None</li>
</ul>
</li>
<li><a href="../host_associated_host_tot_mass/">host-associated➞host_tot_mass</a>  <sub>0..1</sub><ul>
<li>Description: Total mass of the host at collection, the unit depends on host</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 2500 gram None</li>
</ul>
</li>
<li><a href="../host_associated_host_height/">host-associated➞host_height</a>  <sub>0..1</sub><ul>
<li>Description: The height of subject</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 0.1 meter None</li>
</ul>
</li>
<li><a href="../host_associated_host_length/">host-associated➞host_length</a>  <sub>0..1</sub><ul>
<li>Description: The length of subject</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 1 meter None</li>
</ul>
</li>
<li><a href="../host_associated_host_diet/">host-associated➞host_diet</a>  <sub>0..*</sub><ul>
<li>Description: Type of diet depending on the host, for animals omnivore, herbivore etc., for humans high-fat, meditteranean etc.; can include multiple diet types</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: herbivore None</li>
</ul>
</li>
<li><a href="../host_associated_host_last_meal/">host-associated➞host_last_meal</a>  <sub>0..*</sub><ul>
<li>Description: Content of last meal and time since feeding; can include multiple values</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: corn feed;P2H None</li>
</ul>
</li>
<li><a href="../host_associated_host_growth_cond/">host-associated➞host_growth_cond</a>  <sub>0..1</sub><ul>
<li>Description: Literature reference giving growth conditions of the host</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: https://academic.oup.com/icesjms/article/68/2/349/617247 None</li>
</ul>
</li>
<li><a href="../host_associated_host_substrate/">host-associated➞host_substrate</a>  <sub>0..1</sub><ul>
<li>Description: The growth substrate of the host.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: rock None</li>
</ul>
</li>
<li><a href="../host_associated_host_family_relation/">host-associated➞host_family_relation</a>  <sub>0..*</sub><ul>
<li>Description: Familial relationships to other hosts in the same study; can include multiple relationships</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: offspring;Mussel25 None</li>
</ul>
</li>
<li><a href="../host_associated_host_subspecf_genlin/">host-associated➞host_subspecf_genlin</a>  <sub>0..*</sub><ul>
<li>Description: Information about the genetic distinctness of the host organism below the subspecies level e.g., serovar, serotype, biotype, ecotype, variety, cultivar, or any relevant genetic typing schemes like Group I plasmid. Subspecies should not be recorded in this term, but in the NCBI taxonomy. Supply both the lineage name and the lineage rank separated by a colon, e.g., biovar:abc123.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: serovar:Newport, variety:glabrum, cultivar: Red Delicious None</li>
</ul>
</li>
<li><a href="../host_associated_host_genotype/">host-associated➞host_genotype</a>  <sub>0..1</sub><ul>
<li>Description: Observed genotype</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: C57BL/6 None</li>
</ul>
</li>
<li><a href="../host_associated_host_phenotype/">host-associated➞host_phenotype</a>  <sub>0..1</sub><ul>
<li>Description: Phenotype of human or other host. For phenotypic quality ontology (pato) (v 2018-03-27) terms, please see http://purl.bioontology.org/ontology/pato. For Human Phenotype Ontology (HP) (v 2018-06-13) please see http://purl.bioontology.org/ontology/HP</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: elongated [PATO:0001154] None</li>
</ul>
</li>
<li><a href="../host_associated_host_body_temp/">host-associated➞host_body_temp</a>  <sub>0..1</sub><ul>
<li>Description: Core body temperature of the host when sample was collected</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 15 degree Celsius None</li>
</ul>
</li>
<li><a href="../host_associated_host_dry_mass/">host-associated➞host_dry_mass</a>  <sub>0..1</sub><ul>
<li>Description: Measurement of dry mass</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 500 gram None</li>
</ul>
</li>
<li><a href="../host_associated_blood_press_diast/">host-associated➞blood_press_diast</a>  <sub>0..1</sub><ul>
<li>Description: Resting diastolic blood pressure, measured as mm mercury</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../host_associated_blood_press_syst/">host-associated➞blood_press_syst</a>  <sub>0..1</sub><ul>
<li>Description: Resting systolic blood pressure, measured as mm mercury</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../host_associated_host_color/">host-associated➞host_color</a>  <sub>0..1</sub><ul>
<li>Description: The color of host</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
<li><a href="../host_associated_host_shape/">host-associated➞host_shape</a>  <sub>0..1</sub><ul>
<li>Description: Morphological shape of host</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: round None</li>
</ul>
</li>
<li><a href="../host_associated_gravidity/">host-associated➞gravidity</a>  <sub>0..1</sub><ul>
<li>Description: Whether or not subject is gravid, and if yes date due or date post-conception, specifying which is used</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: yes;due date:2018-05-11 None</li>
</ul>
</li>
<li><a href="../host_associated_perturbation/">host-associated➞perturbation</a>  <sub>0..*</sub><ul>
<li>Description: Type of perturbation, e.g. chemical administration, physical disturbance, etc., coupled with perturbation regimen including how many times the perturbation was repeated, how long each perturbation lasted, and the start and end time of the entire perturbation period; can include multiple perturbation types</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: antibiotic addition;R2/2018-05-11T14:30Z/2018-05-11T19:30Z/P1H30M None</li>
</ul>
</li>
<li><a href="../host_associated_salinity/">host-associated➞salinity</a>  <sub>0..1</sub><ul>
<li>Description: The total concentration of all dissolved salts in a liquid or solid sample. While salinity can be measured by a complete chemical analysis, this method is difficult and time consuming. More often, it is instead derived from the conductivity measurement. This is known as practical salinity. These derivations compare the specific conductance of the sample to a salinity standard such as seawater.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 25 practical salinity unit None</li>
</ul>
</li>
<li><a href="../host_associated_oxy_stat_samp/">host-associated➞oxy_stat_samp</a>  <sub>0..1</sub><ul>
<li>Description: Oxygenation status of sample</li>
<li>Range: <a href="../oxy_stat_samp_enum/">oxy_stat_samp_enum</a></li>
<li>Example: aerobic None</li>
</ul>
</li>
<li><a href="../host_associated_organism_count/">host-associated➞organism_count</a>  <sub>0..*</sub><ul>
<li>Description: Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)</li>
<li>Range: <a href="../organism_count_enum/">organism_count_enum</a></li>
<li>Example: total prokaryotes;3.5e7 cells per milliliter;qPCR None</li>
</ul>
</li>
<li><a href="../host_associated_samp_store_temp/">host-associated➞samp_store_temp</a>  <sub>0..1</sub><ul>
<li>Description: Temperature at which sample was stored, e.g. -80 degree Celsius</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: -80 degree Celsius None</li>
</ul>
</li>
<li><a href="../host_associated_samp_store_dur/">host-associated➞samp_store_dur</a>  <sub>0..1</sub><ul>
<li>Description: Duration for which the sample was stored</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: P1Y6M None</li>
</ul>
</li>
<li><a href="../host_associated_samp_store_loc/">host-associated➞samp_store_loc</a>  <sub>0..1</sub><ul>
<li>Description: Location at which sample was stored, usually name of a specific freezer/room</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Freezer no:5 None</li>
</ul>
</li>
<li><a href="../host_associated_host_symbiont/">host-associated➞host_symbiont</a>  <sub>0..*</sub><ul>
<li>Description: The taxonomic name of the organism(s) found living in mutualistic, commensalistic, or parasitic symbiosis with the specific host.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: flukeworms None</li>
</ul>
</li>
<li><a href="../host_associated_misc_param/">host-associated➞misc_param</a>  <sub>0..*</sub><ul>
<li>Description: Any other measurement performed or parameter collected, that is not listed here</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Bicarbonate ion concentration;2075 micromole per kilogram None</li>
</ul>
</li>
<li><a href="../host_associated_samp_name/">host-associated➞samp_name</a>  <sub>1..1</sub><ul>
<li>Description: A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: ISDsoil1 None</li>
</ul>
</li>
<li><a href="../host_associated_project_name/">host-associated➞project_name</a>  <sub>1..1</sub><ul>
<li>Description: Name of the project within which the sequencing was organized</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Forest soil metagenome None</li>
</ul>
</li>
<li><a href="../host_associated_host_disease_stat/">host-associated➞host_disease_stat</a>  <sub>0..1</sub><ul>
<li>Description: List of diseases with which the host has been diagnosed; can include multiple diagnoses. The value of the field depends on host; for humans the terms should be chosen from the DO (Human Disease Ontology) at https://www.disease-ontology.org, non-human host diseases are free text</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: rabies [DOID:11260] None</li>
</ul>
</li>
<li><a href="../host_associated_samp_vol_we_dna_ext/">host-associated➞samp_vol_we_dna_ext</a>  <sub>0..1</sub><ul>
<li>Description: Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001).</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 1500 milliliter None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_samp_taxon_id/">MIGS eukaryote➞samp_taxon_id</a>  <sub>1..1</sub><ul>
<li>Description: NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa sample. Use 'synthetic metagenome’ for mock community/positive controls, or 'blank sample' for negative controls.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Gut Metagenome [NCBI:txid749906] None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_1">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_experimental_factor/">MIGS eukaryote➞experimental_factor</a>  <sub>0..1</sub><ul>
<li>Description: Experimental factors are essentially the variable aspects of an experiment design which can be used to describe an experiment, or set of experiments, in an increasingly detailed manner. This field accepts ontology terms from Experimental Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI). For a browser of EFO (v 2.95) terms, please see http://purl.bioontology.org/ontology/EFO; for a browser of OBI (v 2018-02-12) terms please see http://purl.bioontology.org/ontology/OBI</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: time series design [EFO:EFO_0001779] None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_2">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_neg_cont_type/">MIGS eukaryote➞neg_cont_type</a>  <sub>0..1</sub><ul>
<li>Description: The substance or equipment used as a negative control in an investigation</li>
<li>Range: <a href="../neg_cont_type_enum/">neg_cont_type_enum</a></li>
<li>Example:  None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_3">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_pos_cont_type/">MIGS eukaryote➞pos_cont_type</a>  <sub>0..1</sub><ul>
<li>Description: The substance, mixture, product, or apparatus used to verify that a process which is part of an investigation delivers a true positive.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_4">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_subspecf_gen_lin/">MIGS eukaryote➞subspecf_gen_lin</a>  <sub>0..1</sub><ul>
<li>Description: Information about the genetic distinctness of the sequenced organism below the subspecies level, e.g., serovar, serotype, biotype, ecotype, or any relevant genetic typing schemes like Group I plasmid. Subspecies should not be recorded in this term, but in the NCBI taxonomy. Supply both the lineage name and the lineage rank separated by a colon, e.g., biovar:abc123.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: serovar:Newport None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_5">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_ploidy/">MIGS eukaryote➞ploidy</a>  <sub>0..1</sub><ul>
<li>Description: The ploidy level of the genome (e.g. allopolyploid, haploid, diploid, triploid, tetraploid). It has implications for the downstream study of duplicated gene and regions of the genomes (and perhaps for difficulties in assembly). For terms, please select terms listed under class ploidy (PATO:001374) of Phenotypic Quality Ontology (PATO), and for a browser of PATO (v 2018-03-27) please refer to http://purl.bioontology.org/ontology/PATO</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: allopolyploidy [PATO:0001379] None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_6">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_num_replicons/">MIGS eukaryote➞num_replicons</a>  <sub>0..1</sub><ul>
<li>Description: Reports the number of replicons in a nuclear genome of eukaryotes, in the genome of a bacterium or archaea or the number of segments in a segmented virus. Always applied to the haploid chromosome count of a eukaryote</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 2 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_7">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_extrachrom_elements/">MIGS eukaryote➞extrachrom_elements</a>  <sub>0..1</sub><ul>
<li>Description: Do plasmids exist of significant phenotypic consequence (e.g. ones that determine virulence or antibiotic resistance). Megaplasmids? Other plasmids (borrelia has 15+ plasmids)</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 5 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_8">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_estimated_size/">MIGS eukaryote➞estimated_size</a>  <sub>0..1</sub><ul>
<li>Description: The estimated size of the genome prior to sequencing. Of particular importance in the sequencing of (eukaryotic) genome which could remain in draft form for a long or unspecified period.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 300000 bp None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_9">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_ref_biomaterial/">MIGS eukaryote➞ref_biomaterial</a>  <sub>0..1</sub><ul>
<li>Description: Primary publication if isolated before genome publication; otherwise, primary genome report.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: doi:10.1016/j.syapm.2018.01.009 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_10">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_source_mat_id/">MIGS eukaryote➞source_mat_id</a>  <sub>0..1</sub><ul>
<li>Description: A unique identifier assigned to a material sample (as defined by http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular digital record of a material sample) used for extracting nucleic acids, and subsequent sequencing. The identifier can refer either to the original material collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher, /bio_material, or /culture_collection may or may not share the same value as the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id may both contain 'UAM:Herps:14' , referring to both the specimen voucher and sampled tissue with the same identifier. However, the /culture_collection qualifier may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id would refer to an identifier from some derived culture from which the nucleic acids were extracted (e.g. xatc123 or ark:/2154/R2).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: MPI012345 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_11">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_pathogenicity/">MIGS eukaryote➞pathogenicity</a>  <sub>0..1</sub><ul>
<li>Description: To what is the entity pathogenic</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: human, animal, plant, fungi, bacteria None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_12">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_biotic_relationship/">MIGS eukaryote➞biotic_relationship</a>  <sub>0..1</sub><ul>
<li>Description: Description of relationship(s) between the subject organism and other organism(s) it is associated with. E.g., parasite on species X; mutualist with species Y. The target organism is the subject of the relationship, and the other organism(s) is the object</li>
<li>Range: <a href="../biotic_relationship_enum/">biotic_relationship_enum</a></li>
<li>Example: free living None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_13">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_specific_host/">MIGS eukaryote➞specific_host</a>  <sub>0..1</sub><ul>
<li>Description: Report the host's taxonomic name and/or NCBI taxonomy ID.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Homo sapiens and/or 9606 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_14">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_host_spec_range/">MIGS eukaryote➞host_spec_range</a>  <sub>0..1</sub><ul>
<li>Description: The range and diversity of host species that an organism is capable of infecting, defined by NCBI taxonomy identifier.</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 9606 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_15">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_trophic_level/">MIGS eukaryote➞trophic_level</a>  <sub>0..1</sub><ul>
<li>Description: Trophic levels are the feeding position in a food chain. Microbes can be a range of producers (e.g. chemolithotroph)</li>
<li>Range: <a href="../trophic_level_enum/">trophic_level_enum</a></li>
<li>Example: heterotroph None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_16">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_propagation/">MIGS eukaryote➞propagation</a>  <sub>0..1</sub><ul>
<li>Description: The type of reproduction from the parent stock. Values for this field is specific to different taxa. For phage or virus: lytic/lysogenic/temperate/obligately lytic. For plasmids: incompatibility group. For eukaryotes: sexual/asexual.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: lytic None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_17">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_isol_growth_condt/">MIGS eukaryote➞isol_growth_condt</a>  <sub>1..1</sub><ul>
<li>Description: Publication reference in the form of pubmed ID (pmid), digital object identifier (doi) or url for isolation and growth condition specifications of the organism/material</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: doi: 10.1016/j.syapm.2018.01.009 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_18">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_samp_collec_device/">MIGS eukaryote➞samp_collec_device</a>  <sub>0..1</sub><ul>
<li>Description: The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: swab, biopsy, niskin bottle, push core, drag swab [GENEPIO:0002713] None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_19">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_samp_collec_method/">MIGS eukaryote➞samp_collec_method</a>  <sub>0..1</sub><ul>
<li>Description: The method employed for collecting the sample.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: swabbing None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_20">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_samp_mat_process/">MIGS eukaryote➞samp_mat_process</a>  <sub>0..1</sub><ul>
<li>Description: A brief description of any processing applied to the sample during or after retrieving the sample from environment, or a link to the relevant protocol(s) performed.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: filtering of seawater, storing samples in ethanol None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_21">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_samp_size/">MIGS eukaryote➞samp_size</a>  <sub>0..1</sub><ul>
<li>Description: The total amount or size (volume (ml), mass (g) or area (m2) ) of sample collected.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 5 liter None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_22">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_nucl_acid_ext/">MIGS eukaryote➞nucl_acid_ext</a>  <sub>0..1</sub><ul>
<li>Description: A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the material separation to recover the nucleic acid fraction from a sample</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: https://mobio.com/media/wysiwyg/pdfs/protocols/12888.pdf None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_23">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_nucl_acid_amp/">MIGS eukaryote➞nucl_acid_amp</a>  <sub>0..1</sub><ul>
<li>Description: A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the enzymatic amplification (PCR, TMA, NASBA) of specific nucleic acids</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: https://phylogenomics.me/protocols/16s-pcr-protocol/ None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_24">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_lib_size/">MIGS eukaryote➞lib_size</a>  <sub>0..1</sub><ul>
<li>Description: Total number of clones in the library prepared for the project</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 50 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_25">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_lib_reads_seqd/">MIGS eukaryote➞lib_reads_seqd</a>  <sub>0..1</sub><ul>
<li>Description: Total number of clones sequenced from the library</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 20 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_26">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_lib_layout/">MIGS eukaryote➞lib_layout</a>  <sub>0..1</sub><ul>
<li>Description: Specify whether to expect single, paired, or other configuration of reads</li>
<li>Range: <a href="../lib_layout_enum/">lib_layout_enum</a></li>
<li>Example: paired None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_27">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_lib_vector/">MIGS eukaryote➞lib_vector</a>  <sub>0..1</sub><ul>
<li>Description: Cloning vector type(s) used in construction of libraries</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Bacteriophage P1 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_28">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_lib_screen/">MIGS eukaryote➞lib_screen</a>  <sub>0..1</sub><ul>
<li>Description: Specific enrichment or screening methods applied before and/or after creating libraries</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: enriched, screened, normalized None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_29">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_adapters/">MIGS eukaryote➞adapters</a>  <sub>0..1</sub><ul>
<li>Description: Adapters provide priming sequences for both amplification and sequencing of the sample-library fragments. Both adapters should be reported; in uppercase letters</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: AATGATACGGCGACCACCGAGATCTACACGCT;CAAGCAGAAGACGGCATACGAGAT None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_30">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_seq_meth/">MIGS eukaryote➞seq_meth</a>  <sub>1..1</sub><ul>
<li>Description: Sequencing machine used. Where possible the term should be taken from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 454 Genome Sequencer FLX [OBI:0000702] None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_31">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_tax_ident/">MIGS eukaryote➞tax_ident</a>  <sub>0..1</sub><ul>
<li>Description: The phylogenetic marker(s) used to assign an organism name to the SAG or MAG</li>
<li>Range: <a href="../tax_ident_enum/">tax_ident_enum</a></li>
<li>Example: other: rpoB gene None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_32">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_assembly_qual/">MIGS eukaryote➞assembly_qual</a>  <sub>1..1</sub><ul>
<li>Description: The assembly quality category is based on sets of criteria outlined for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities with a consensus error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments where gaps span repetitive regions. Presence of the 23S, 16S and 5S rRNA genes and at least 18 tRNAs. Medium Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Low Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Assembly statistics include, but are not limited to total assembly size, number of contigs, contig N50/L50, and maximum contig length. For MIUVIG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities, with extensive manual review and editing to annotate putative gene functions and transcriptional units. High-quality draft genome: One or multiple fragments, totaling ≥ 90% of the expected genome or replicon sequence or predicted complete. Genome fragment(s): One or multiple fragments, totalling &lt; 90% of the expected genome or replicon sequence, or for which no genome size could be estimated</li>
<li>Range: <a href="../assembly_qual_enum/">assembly_qual_enum</a></li>
<li>Example: High-quality draft genome None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_33">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_assembly_name/">MIGS eukaryote➞assembly_name</a>  <sub>0..1</sub><ul>
<li>Description: Name/version of the assembly provided by the submitter that is used in the genome browsers and in the community</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: HuRef, JCVI_ISG_i3_1.0 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_34">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_assembly_software/">MIGS eukaryote➞assembly_software</a>  <sub>1..1</sub><ul>
<li>Description: Tool(s) used for assembly, including version number and parameters</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: metaSPAdes;3.11.0;kmer set 21,33,55,77,99,121, default parameters otherwise None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_35">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_annot/">MIGS eukaryote➞annot</a>  <sub>0..1</sub><ul>
<li>Description: Tool used for annotation, or for cases where annotation was provided by a community jamboree or model organism database rather than by a specific submitter</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: prokka None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_36">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_number_contig/">MIGS eukaryote➞number_contig</a>  <sub>1..1</sub><ul>
<li>Description: Total number of contigs in the cleaned/submitted assembly that makes up a given genome, SAG, MAG, or UViG</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 40 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_37">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_feat_pred/">MIGS eukaryote➞feat_pred</a>  <sub>0..1</sub><ul>
<li>Description: Method used to predict UViGs features such as ORFs, integration site, etc.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Prodigal;2.6.3;default parameters None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_38">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_ref_db/">MIGS eukaryote➞ref_db</a>  <sub>0..1</sub><ul>
<li>Description: List of database(s) used for ORF annotation, along with version number and reference to website or publication</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: pVOGs;5;http://dmk-brain.ecn.uiowa.edu/pVOGs/ Grazziotin et al. 2017 doi:10.1093/nar/gkw975 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_39">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_sim_search_meth/">MIGS eukaryote➞sim_search_meth</a>  <sub>0..1</sub><ul>
<li>Description: Tool used to compare ORFs with database, along with version and cutoffs used</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: HMMER3;3.1b2;hmmsearch, cutoff of 50 on score None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_40">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_tax_class/">MIGS eukaryote➞tax_class</a>  <sub>0..1</sub><ul>
<li>Description: Method used for taxonomic classification, along with reference database used, classification rank, and thresholds used to classify new genomes</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: vConTACT vContact2 (references from NCBI RefSeq v83, genus rank classification, default parameters) None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_41">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_compl_score/">MIGS eukaryote➞compl_score</a>  <sub>0..1</sub><ul>
<li>Description: Completeness score is typically based on either the fraction of markers found as compared to a database or the percent of a genome found as compared to a closely related reference genome. High Quality Draft: &gt;90%, Medium Quality Draft: &gt;50%, and Low Quality Draft: &lt; 50% should have the indicated completeness scores</li>
<li>Range: <a href="../compl_score_enum/">compl_score_enum</a></li>
<li>Example: med;60% None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_42">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_compl_software/">MIGS eukaryote➞compl_software</a>  <sub>0..1</sub><ul>
<li>Description: Tools used for completion estimate, i.e. checkm, anvi'o, busco</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: checkm None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_43">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_associated_resource/">MIGS eukaryote➞associated resource</a>  <sub>0..1</sub><ul>
<li>Description: A related resource that is referenced, cited, or otherwise associated to the sequence.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: http://www.earthmicrobiome.org/ None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-eukaryote_44">Mixed in from MIGS eukaryote:</h3>
<ul>
<li><a href="../MIGS_eukaryote_sop/">MIGS eukaryote➞sop</a>  <sub>0..1</sub><ul>
<li>Description: Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: http://press.igsb.anl.gov/earthmicrobiome/protocols-and-standards/its/ None</li>
</ul>
</li>
</ul>
              
            </div>
          </div>
          <footer>
  

  <hr/>

  <div role="contentinfo">
    <!-- Copyright etc -->
    
  </div>

  Built with <a href="https://www.mkdocs.org/">MkDocs</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
</footer>
      
        </div>
      </div>

    </section>

  </div>

  <div class="rst-versions" role="note" aria-label="versions">
  <span class="rst-current-version" data-toggle="rst-current-version">
    
    
    
  </span>
</div>
    <script>var base_url = '..';</script>
    <script src="../js/theme_extra.js" defer></script>
    <script src="../js/theme.js" defer></script>
      <script src="../search/main.js" defer></script>
    <script defer>
        window.onload = function () {
            SphinxRtdTheme.Navigation.enable(true);
        };
    </script>

</body>
</html>
