<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  
  
  <link rel="shortcut icon" href="../img/favicon.ico">
  <title>Class: plant-associated MIGS bacteria - MIxS</title>
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Lato:400,700|Roboto+Slab:400,700|Inconsolata:400,700" />

  <link rel="stylesheet" href="../css/theme.css" />
  <link rel="stylesheet" href="../css/theme_extra.css" />
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/styles/github.min.css" />
  
  <script>
    // Current page data
    var mkdocs_page_name = "Class: plant-associated MIGS bacteria";
    var mkdocs_page_input_path = "Plant-associatedMIGSBacteria.md";
    var mkdocs_page_url = null;
  </script>
  
  <script src="../js/jquery-2.1.1.min.js" defer></script>
  <script src="../js/modernizr-2.8.3.min.js" defer></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/highlight.min.js"></script>
  <script>hljs.initHighlightingOnLoad();</script> 
</head>

<body class="wy-body-for-nav" role="document">

  <div class="wy-grid-for-nav">

    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side stickynav">
    <div class="wy-side-scroll">
      <div class="wy-side-nav-search">
        <a href=".." class="icon icon-home"> MIxS</a>
        <div role="search">
  <form id ="rtd-search-form" class="wy-form" action="../search.html" method="get">
      <input type="text" name="q" placeholder="Search docs" title="Type search term here" />
  </form>
</div>
      </div>

      <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
                <ul>
                    <li class="toctree-l1"><a class="" href="../index.md">Index</a>
                    </li>
                </ul>
      </div>
    </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" role="navigation" aria-label="top navigation">
        <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
        <a href="..">MIxS</a>
      </nav>

      
      <div class="wy-nav-content">
        <div class="rst-content">
          <div role="navigation" aria-label="breadcrumbs navigation">
  <ul class="wy-breadcrumbs">
    <li><a href="..">Docs</a> &raquo;</li>
    
      
    
    <li>Class: plant-associated MIGS bacteria</li>
    <li class="wy-breadcrumbs-aside">
      
    </li>
  </ul>
  
  <hr/>
</div>

          <div role="main">
            <div class="section">
              
                <h1 id="class-plant-associated-migs-bacteria">Class: plant-associated MIGS bacteria</h1>
<p>Combinatorial checklist Minimal Information about a Genome Sequence: cultured bacteria/archaea with environmental package plant-associated</p>
<p>URI: <a href="https://w3id.org/mixs/vocab/Plant-associatedMIGSBacteria">mixs.vocab:Plant-associatedMIGSBacteria</a></p>
<p><a href="https://yuml.me/diagram/nofunky;dir:TB/class/[QuantityValue],[Plant-associatedMIGSBacteria&#124;samp_taxon_id:string;experimental_factor:string%20%3F;neg_cont_type:neg_cont_type_enum%20%3F;pos_cont_type:string%20%3F;subspecf_gen_lin:string%20%3F;num_replicons:integer;extrachrom_elements:integer%20%3F;estimated_size:string%20%3F;ref_biomaterial:string;source_mat_id:string%20%3F;pathogenicity:string%20%3F;biotic_relationship:biotic_relationship_enum%20%3F;specific_host:string%20%3F;host_spec_range:integer%20%3F;trophic_level:trophic_level_enum%20%3F;encoded_traits:string%20%3F;rel_to_oxygen:rel_to_oxygen_enum%20%3F;isol_growth_condt:string;samp_collec_device:string%20%3F;samp_collec_method:string%20%3F;samp_mat_process:string%20%3F;nucl_acid_ext:string%20%3F;nucl_acid_amp:string%20%3F;lib_size:integer%20%3F;lib_reads_seqd:integer%20%3F;lib_layout:lib_layout_enum%20%3F;lib_vector:string%20%3F;lib_screen:string%20%3F;adapters:string%20%3F;seq_meth:string;tax_ident:tax_ident_enum%20%3F;assembly_qual:assembly_qual_enum;assembly_name:string%20%3F;assembly_software:string;annot:string%20%3F;number_contig:integer;feat_pred:string%20%3F;ref_db:string%20%3F;sim_search_meth:string%20%3F;tax_class:string%20%3F;compl_score:compl_score_enum%20%3F;compl_software:string%20%3F;associated_resource:string%20%3F;sop:string%20%3F;lat_lon(i):string%20%3F;geo_loc_name(i):string%20%3F;collection_date(i):date%20%3F;env_broad_scale(i):string%20%3F;env_local_scale(i):string%20%3F;env_medium(i):string%20%3F;air_temp_regm(i):string%20*;ances_data(i):string%20%3F;antibiotic_regm(i):string%20*;biol_stat(i):biol_stat_enum%20%3F;biotic_regm(i):string%20%3F;chem_administration(i):string%20*;chem_mutagen(i):string%20*;climate_environment(i):string%20*;cult_root_med(i):string%20%3F;fertilizer_regm(i):string%20*;fungicide_regm(i):string%20*;gaseous_environment(i):string%20*;genetic_mod(i):string%20%3F;gravity(i):string%20*;growth_facil(i):string%20%3F;growth_habit(i):growth_habit_enum%20%3F;growth_hormone_regm(i):string%20*;herbicide_regm(i):string%20*;host_common_name(i):string%20%3F;host_genotype(i):string%20%3F;host_subspecf_genlin(i):string%20*;host_life_stage(i):string%20%3F;host_phenotype(i):string%20%3F;host_taxid(i):string%20%3F;humidity_regm(i):string%20*;light_regm(i):string%20%3F;mechanical_damage(i):string%20*;mineral_nutr_regm(i):string%20*;misc_param(i):string%20*;non_min_nutr_regm(i):string%20*;organism_count(i):organism_count_enum%20*;oxy_stat_samp(i):oxy_stat_samp_enum%20%3F;ph_regm(i):string%20*;perturbation(i):string%20*;pesticide_regm(i):string%20*;plant_growth_med(i):plant_growth_med_enum%20%3F;plant_product(i):string%20%3F;plant_sex(i):plant_sex_enum%20%3F;plant_struc(i):string%20%3F;radiation_regm(i):string%20*;rainfall_regm(i):string%20*;root_cond(i):string%20%3F;root_med_carbon(i):string%20%3F;root_med_macronutr(i):string%20%3F;root_med_micronutr(i):string%20%3F;root_med_suppl(i):string%20%3F;root_med_ph(i):double%20%3F;root_med_regl(i):string%20%3F;root_med_solid(i):string%20%3F;salt_regm(i):string%20*;samp_capt_status(i):samp_capt_status_enum%20%3F;samp_dis_stage(i):samp_dis_stage_enum%20%3F;samp_store_dur(i):string%20%3F;samp_store_loc(i):string%20%3F;season_environment(i):string%20*;standing_water_regm(i):string%20*;tiss_cult_growth_med(i):string%20%3F;water_temp_regm(i):string%20*;watering_regm(i):string%20*;host_symbiont(i):string%20*;samp_name(i):string;project_name(i):string;host_disease_stat(i):string%20%3F]uses%20-.-&gt;[MIGSBacteria],[Plant-associated]^-[Plant-associatedMIGSBacteria],[Plant-associated],[MIGSBacteria]"><img alt="img" src="https://yuml.me/diagram/nofunky;dir:TB/class/[QuantityValue],[Plant-associatedMIGSBacteria&#124;samp_taxon_id:string;experimental_factor:string%20%3F;neg_cont_type:neg_cont_type_enum%20%3F;pos_cont_type:string%20%3F;subspecf_gen_lin:string%20%3F;num_replicons:integer;extrachrom_elements:integer%20%3F;estimated_size:string%20%3F;ref_biomaterial:string;source_mat_id:string%20%3F;pathogenicity:string%20%3F;biotic_relationship:biotic_relationship_enum%20%3F;specific_host:string%20%3F;host_spec_range:integer%20%3F;trophic_level:trophic_level_enum%20%3F;encoded_traits:string%20%3F;rel_to_oxygen:rel_to_oxygen_enum%20%3F;isol_growth_condt:string;samp_collec_device:string%20%3F;samp_collec_method:string%20%3F;samp_mat_process:string%20%3F;nucl_acid_ext:string%20%3F;nucl_acid_amp:string%20%3F;lib_size:integer%20%3F;lib_reads_seqd:integer%20%3F;lib_layout:lib_layout_enum%20%3F;lib_vector:string%20%3F;lib_screen:string%20%3F;adapters:string%20%3F;seq_meth:string;tax_ident:tax_ident_enum%20%3F;assembly_qual:assembly_qual_enum;assembly_name:string%20%3F;assembly_software:string;annot:string%20%3F;number_contig:integer;feat_pred:string%20%3F;ref_db:string%20%3F;sim_search_meth:string%20%3F;tax_class:string%20%3F;compl_score:compl_score_enum%20%3F;compl_software:string%20%3F;associated_resource:string%20%3F;sop:string%20%3F;lat_lon(i):string%20%3F;geo_loc_name(i):string%20%3F;collection_date(i):date%20%3F;env_broad_scale(i):string%20%3F;env_local_scale(i):string%20%3F;env_medium(i):string%20%3F;air_temp_regm(i):string%20*;ances_data(i):string%20%3F;antibiotic_regm(i):string%20*;biol_stat(i):biol_stat_enum%20%3F;biotic_regm(i):string%20%3F;chem_administration(i):string%20*;chem_mutagen(i):string%20*;climate_environment(i):string%20*;cult_root_med(i):string%20%3F;fertilizer_regm(i):string%20*;fungicide_regm(i):string%20*;gaseous_environment(i):string%20*;genetic_mod(i):string%20%3F;gravity(i):string%20*;growth_facil(i):string%20%3F;growth_habit(i):growth_habit_enum%20%3F;growth_hormone_regm(i):string%20*;herbicide_regm(i):string%20*;host_common_name(i):string%20%3F;host_genotype(i):string%20%3F;host_subspecf_genlin(i):string%20*;host_life_stage(i):string%20%3F;host_phenotype(i):string%20%3F;host_taxid(i):string%20%3F;humidity_regm(i):string%20*;light_regm(i):string%20%3F;mechanical_damage(i):string%20*;mineral_nutr_regm(i):string%20*;misc_param(i):string%20*;non_min_nutr_regm(i):string%20*;organism_count(i):organism_count_enum%20*;oxy_stat_samp(i):oxy_stat_samp_enum%20%3F;ph_regm(i):string%20*;perturbation(i):string%20*;pesticide_regm(i):string%20*;plant_growth_med(i):plant_growth_med_enum%20%3F;plant_product(i):string%20%3F;plant_sex(i):plant_sex_enum%20%3F;plant_struc(i):string%20%3F;radiation_regm(i):string%20*;rainfall_regm(i):string%20*;root_cond(i):string%20%3F;root_med_carbon(i):string%20%3F;root_med_macronutr(i):string%20%3F;root_med_micronutr(i):string%20%3F;root_med_suppl(i):string%20%3F;root_med_ph(i):double%20%3F;root_med_regl(i):string%20%3F;root_med_solid(i):string%20%3F;salt_regm(i):string%20*;samp_capt_status(i):samp_capt_status_enum%20%3F;samp_dis_stage(i):samp_dis_stage_enum%20%3F;samp_store_dur(i):string%20%3F;samp_store_loc(i):string%20%3F;season_environment(i):string%20*;standing_water_regm(i):string%20*;tiss_cult_growth_med(i):string%20%3F;water_temp_regm(i):string%20*;watering_regm(i):string%20*;host_symbiont(i):string%20*;samp_name(i):string;project_name(i):string;host_disease_stat(i):string%20%3F]uses%20-.-&gt;[MIGSBacteria],[Plant-associated]^-[Plant-associatedMIGSBacteria],[Plant-associated],[MIGSBacteria]" /></a></p>
<h2 id="parents">Parents</h2>
<ul>
<li>is_a: <a href="../Plant-associated/">Plant-associated</a> - plant-associated</li>
</ul>
<h2 id="uses-mixin">Uses Mixin</h2>
<ul>
<li>mixin: <a href="../MIGSBacteria/">MIGSBacteria</a> - Minimal Information about a Genome Sequence: cultured bacteria/archaea</li>
</ul>
<h2 id="attributes">Attributes</h2>
<h3 id="inherited-from-plant-associated">Inherited from plant-associated:</h3>
<ul>
<li><a href="../lat_lon/">lat_lon</a>  <sub>0..1</sub><ul>
<li>Description: The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 50.586825 6.408977 None</li>
</ul>
</li>
<li><a href="../plant_associated_depth/">plant-associated➞depth</a>  <sub>0..1</sub><ul>
<li>Description: The vertical distance below local surface, e.g. for sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 10 meter None</li>
</ul>
</li>
<li><a href="../alt/">alt</a>  <sub>0..1</sub><ul>
<li>Description: Altitude is a term used to identify heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth's surface. In this context, the altitude measurement is the vertical distance between the earth's surface above sea level and the sampled position in the air</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 100 meter None</li>
</ul>
</li>
<li><a href="../plant_associated_elev/">plant-associated➞elev</a>  <sub>0..1</sub><ul>
<li>Description: Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth's surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 100 meter None</li>
</ul>
</li>
<li><a href="../plant_associated_temp/">plant-associated➞temp</a>  <sub>0..1</sub><ul>
<li>Description: Temperature of the sample at the time of sampling.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 25 degree Celsius None</li>
</ul>
</li>
<li><a href="../geo_loc_name/">geo_loc_name</a>  <sub>0..1</sub><ul>
<li>Description: The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: USA: Maryland, Bethesda None</li>
</ul>
</li>
<li><a href="../collection_date/">collection_date</a>  <sub>0..1</sub><ul>
<li>Description: The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant</li>
<li>Range: <a href="../types/Date/">Date</a></li>
<li>Example: 2018-05-11T10:00:00+01:00; 2018-05-11 None</li>
</ul>
</li>
<li><a href="../env_broad_scale/">env_broad_scale</a>  <sub>0..1</sub><ul>
<li>Description: Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO’s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: oceanic epipelagic zone biome [ENVO:01000033] for annotating a water sample from the photic zone in middle of the Atlantic Ocean None</li>
</ul>
</li>
<li><a href="../env_local_scale/">env_local_scale</a>  <sub>0..1</sub><ul>
<li>Description: Report the entity or entities which are in the sample or specimen’s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: litter layer [ENVO:01000338]; Annotating a pooled sample taken from various vegetation layers in a forest consider: canopy [ENVO:00000047]|herb and fern layer [ENVO:01000337]|litter layer [ENVO:01000338]|understory [01000335]|shrub layer [ENVO:01000336]. None</li>
</ul>
</li>
<li><a href="../env_medium/">env_medium</a>  <sub>0..1</sub><ul>
<li>Description: Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of 'environmental material' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: soil [ENVO:00001998]; Annotating a fish swimming in the upper 100 m of the Atlantic Ocean, consider: ocean water [ENVO:00002151]. Example: Annotating a duck on a pond consider: pond water [ENVO:00002228]|air [ENVO_00002005] None</li>
</ul>
</li>
<li><a href="../plant_associated_air_temp_regm/">plant-associated➞air_temp_regm</a>  <sub>0..*</sub><ul>
<li>Description: Information about treatment involving an exposure to varying temperatures; should include the temperature, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include different temperature regimens</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 25 degree Celsius;R2/2018-05-11T14:30/2018-05-11T19:30/P1H30M None</li>
</ul>
</li>
<li><a href="../plant_associated_ances_data/">plant-associated➞ances_data</a>  <sub>0..1</sub><ul>
<li>Description: Information about either pedigree or other ancestral information description (e.g. parental variety in case of mutant or selection), e.g. A/3*B (meaning [(A x B) x B] x B)</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: A/3*B None</li>
</ul>
</li>
<li><a href="../plant_associated_antibiotic_regm/">plant-associated➞antibiotic_regm</a>  <sub>0..*</sub><ul>
<li>Description: Information about treatment involving antibiotic administration; should include the name of antibiotic, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple antibiotic regimens</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: penicillin;5 milligram;R2/2018-05-11T14:30/2018-05-11T19:30/P1H30M None</li>
</ul>
</li>
<li><a href="../plant_associated_biol_stat/">plant-associated➞biol_stat</a>  <sub>0..1</sub><ul>
<li>Description: The level of genome modification.</li>
<li>Range: <a href="../biol_stat_enum/">biol_stat_enum</a></li>
<li>Example: natural None</li>
</ul>
</li>
<li><a href="../plant_associated_biotic_regm/">plant-associated➞biotic_regm</a>  <sub>0..1</sub><ul>
<li>Description: Information about treatment(s) involving use of biotic factors, such as bacteria, viruses or fungi.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: sample inoculated with Rhizobium spp. Culture None</li>
</ul>
</li>
<li><a href="../plant_associated_chem_administration/">plant-associated➞chem_administration</a>  <sub>0..*</sub><ul>
<li>Description: List of chemical compounds administered to the host or site where sampling occurred, and when (e.g. Antibiotics, n fertilizer, air filter); can include multiple compounds. For chemical entities of biological interest ontology (chebi) (v 163), http://purl.bioontology.org/ontology/chebi</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: agar [CHEBI:2509];2018-05-11T20:00Z None</li>
</ul>
</li>
<li><a href="../plant_associated_chem_mutagen/">plant-associated➞chem_mutagen</a>  <sub>0..*</sub><ul>
<li>Description: Treatment involving use of mutagens; should include the name of mutagen, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple mutagen regimens</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: nitrous acid;0.5 milligram per liter;R2/2018-05-11T14:30/2018-05-11T19:30/P1H30M None</li>
</ul>
</li>
<li><a href="../plant_associated_climate_environment/">plant-associated➞climate_environment</a>  <sub>0..*</sub><ul>
<li>Description: Treatment involving an exposure to a particular climate; treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple climates</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: tropical climate;R2/2018-05-11T14:30/2018-05-11T19:30/P1H30M None</li>
</ul>
</li>
<li><a href="../plant_associated_cult_root_med/">plant-associated➞cult_root_med</a>  <sub>0..1</sub><ul>
<li>Description: Name or reference for the hydroponic or in vitro culture rooting medium; can be the name of a commonly used medium or reference to a specific medium, e.g. Murashige and Skoog medium. If the medium has not been formally published, use the rooting medium descriptors.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: http://himedialabs.com/TD/PT158.pdf None</li>
</ul>
</li>
<li><a href="../plant_associated_fertilizer_regm/">plant-associated➞fertilizer_regm</a>  <sub>0..*</sub><ul>
<li>Description: Information about treatment involving the use of fertilizers; should include the name of fertilizer, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple fertilizer regimens</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: urea;0.6 milligram per liter;R2/2018-05-11:T14:30/2018-05-11T19:30/P1H30M None</li>
</ul>
</li>
<li><a href="../plant_associated_fungicide_regm/">plant-associated➞fungicide_regm</a>  <sub>0..*</sub><ul>
<li>Description: Information about treatment involving use of fungicides; should include the name of fungicide, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple fungicide regimens</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: bifonazole;1 mole per liter;R2/2018-05-11T14:30/2018-05-11T19:30/P1H30M None</li>
</ul>
</li>
<li><a href="../plant_associated_gaseous_environment/">plant-associated➞gaseous_environment</a>  <sub>0..*</sub><ul>
<li>Description: Use of conditions with differing gaseous environments; should include the name of gaseous compound, amount administered, treatment duration, interval and total experimental duration; can include multiple gaseous environment regimens</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: nitric oxide;0.5 micromole per liter;R2/2018-05-11T14:30/2018-05-11T19:30/P1H30M None</li>
</ul>
</li>
<li><a href="../plant_associated_genetic_mod/">plant-associated➞genetic_mod</a>  <sub>0..1</sub><ul>
<li>Description: Genetic modifications of the genome of an organism, which may occur naturally by spontaneous mutation, or be introduced by some experimental means, e.g. specification of a transgene or the gene knocked-out or details of transient transfection</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: aox1A transgenic None</li>
</ul>
</li>
<li><a href="../plant_associated_gravity/">plant-associated➞gravity</a>  <sub>0..*</sub><ul>
<li>Description: Information about treatment involving use of gravity factor to study various types of responses in presence, absence or modified levels of gravity; treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple treatments</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 12 g;R2/2018-05-11T14:30/2018-05-11T19:30/P1H30M None</li>
</ul>
</li>
<li><a href="../plant_associated_growth_facil/">plant-associated➞growth_facil</a>  <sub>0..1</sub><ul>
<li>Description: Type of facility where the sampled plant was grown; controlled vocabulary: growth chamber, open top chamber, glasshouse, experimental garden, field. Alternatively use Crop Ontology (CO) terms, see http://www.cropontology.org/ontology/CO_715/Crop%20Research</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Growth chamber [CO_715:0000189] None</li>
</ul>
</li>
<li><a href="../plant_associated_growth_habit/">plant-associated➞growth_habit</a>  <sub>0..1</sub><ul>
<li>Description: Characteristic shape, appearance or growth form of a plant species</li>
<li>Range: <a href="../growth_habit_enum/">growth_habit_enum</a></li>
<li>Example: spreading None</li>
</ul>
</li>
<li><a href="../plant_associated_growth_hormone_regm/">plant-associated➞growth_hormone_regm</a>  <sub>0..*</sub><ul>
<li>Description: Information about treatment involving use of growth hormones; should include the name of growth hormone, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple growth hormone regimens</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: abscisic acid;0.5 milligram per liter;R2/2018-05-11T14:30/2018-05-11T19:30/P1H30M None</li>
</ul>
</li>
<li><a href="../plant_associated_herbicide_regm/">plant-associated➞herbicide_regm</a>  <sub>0..*</sub><ul>
<li>Description: Information about treatment involving use of herbicides; information about treatment involving use of growth hormones; should include the name of herbicide, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple regimens</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: atrazine;10 milligram per liter;R2/2018-05-11T14:30/2018-05-11T19:30/P1H30M None</li>
</ul>
</li>
<li><a href="../plant_associated_host_age/">plant-associated➞host_age</a>  <sub>0..1</sub><ul>
<li>Description: Age of host at the time of sampling; relevant scale depends on species and study, e.g. Could be seconds for amoebae or centuries for trees</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 10 days None</li>
</ul>
</li>
<li><a href="../plant_associated_host_common_name/">plant-associated➞host_common_name</a>  <sub>0..1</sub><ul>
<li>Description: Common name of the host.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: human None</li>
</ul>
</li>
<li><a href="../plant_associated_host_dry_mass/">plant-associated➞host_dry_mass</a>  <sub>0..1</sub><ul>
<li>Description: Measurement of dry mass</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 500 gram None</li>
</ul>
</li>
<li><a href="../plant_associated_host_genotype/">plant-associated➞host_genotype</a>  <sub>0..1</sub><ul>
<li>Description: Observed genotype</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: C57BL/6 None</li>
</ul>
</li>
<li><a href="../plant_associated_host_height/">plant-associated➞host_height</a>  <sub>0..1</sub><ul>
<li>Description: The height of subject</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 0.1 meter None</li>
</ul>
</li>
<li><a href="../plant_associated_host_subspecf_genlin/">plant-associated➞host_subspecf_genlin</a>  <sub>0..*</sub><ul>
<li>Description: Information about the genetic distinctness of the host organism below the subspecies level e.g., serovar, serotype, biotype, ecotype, variety, cultivar, or any relevant genetic typing schemes like Group I plasmid. Subspecies should not be recorded in this term, but in the NCBI taxonomy. Supply both the lineage name and the lineage rank separated by a colon, e.g., biovar:abc123.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: serovar:Newport, variety:glabrum, cultivar: Red Delicious None</li>
</ul>
</li>
<li><a href="../plant_associated_host_length/">plant-associated➞host_length</a>  <sub>0..1</sub><ul>
<li>Description: The length of subject</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 1 meter None</li>
</ul>
</li>
<li><a href="../plant_associated_host_life_stage/">plant-associated➞host_life_stage</a>  <sub>0..1</sub><ul>
<li>Description: Description of life stage of host</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: adult None</li>
</ul>
</li>
<li><a href="../plant_associated_host_phenotype/">plant-associated➞host_phenotype</a>  <sub>0..1</sub><ul>
<li>Description: Phenotype of human or other host. For phenotypic quality ontology (pato) (v 2018-03-27) terms, please see http://purl.bioontology.org/ontology/pato. For Human Phenotype Ontology (HP) (v 2018-06-13) please see http://purl.bioontology.org/ontology/HP</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: elongated [PATO:0001154] None</li>
</ul>
</li>
<li><a href="../plant_associated_host_taxid/">plant-associated➞host_taxid</a>  <sub>0..1</sub><ul>
<li>Description: NCBI taxon id of the host, e.g. 9606</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 7955 None</li>
</ul>
</li>
<li><a href="../plant_associated_host_tot_mass/">plant-associated➞host_tot_mass</a>  <sub>0..1</sub><ul>
<li>Description: Total mass of the host at collection, the unit depends on host</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 2500 gram None</li>
</ul>
</li>
<li><a href="../plant_associated_host_wet_mass/">plant-associated➞host_wet_mass</a>  <sub>0..1</sub><ul>
<li>Description: Measurement of wet mass</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 1500 gram None</li>
</ul>
</li>
<li><a href="../plant_associated_humidity_regm/">plant-associated➞humidity_regm</a>  <sub>0..*</sub><ul>
<li>Description: Information about treatment involving an exposure to varying degree of humidity; information about treatment involving use of growth hormones; should include amount of humidity administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple regimens</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 25 gram per cubic meter;R2/2018-05-11T14:30/2018-05-11T19:30/P1H30M None</li>
</ul>
</li>
<li><a href="../plant_associated_light_regm/">plant-associated➞light_regm</a>  <sub>0..1</sub><ul>
<li>Description: Information about treatment(s) involving exposure to light, including both light intensity and quality.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: incandescant light;10 lux;450 nanometer None</li>
</ul>
</li>
<li><a href="../plant_associated_mechanical_damage/">plant-associated➞mechanical_damage</a>  <sub>0..*</sub><ul>
<li>Description: Information about any mechanical damage exerted on the plant; can include multiple damages and sites</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: pruning;bark None</li>
</ul>
</li>
<li><a href="../plant_associated_mineral_nutr_regm/">plant-associated➞mineral_nutr_regm</a>  <sub>0..*</sub><ul>
<li>Description: Information about treatment involving the use of mineral supplements; should include the name of mineral nutrient, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple mineral nutrient regimens</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: potassium;15 gram;R2/2018-05-11T14:30/2018-05-11T19:30/P1H30M None</li>
</ul>
</li>
<li><a href="../plant_associated_non_min_nutr_regm/">plant-associated➞non_min_nutr_regm</a>  <sub>0..*</sub><ul>
<li>Description: Information about treatment involving the exposure of plant to non-mineral nutrient such as oxygen, hydrogen or carbon; should include the name of non-mineral nutrient, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple non-mineral nutrient regimens</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: carbon dioxide;10 mole per liter;R2/2018-05-11T14:30/2018-05-11T19:30/P1H30M None</li>
</ul>
</li>
<li><a href="../plant_associated_organism_count/">plant-associated➞organism_count</a>  <sub>0..*</sub><ul>
<li>Description: Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)</li>
<li>Range: <a href="../organism_count_enum/">organism_count_enum</a></li>
<li>Example: total prokaryotes;3.5e7 cells per milliliter;qPCR None</li>
</ul>
</li>
<li><a href="../plant_associated_oxy_stat_samp/">plant-associated➞oxy_stat_samp</a>  <sub>0..1</sub><ul>
<li>Description: Oxygenation status of sample</li>
<li>Range: <a href="../oxy_stat_samp_enum/">oxy_stat_samp_enum</a></li>
<li>Example: aerobic None</li>
</ul>
</li>
<li><a href="../plant_associated_ph_regm/">plant-associated➞ph_regm</a>  <sub>0..*</sub><ul>
<li>Description: Information about treatment involving exposure of plants to varying levels of ph of the growth media, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple regimen</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 7.6;R2/2018-05-11:T14:30/2018-05-11T19:30/P1H30M None</li>
</ul>
</li>
<li><a href="../plant_associated_perturbation/">plant-associated➞perturbation</a>  <sub>0..*</sub><ul>
<li>Description: Type of perturbation, e.g. chemical administration, physical disturbance, etc., coupled with perturbation regimen including how many times the perturbation was repeated, how long each perturbation lasted, and the start and end time of the entire perturbation period; can include multiple perturbation types</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: antibiotic addition;R2/2018-05-11T14:30Z/2018-05-11T19:30Z/P1H30M None</li>
</ul>
</li>
<li><a href="../plant_associated_pesticide_regm/">plant-associated➞pesticide_regm</a>  <sub>0..*</sub><ul>
<li>Description: Information about treatment involving use of insecticides; should include the name of pesticide, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple pesticide regimens</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: pyrethrum;0.6 milligram per liter;R2/2018-05-11T14:30/2018-05-11T19:30/P1H30M None</li>
</ul>
</li>
<li><a href="../plant_associated_plant_growth_med/">plant-associated➞plant_growth_med</a>  <sub>0..1</sub><ul>
<li>Description: Specification of the media for growing the plants or tissue cultured samples, e.g. soil, aeroponic, hydroponic, in vitro solid culture medium, in vitro liquid culture medium. Recommended value is a specific value from EO:plant growth medium (follow this link for terms http://purl.obolibrary.org/obo/EO_0007147) or other controlled vocabulary</li>
<li>Range: <a href="../plant_growth_med_enum/">plant_growth_med_enum</a></li>
<li>Example: hydroponic plant culture media [EO:0007067] None</li>
</ul>
</li>
<li><a href="../plant_associated_plant_product/">plant-associated➞plant_product</a>  <sub>0..1</sub><ul>
<li>Description: Substance produced by the plant, where the sample was obtained from</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: xylem sap [PO:0025539] None</li>
</ul>
</li>
<li><a href="../plant_associated_plant_sex/">plant-associated➞plant_sex</a>  <sub>0..1</sub><ul>
<li>Description: Sex of the reproductive parts on the whole plant, e.g. pistillate, staminate, monoecieous, hermaphrodite.</li>
<li>Range: <a href="../plant_sex_enum/">plant_sex_enum</a></li>
<li>Example: Hermaphroditic None</li>
</ul>
</li>
<li><a href="../plant_associated_plant_struc/">plant-associated➞plant_struc</a>  <sub>0..1</sub><ul>
<li>Description: Name of plant structure the sample was obtained from; for Plant Ontology (PO) (v releases/2017-12-14) terms, see http://purl.bioontology.org/ontology/PO, e.g. petiole epidermis (PO_0000051). If an individual flower is sampled, the sex of it can be recorded here.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: epidermis [PO:0005679] None</li>
</ul>
</li>
<li><a href="../plant_associated_radiation_regm/">plant-associated➞radiation_regm</a>  <sub>0..*</sub><ul>
<li>Description: Information about treatment involving exposure of plant or a plant part to a particular radiation regimen; should include the radiation type, amount or intensity administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple radiation regimens</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: gamma radiation;60 gray;R2/2018-05-11T14:30/2018-05-11T19:30/P1H30M None</li>
</ul>
</li>
<li><a href="../plant_associated_rainfall_regm/">plant-associated➞rainfall_regm</a>  <sub>0..*</sub><ul>
<li>Description: Information about treatment involving an exposure to a given amount of rainfall, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple regimens</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 15 millimeter;R2/2018-05-11T14:30/2018-05-11T19:30/P1H30M None</li>
</ul>
</li>
<li><a href="../plant_associated_root_cond/">plant-associated➞root_cond</a>  <sub>0..1</sub><ul>
<li>Description: Relevant rooting conditions such as field plot size, sowing density, container dimensions, number of plants per container.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: http://himedialabs.com/TD/PT158.pdf None</li>
</ul>
</li>
<li><a href="../plant_associated_root_med_carbon/">plant-associated➞root_med_carbon</a>  <sub>0..1</sub><ul>
<li>Description: Source of organic carbon in the culture rooting medium; e.g. sucrose.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: sucrose None</li>
</ul>
</li>
<li><a href="../plant_associated_root_med_macronutr/">plant-associated➞root_med_macronutr</a>  <sub>0..1</sub><ul>
<li>Description: Measurement of the culture rooting medium macronutrients (N,P, K, Ca, Mg, S); e.g. KH2PO4 (170¬†mg/L).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: KH2PO4;170¬†milligram per liter None</li>
</ul>
</li>
<li><a href="../plant_associated_root_med_micronutr/">plant-associated➞root_med_micronutr</a>  <sub>0..1</sub><ul>
<li>Description: Measurement of the culture rooting medium micronutrients (Fe, Mn, Zn, B, Cu, Mo); e.g. H3BO3 (6.2¬†mg/L).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: H3BO3;6.2¬†milligram per liter None</li>
</ul>
</li>
<li><a href="../plant_associated_root_med_suppl/">plant-associated➞root_med_suppl</a>  <sub>0..1</sub><ul>
<li>Description: Organic supplements of the culture rooting medium, such as vitamins, amino acids, organic acids, antibiotics activated charcoal; e.g. nicotinic acid (0.5¬†mg/L).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: nicotinic acid;0.5 milligram per liter None</li>
</ul>
</li>
<li><a href="../plant_associated_root_med_ph/">plant-associated➞root_med_ph</a>  <sub>0..1</sub><ul>
<li>Description: pH measurement of the culture rooting medium; e.g. 5.5.</li>
<li>Range: <a href="../types/Double/">Double</a></li>
<li>Example: 7.5 None</li>
</ul>
</li>
<li><a href="../plant_associated_root_med_regl/">plant-associated➞root_med_regl</a>  <sub>0..1</sub><ul>
<li>Description: Growth regulators in the culture rooting medium such as cytokinins, auxins, gybberellins, abscisic acid; e.g. 0.5¬†mg/L NAA.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: abscisic acid;0.75 milligram per liter None</li>
</ul>
</li>
<li><a href="../plant_associated_root_med_solid/">plant-associated➞root_med_solid</a>  <sub>0..1</sub><ul>
<li>Description: Specification of the solidifying agent in the culture rooting medium; e.g. agar.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: agar None</li>
</ul>
</li>
<li><a href="../plant_associated_salt_regm/">plant-associated➞salt_regm</a>  <sub>0..*</sub><ul>
<li>Description: Information about treatment involving use of salts as supplement to liquid and soil growth media; should include the name of salt, amount administered, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple salt regimens</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: NaCl;5 gram per liter;R2/2018-05-11T14:30/2018-05-11T19:30/P1H30M None</li>
</ul>
</li>
<li><a href="../plant_associated_samp_capt_status/">plant-associated➞samp_capt_status</a>  <sub>0..1</sub><ul>
<li>Description: Reason for the sample</li>
<li>Range: <a href="../samp_capt_status_enum/">samp_capt_status_enum</a></li>
<li>Example: farm sample None</li>
</ul>
</li>
<li><a href="../plant_associated_samp_dis_stage/">plant-associated➞samp_dis_stage</a>  <sub>0..1</sub><ul>
<li>Description: Stage of the disease at the time of sample collection, e.g. inoculation, penetration, infection, growth and reproduction, dissemination of pathogen.</li>
<li>Range: <a href="../samp_dis_stage_enum/">samp_dis_stage_enum</a></li>
<li>Example: infection None</li>
</ul>
</li>
<li><a href="../plant_associated_salinity/">plant-associated➞salinity</a>  <sub>0..1</sub><ul>
<li>Description: The total concentration of all dissolved salts in a liquid or solid sample. While salinity can be measured by a complete chemical analysis, this method is difficult and time consuming. More often, it is instead derived from the conductivity measurement. This is known as practical salinity. These derivations compare the specific conductance of the sample to a salinity standard such as seawater.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 25 practical salinity unit None</li>
</ul>
</li>
<li><a href="../plant_associated_samp_store_dur/">plant-associated➞samp_store_dur</a>  <sub>0..1</sub><ul>
<li>Description: Duration for which the sample was stored</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: P1Y6M None</li>
</ul>
</li>
<li><a href="../plant_associated_samp_store_loc/">plant-associated➞samp_store_loc</a>  <sub>0..1</sub><ul>
<li>Description: Location at which sample was stored, usually name of a specific freezer/room</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Freezer no:5 None</li>
</ul>
</li>
<li><a href="../plant_associated_samp_store_temp/">plant-associated➞samp_store_temp</a>  <sub>0..1</sub><ul>
<li>Description: Temperature at which sample was stored, e.g. -80 degree Celsius</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: -80 degree Celsius None</li>
</ul>
</li>
<li><a href="../plant_associated_season_environment/">plant-associated➞season_environment</a>  <sub>0..*</sub><ul>
<li>Description: Treatment involving an exposure to a particular season (e.g. Winter, summer, rabi, rainy etc.), treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: rainy;R2/2018-05-11T14:30/2018-05-11T19:30/P1H30M None</li>
</ul>
</li>
<li><a href="../plant_associated_standing_water_regm/">plant-associated➞standing_water_regm</a>  <sub>0..*</sub><ul>
<li>Description: Treatment involving an exposure to standing water during a plant's life span, types can be flood water or standing water, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple regimens</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: standing water;R2/2018-05-11T14:30/2018-05-11T19:30/P1H30M None</li>
</ul>
</li>
<li><a href="../plant_associated_tiss_cult_growth_med/">plant-associated➞tiss_cult_growth_med</a>  <sub>0..1</sub><ul>
<li>Description: Description of plant tissue culture growth media used</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: https://link.springer.com/content/pdf/10.1007/BF02796489.pdf None</li>
</ul>
</li>
<li><a href="../plant_associated_water_temp_regm/">plant-associated➞water_temp_regm</a>  <sub>0..*</sub><ul>
<li>Description: Information about treatment involving an exposure to water with varying degree of temperature, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple regimens</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 15 degree Celsius;R2/2018-05-11T14:30/2018-05-11T19:30/P1H30M None</li>
</ul>
</li>
<li><a href="../plant_associated_watering_regm/">plant-associated➞watering_regm</a>  <sub>0..*</sub><ul>
<li>Description: Information about treatment involving an exposure to watering frequencies, treatment regimen including how many times the treatment was repeated, how long each treatment lasted, and the start and end time of the entire treatment; can include multiple regimens</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 1 liter;R2/2018-05-11T14:30/2018-05-11T19:30/P1H30M None</li>
</ul>
</li>
<li><a href="../plant_associated_host_symbiont/">plant-associated➞host_symbiont</a>  <sub>0..*</sub><ul>
<li>Description: The taxonomic name of the organism(s) found living in mutualistic, commensalistic, or parasitic symbiosis with the specific host.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: flukeworms None</li>
</ul>
</li>
<li><a href="../misc_param/">misc_param</a>  <sub>0..*</sub><ul>
<li>Description: Any other measurement performed or parameter collected, that is not listed here</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Bicarbonate ion concentration;2075 micromole per kilogram None</li>
</ul>
</li>
<li><a href="../plant_associated_samp_name/">plant-associated➞samp_name</a>  <sub>1..1</sub><ul>
<li>Description: A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: ISDsoil1 None</li>
</ul>
</li>
<li><a href="../plant_associated_project_name/">plant-associated➞project_name</a>  <sub>1..1</sub><ul>
<li>Description: Name of the project within which the sequencing was organized</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Forest soil metagenome None</li>
</ul>
</li>
<li><a href="../plant_associated_host_disease_stat/">plant-associated➞host_disease_stat</a>  <sub>0..1</sub><ul>
<li>Description: List of diseases with which the host has been diagnosed; can include multiple diagnoses. The value of the field depends on host; for humans the terms should be chosen from the DO (Human Disease Ontology) at https://www.disease-ontology.org, non-human host diseases are free text</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: rabies [DOID:11260] None</li>
</ul>
</li>
<li><a href="../plant_associated_samp_vol_we_dna_ext/">plant-associated➞samp_vol_we_dna_ext</a>  <sub>0..1</sub><ul>
<li>Description: Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001).</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 1500 milliliter None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_samp_taxon_id/">MIGS bacteria➞samp_taxon_id</a>  <sub>1..1</sub><ul>
<li>Description: NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa sample. Use 'synthetic metagenome’ for mock community/positive controls, or 'blank sample' for negative controls.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Gut Metagenome [NCBI:txid749906] None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_1">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_experimental_factor/">MIGS bacteria➞experimental_factor</a>  <sub>0..1</sub><ul>
<li>Description: Experimental factors are essentially the variable aspects of an experiment design which can be used to describe an experiment, or set of experiments, in an increasingly detailed manner. This field accepts ontology terms from Experimental Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI). For a browser of EFO (v 2.95) terms, please see http://purl.bioontology.org/ontology/EFO; for a browser of OBI (v 2018-02-12) terms please see http://purl.bioontology.org/ontology/OBI</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: time series design [EFO:EFO_0001779] None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_2">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_neg_cont_type/">MIGS bacteria➞neg_cont_type</a>  <sub>0..1</sub><ul>
<li>Description: The substance or equipment used as a negative control in an investigation</li>
<li>Range: <a href="../neg_cont_type_enum/">neg_cont_type_enum</a></li>
<li>Example:  None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_3">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_pos_cont_type/">MIGS bacteria➞pos_cont_type</a>  <sub>0..1</sub><ul>
<li>Description: The substance, mixture, product, or apparatus used to verify that a process which is part of an investigation delivers a true positive.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example:  None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_4">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_subspecf_gen_lin/">MIGS bacteria➞subspecf_gen_lin</a>  <sub>0..1</sub><ul>
<li>Description: Information about the genetic distinctness of the sequenced organism below the subspecies level, e.g., serovar, serotype, biotype, ecotype, or any relevant genetic typing schemes like Group I plasmid. Subspecies should not be recorded in this term, but in the NCBI taxonomy. Supply both the lineage name and the lineage rank separated by a colon, e.g., biovar:abc123.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: serovar:Newport None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_5">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_num_replicons/">MIGS bacteria➞num_replicons</a>  <sub>1..1</sub><ul>
<li>Description: Reports the number of replicons in a nuclear genome of eukaryotes, in the genome of a bacterium or archaea or the number of segments in a segmented virus. Always applied to the haploid chromosome count of a eukaryote</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 2 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_6">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_extrachrom_elements/">MIGS bacteria➞extrachrom_elements</a>  <sub>0..1</sub><ul>
<li>Description: Do plasmids exist of significant phenotypic consequence (e.g. ones that determine virulence or antibiotic resistance). Megaplasmids? Other plasmids (borrelia has 15+ plasmids)</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 5 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_7">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_estimated_size/">MIGS bacteria➞estimated_size</a>  <sub>0..1</sub><ul>
<li>Description: The estimated size of the genome prior to sequencing. Of particular importance in the sequencing of (eukaryotic) genome which could remain in draft form for a long or unspecified period.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 300000 bp None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_8">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_ref_biomaterial/">MIGS bacteria➞ref_biomaterial</a>  <sub>1..1</sub><ul>
<li>Description: Primary publication if isolated before genome publication; otherwise, primary genome report.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: doi:10.1016/j.syapm.2018.01.009 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_9">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_source_mat_id/">MIGS bacteria➞source_mat_id</a>  <sub>0..1</sub><ul>
<li>Description: A unique identifier assigned to a material sample (as defined by http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular digital record of a material sample) used for extracting nucleic acids, and subsequent sequencing. The identifier can refer either to the original material collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher, /bio_material, or /culture_collection may or may not share the same value as the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id may both contain 'UAM:Herps:14' , referring to both the specimen voucher and sampled tissue with the same identifier. However, the /culture_collection qualifier may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id would refer to an identifier from some derived culture from which the nucleic acids were extracted (e.g. xatc123 or ark:/2154/R2).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: MPI012345 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_10">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_pathogenicity/">MIGS bacteria➞pathogenicity</a>  <sub>0..1</sub><ul>
<li>Description: To what is the entity pathogenic</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: human, animal, plant, fungi, bacteria None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_11">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_biotic_relationship/">MIGS bacteria➞biotic_relationship</a>  <sub>0..1</sub><ul>
<li>Description: Description of relationship(s) between the subject organism and other organism(s) it is associated with. E.g., parasite on species X; mutualist with species Y. The target organism is the subject of the relationship, and the other organism(s) is the object</li>
<li>Range: <a href="../biotic_relationship_enum/">biotic_relationship_enum</a></li>
<li>Example: free living None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_12">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_specific_host/">MIGS bacteria➞specific_host</a>  <sub>0..1</sub><ul>
<li>Description: Report the host's taxonomic name and/or NCBI taxonomy ID.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Homo sapiens and/or 9606 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_13">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_host_spec_range/">MIGS bacteria➞host_spec_range</a>  <sub>0..1</sub><ul>
<li>Description: The range and diversity of host species that an organism is capable of infecting, defined by NCBI taxonomy identifier.</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 9606 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_14">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_trophic_level/">MIGS bacteria➞trophic_level</a>  <sub>0..1</sub><ul>
<li>Description: Trophic levels are the feeding position in a food chain. Microbes can be a range of producers (e.g. chemolithotroph)</li>
<li>Range: <a href="../trophic_level_enum/">trophic_level_enum</a></li>
<li>Example: heterotroph None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_15">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_encoded_traits/">MIGS bacteria➞encoded_traits</a>  <sub>0..1</sub><ul>
<li>Description: Should include key traits like antibiotic resistance or xenobiotic degradation phenotypes for plasmids, converting genes for phage</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: beta-lactamase class A None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_16">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_rel_to_oxygen/">MIGS bacteria➞rel_to_oxygen</a>  <sub>0..1</sub><ul>
<li>Description: Is this organism an aerobe, anaerobe? Please note that aerobic and anaerobic are valid descriptors for microbial environments</li>
<li>Range: <a href="../rel_to_oxygen_enum/">rel_to_oxygen_enum</a></li>
<li>Example: aerobe None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_17">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_isol_growth_condt/">MIGS bacteria➞isol_growth_condt</a>  <sub>1..1</sub><ul>
<li>Description: Publication reference in the form of pubmed ID (pmid), digital object identifier (doi) or url for isolation and growth condition specifications of the organism/material</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: doi: 10.1016/j.syapm.2018.01.009 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_18">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_samp_collec_device/">MIGS bacteria➞samp_collec_device</a>  <sub>0..1</sub><ul>
<li>Description: The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: swab, biopsy, niskin bottle, push core, drag swab [GENEPIO:0002713] None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_19">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_samp_collec_method/">MIGS bacteria➞samp_collec_method</a>  <sub>0..1</sub><ul>
<li>Description: The method employed for collecting the sample.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: swabbing None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_20">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_samp_mat_process/">MIGS bacteria➞samp_mat_process</a>  <sub>0..1</sub><ul>
<li>Description: A brief description of any processing applied to the sample during or after retrieving the sample from environment, or a link to the relevant protocol(s) performed.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: filtering of seawater, storing samples in ethanol None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_21">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_samp_size/">MIGS bacteria➞samp_size</a>  <sub>0..1</sub><ul>
<li>Description: The total amount or size (volume (ml), mass (g) or area (m2) ) of sample collected.</li>
<li>Range: <a href="../QuantityValue/">QuantityValue</a></li>
<li>Example: 5 liter None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_22">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_nucl_acid_ext/">MIGS bacteria➞nucl_acid_ext</a>  <sub>0..1</sub><ul>
<li>Description: A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the material separation to recover the nucleic acid fraction from a sample</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: https://mobio.com/media/wysiwyg/pdfs/protocols/12888.pdf None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_23">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_nucl_acid_amp/">MIGS bacteria➞nucl_acid_amp</a>  <sub>0..1</sub><ul>
<li>Description: A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the enzymatic amplification (PCR, TMA, NASBA) of specific nucleic acids</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: https://phylogenomics.me/protocols/16s-pcr-protocol/ None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_24">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_lib_size/">MIGS bacteria➞lib_size</a>  <sub>0..1</sub><ul>
<li>Description: Total number of clones in the library prepared for the project</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 50 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_25">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_lib_reads_seqd/">MIGS bacteria➞lib_reads_seqd</a>  <sub>0..1</sub><ul>
<li>Description: Total number of clones sequenced from the library</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 20 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_26">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_lib_layout/">MIGS bacteria➞lib_layout</a>  <sub>0..1</sub><ul>
<li>Description: Specify whether to expect single, paired, or other configuration of reads</li>
<li>Range: <a href="../lib_layout_enum/">lib_layout_enum</a></li>
<li>Example: paired None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_27">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_lib_vector/">MIGS bacteria➞lib_vector</a>  <sub>0..1</sub><ul>
<li>Description: Cloning vector type(s) used in construction of libraries</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Bacteriophage P1 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_28">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_lib_screen/">MIGS bacteria➞lib_screen</a>  <sub>0..1</sub><ul>
<li>Description: Specific enrichment or screening methods applied before and/or after creating libraries</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: enriched, screened, normalized None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_29">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_adapters/">MIGS bacteria➞adapters</a>  <sub>0..1</sub><ul>
<li>Description: Adapters provide priming sequences for both amplification and sequencing of the sample-library fragments. Both adapters should be reported; in uppercase letters</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: AATGATACGGCGACCACCGAGATCTACACGCT;CAAGCAGAAGACGGCATACGAGAT None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_30">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_seq_meth/">MIGS bacteria➞seq_meth</a>  <sub>1..1</sub><ul>
<li>Description: Sequencing machine used. Where possible the term should be taken from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103).</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: 454 Genome Sequencer FLX [OBI:0000702] None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_31">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_tax_ident/">MIGS bacteria➞tax_ident</a>  <sub>0..1</sub><ul>
<li>Description: The phylogenetic marker(s) used to assign an organism name to the SAG or MAG</li>
<li>Range: <a href="../tax_ident_enum/">tax_ident_enum</a></li>
<li>Example: other: rpoB gene None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_32">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_assembly_qual/">MIGS bacteria➞assembly_qual</a>  <sub>1..1</sub><ul>
<li>Description: The assembly quality category is based on sets of criteria outlined for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities with a consensus error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments where gaps span repetitive regions. Presence of the 23S, 16S and 5S rRNA genes and at least 18 tRNAs. Medium Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Low Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Assembly statistics include, but are not limited to total assembly size, number of contigs, contig N50/L50, and maximum contig length. For MIUVIG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities, with extensive manual review and editing to annotate putative gene functions and transcriptional units. High-quality draft genome: One or multiple fragments, totaling ≥ 90% of the expected genome or replicon sequence or predicted complete. Genome fragment(s): One or multiple fragments, totalling &lt; 90% of the expected genome or replicon sequence, or for which no genome size could be estimated</li>
<li>Range: <a href="../assembly_qual_enum/">assembly_qual_enum</a></li>
<li>Example: High-quality draft genome None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_33">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_assembly_name/">MIGS bacteria➞assembly_name</a>  <sub>0..1</sub><ul>
<li>Description: Name/version of the assembly provided by the submitter that is used in the genome browsers and in the community</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: HuRef, JCVI_ISG_i3_1.0 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_34">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_assembly_software/">MIGS bacteria➞assembly_software</a>  <sub>1..1</sub><ul>
<li>Description: Tool(s) used for assembly, including version number and parameters</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: metaSPAdes;3.11.0;kmer set 21,33,55,77,99,121, default parameters otherwise None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_35">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_annot/">MIGS bacteria➞annot</a>  <sub>0..1</sub><ul>
<li>Description: Tool used for annotation, or for cases where annotation was provided by a community jamboree or model organism database rather than by a specific submitter</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: prokka None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_36">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_number_contig/">MIGS bacteria➞number_contig</a>  <sub>1..1</sub><ul>
<li>Description: Total number of contigs in the cleaned/submitted assembly that makes up a given genome, SAG, MAG, or UViG</li>
<li>Range: <a href="../types/Integer/">Integer</a></li>
<li>Example: 40 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_37">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_feat_pred/">MIGS bacteria➞feat_pred</a>  <sub>0..1</sub><ul>
<li>Description: Method used to predict UViGs features such as ORFs, integration site, etc.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: Prodigal;2.6.3;default parameters None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_38">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_ref_db/">MIGS bacteria➞ref_db</a>  <sub>0..1</sub><ul>
<li>Description: List of database(s) used for ORF annotation, along with version number and reference to website or publication</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: pVOGs;5;http://dmk-brain.ecn.uiowa.edu/pVOGs/ Grazziotin et al. 2017 doi:10.1093/nar/gkw975 None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_39">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_sim_search_meth/">MIGS bacteria➞sim_search_meth</a>  <sub>0..1</sub><ul>
<li>Description: Tool used to compare ORFs with database, along with version and cutoffs used</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: HMMER3;3.1b2;hmmsearch, cutoff of 50 on score None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_40">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_tax_class/">MIGS bacteria➞tax_class</a>  <sub>0..1</sub><ul>
<li>Description: Method used for taxonomic classification, along with reference database used, classification rank, and thresholds used to classify new genomes</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: vConTACT vContact2 (references from NCBI RefSeq v83, genus rank classification, default parameters) None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_41">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_compl_score/">MIGS bacteria➞compl_score</a>  <sub>0..1</sub><ul>
<li>Description: Completeness score is typically based on either the fraction of markers found as compared to a database or the percent of a genome found as compared to a closely related reference genome. High Quality Draft: &gt;90%, Medium Quality Draft: &gt;50%, and Low Quality Draft: &lt; 50% should have the indicated completeness scores</li>
<li>Range: <a href="../compl_score_enum/">compl_score_enum</a></li>
<li>Example: med;60% None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_42">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_compl_software/">MIGS bacteria➞compl_software</a>  <sub>0..1</sub><ul>
<li>Description: Tools used for completion estimate, i.e. checkm, anvi'o, busco</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: checkm None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_43">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_associated_resource/">MIGS bacteria➞associated resource</a>  <sub>0..1</sub><ul>
<li>Description: A related resource that is referenced, cited, or otherwise associated to the sequence.</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: http://www.earthmicrobiome.org/ None</li>
</ul>
</li>
</ul>
<h3 id="mixed-in-from-migs-bacteria_44">Mixed in from MIGS bacteria:</h3>
<ul>
<li><a href="../MIGS_bacteria_sop/">MIGS bacteria➞sop</a>  <sub>0..1</sub><ul>
<li>Description: Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences</li>
<li>Range: <a href="../types/String/">String</a></li>
<li>Example: http://press.igsb.anl.gov/earthmicrobiome/protocols-and-standards/its/ None</li>
</ul>
</li>
</ul>
              
            </div>
          </div>
          <footer>
  

  <hr/>

  <div role="contentinfo">
    <!-- Copyright etc -->
    
  </div>

  Built with <a href="https://www.mkdocs.org/">MkDocs</a> using a <a href="https://github.com/snide/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>.
</footer>
      
        </div>
      </div>

    </section>

  </div>

  <div class="rst-versions" role="note" aria-label="versions">
  <span class="rst-current-version" data-toggle="rst-current-version">
    
    
    
  </span>
</div>
    <script>var base_url = '..';</script>
    <script src="../js/theme_extra.js" defer></script>
    <script src="../js/theme.js" defer></script>
      <script src="../search/main.js" defer></script>
    <script defer>
        window.onload = function () {
            SphinxRtdTheme.Navigation.enable(true);
        };
    </script>

</body>
</html>
