<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="X-UA-Compatible" content="IE=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1.0">
        
        
        <link rel="canonical" href="https://genomicsstandardsconsortium.github.io/mixs/HydrocarbonResources-fluidsSwabsMIGSOrg/">
        <link rel="shortcut icon" href="../img/favicon.ico">
        <title>HydrocarbonResources-fluidsSwabsMIGSOrg - MIxS</title>
        <link href="../css/bootstrap.min.css" rel="stylesheet">
        <link href="../css/font-awesome.min.css" rel="stylesheet">
        <link href="../css/base.css" rel="stylesheet">
        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/styles/github.min.css">

        <script src="../js/jquery-1.10.2.min.js" defer></script>
        <script src="../js/bootstrap.min.js" defer></script>
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/10.5.0/highlight.min.js"></script>
        <script>hljs.initHighlightingOnLoad();</script> 
    </head>

    <body>
        <div class="navbar fixed-top navbar-expand-lg navbar-dark bg-dark">
            <div class="container">
                <a class="navbar-brand" href="..">MIxS</a>
                <!-- Expander button -->
                <button type="button" class="navbar-toggler" data-toggle="collapse" data-target="#navbar-collapse">
                    <span class="navbar-toggler-icon"></span>
                </button>

                <!-- Expanded navigation -->
                <div id="navbar-collapse" class="navbar-collapse collapse">
                        <!-- Main navigation -->
                        <ul class="nav navbar-nav">
                            <li class="navitem">
                                <a href=".." class="nav-link">About</a>
                            </li>
                            <li class="dropdown">
                                <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown">Introduction <b class="caret"></b></a>
                                <ul class="dropdown-menu">
                                    
<li>
    <a href="../introduction/background/" class="dropdown-item">Background</a>
</li>
                                    
<li>
    <a href="../introduction/citation/" class="dropdown-item">Cite</a>
</li>
                                    
<li>
    <a href="../introduction/contents/" class="dropdown-item">Contents</a>
</li>
                                </ul>
                            </li>
                        </ul>

                    <ul class="nav navbar-nav ml-auto">
                        <li class="nav-item">
                            <a href="#" class="nav-link" data-toggle="modal" data-target="#mkdocs_search_modal">
                                <i class="fa fa-search"></i> Search
                            </a>
                        </li>
                            <li class="nav-item">
                                <a href="https://github.com/GenomicsStandardsConsortium/mixs/edit/master/docs/HydrocarbonResources-fluidsSwabsMIGSOrg.md" class="nav-link"><i class="fa fa-github"></i> Edit on GitHub</a>
                            </li>
                    </ul>
                </div>
            </div>
        </div>

        <div class="container">
            <div class="row">
                    <div class="col-md-3"><div class="navbar-light navbar-expand-md bs-sidebar hidden-print affix" role="complementary">
    <div class="navbar-header">
        <button type="button" class="navbar-toggler collapsed" data-toggle="collapse" data-target="#toc-collapse" title="Table of Contents">
            <span class="fa fa-angle-down"></span>
        </button>
    </div>

    
    <div id="toc-collapse" class="navbar-collapse collapse card bg-secondary">
        <ul class="nav flex-column">
            
            <li class="nav-item" data-level="1"><a href="#hydrocarbonresources-fluidsswabsmigsorg" class="nav-link">HydrocarbonResources-fluidsSwabsMIGSOrg</a>
              <ul class="nav flex-column">
            <li class="nav-item" data-level="2"><a href="#inheritance" class="nav-link">Inheritance</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#slots" class="nav-link">Slots</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#usages" class="nav-link">Usages</a>
              <ul class="nav flex-column">
              </ul>
            </li>
            <li class="nav-item" data-level="2"><a href="#identifier-and-mapping-information" class="nav-link">Identifier and Mapping Information</a>
              <ul class="nav flex-column">
              </ul>
            </li>
              </ul>
            </li>
        </ul>
    </div>
</div></div>
                    <div class="col-md-9" role="main">

<h1 id="hydrocarbonresources-fluidsswabsmigsorg">HydrocarbonResources-fluidsSwabsMIGSOrg</h1>
<p>Combinatorial checklist Minimal Information about a Genome Sequence: org with environmental package hydrocarbon resources-fluids_swabs</p>
<p>URI: <a href="https://w3id.org/mixs/vocab/HydrocarbonResources-fluidsSwabsMIGSOrg">mixs.vocab:HydrocarbonResources-fluidsSwabsMIGSOrg</a></p>
<h2 id="inheritance">Inheritance</h2>
<ul>
<li><a href="../HydrocarbonResources-fluidsSwabs/">HydrocarbonResources-fluidsSwabs</a><ul>
<li><strong>HydrocarbonResources-fluidsSwabsMIGSOrg</strong> [ MIGS org]</li>
</ul>
</li>
</ul>
<!-- no inheritance hierarchy -->

<h2 id="slots">Slots</h2>
<table>
<thead>
<tr>
<th>MIxS ID</th>
<th>Name</th>
<th>Multiplicity</th>
<th>Description</th>
<th>Pattern</th>
<th>Info</th>
</tr>
</thead>
<tbody>
<tr>
<td>MIXS:0001107</td>
<td><a href="../samp_name/">samp_name</a></td>
<td>1..1</td>
<td>A local identifier or name that for the material sample used for extracting nucleic acids, and subsequent sequencing. It can refer either to the original material collected or to any derived sub-samples. It can have any format, but we suggest that you make it concise, unique and consistent within your lab, and as informative as possible. INSDC requires every sample name from a single Submitter to be unique. Use of a globally unique identifier for the field source_mat_id is recommended in addition to sample_name.</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001320</td>
<td><a href="../samp_taxon_id/">samp_taxon_id</a></td>
<td>1..1</td>
<td>NCBI taxon id of the sample.  Maybe be a single taxon or mixed taxa sample. Use 'synthetic metagenome’ for mock community/positive controls, or 'blank sample' for negative controls.</td>
<td>{text} [NCBI:txid]</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000092</td>
<td><a href="../project_name/">project_name</a></td>
<td>1..1</td>
<td>Name of the project within which the sequencing was organized</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000008</td>
<td><a href="../experimental_factor/">experimental_factor</a></td>
<td>0..1</td>
<td>Experimental factors are essentially the variable aspects of an experiment design which can be used to describe an experiment, or set of experiments, in an increasingly detailed manner. This field accepts ontology terms from Experimental Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI). For a browser of EFO (v 2.95) terms, please see http://purl.bioontology.org/ontology/EFO; for a browser of OBI (v 2018-02-12) terms please see http://purl.bioontology.org/ontology/OBI</td>
<td>{termLabel} {[termID]}</td>
<td>{text}</td>
</tr>
<tr>
<td>MIXS:0000009</td>
<td><a href="../lat_lon/">lat_lon</a></td>
<td>1..1</td>
<td>The geographical origin of the sample as defined by latitude and longitude. The values should be reported in decimal degrees and in WGS84 system</td>
<td>{float} {float}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000010</td>
<td><a href="../geo_loc_name/">geo_loc_name</a></td>
<td>1..1</td>
<td>The geographical origin of the sample as defined by the country or sea name followed by specific region name. Country or sea names should be chosen from the INSDC country list (http://insdc.org/country.html), or the GAZ ontology (http://purl.bioontology.org/ontology/GAZ)</td>
<td>{term}: {term}, {text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000011</td>
<td><a href="../collection_date/">collection_date</a></td>
<td>1..1</td>
<td>The time of sampling, either as an instance (single point in time) or interval. In case no exact time is available, the date/time can be right truncated i.e. all of these are valid times: 2008-01-23T19:23:10+00:00; 2008-01-23T19:23:10; 2008-01-23; 2008-01; 2008; Except: 2008-01; 2008 all are ISO8601 compliant</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001321</td>
<td><a href="../neg_cont_type/">neg_cont_type</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The substance or equipment used as a negative control in an investigation</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001322</td>
<td><a href="../pos_cont_type/">pos_cont_type</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The substance, mixture, product, or apparatus used to verify that a process which is part of an investigation delivers a true positive.</td>
<td>{term} or {text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000012</td>
<td><a href="../env_broad_scale/">env_broad_scale</a></td>
<td>1..1</td>
<td>Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO’s biome class:  http://purl.obolibrary.org/obo/ENVO_00000428. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000013</td>
<td><a href="../env_local_scale/">env_local_scale</a></td>
<td>1..1</td>
<td>Report the entity or entities which are in the sample or specimen’s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for env_broad_scale. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS.</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000014</td>
<td><a href="../env_medium/">env_medium</a></td>
<td>1..1</td>
<td>Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of 'environmental material' (http://purl.obolibrary.org/obo/ENVO_00010483). EnvO documentation about how to use the field: https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000020</td>
<td><a href="../subspecf_gen_lin/">subspecf_gen_lin</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Information about the genetic distinctness of the sequenced organism below the subspecies level, e.g., serovar, serotype, biotype, ecotype, or any relevant genetic typing schemes like Group I plasmid. Subspecies should not be recorded in this term, but in the NCBI taxonomy. Supply both the lineage name and the lineage rank separated by a colon, e.g., biovar:abc123.</td>
<td>{rank name}:{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000023</td>
<td><a href="../extrachrom_elements/">extrachrom_elements</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Do plasmids exist of significant phenotypic consequence (e.g. ones that determine virulence or antibiotic resistance). Megaplasmids? Other plasmids (borrelia has 15+ plasmids)</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000024</td>
<td><a href="../estimated_size/">estimated_size</a></td>
<td>0..1</td>
<td>The estimated size of the genome prior to sequencing. Of particular importance in the sequencing of (eukaryotic) genome which could remain in draft form for a long or unspecified period.</td>
<td>{integer} bp</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000025</td>
<td><a href="../ref_biomaterial/">ref_biomaterial</a></td>
<td>0..1</td>
<td>Primary publication if isolated before genome publication; otherwise, primary genome report.</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000026</td>
<td><a href="../source_mat_id/">source_mat_id</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A unique identifier assigned to a material sample (as defined by http://rs.tdwg.org/dwc/terms/materialSampleID, and as opposed to a particular digital record of a material sample) used for extracting nucleic acids, and subsequent sequencing. The identifier can refer either to the original material collected or to any derived sub-samples. The INSDC qualifiers /specimen_voucher, /bio_material, or /culture_collection may or may not share the same value as the source_mat_id field. For instance, the /specimen_voucher qualifier and source_mat_id may both contain 'UAM:Herps:14' , referring to both the specimen voucher and sampled tissue with the same identifier. However, the /culture_collection qualifier may refer to a value from an initial culture (e.g. ATCC:11775) while source_mat_id would refer to an identifier from some derived culture from which the nucleic acids were extracted (e.g. xatc123 or ark:/2154/R2).</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000003</td>
<td><a href="../isol_growth_condt/">isol_growth_condt</a></td>
<td>1..1</td>
<td>Publication reference in the form of pubmed ID (pmid), digital object identifier (doi) or url for isolation and growth condition specifications of the organism/material</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000002</td>
<td><a href="../samp_collec_device/">samp_collec_device</a></td>
<td>0..1</td>
<td>The device used to collect an environmental sample. This field accepts terms listed under environmental sampling device (http://purl.obolibrary.org/obo/ENVO). This field also accepts terms listed under specimen collection device (http://purl.obolibrary.org/obo/GENEPIO_0002094).</td>
<td>{termLabel} {[termID]}</td>
<td>{text}</td>
</tr>
<tr>
<td>MIXS:0001225</td>
<td><a href="../samp_collec_method/">samp_collec_method</a></td>
<td>0..1</td>
<td>The method employed for collecting the sample.</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000016</td>
<td><a href="../samp_mat_process/">samp_mat_process</a></td>
<td>0..1</td>
<td>A brief description of any processing applied to the sample during or after retrieving the sample from environment, or a link to the relevant protocol(s) performed.</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000001</td>
<td><a href="../samp_size/">samp_size</a></td>
<td>0..1</td>
<td>The total amount or size (volume (ml), mass (g) or area (m2) ) of sample collected.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000111</td>
<td><a href="../samp_vol_we_dna_ext/">samp_vol_we_dna_ext</a></td>
<td>0..1</td>
<td>Volume (ml) or mass (g) of total collected sample processed for DNA extraction. Note: total sample collected should be entered under the term Sample Size (MIXS:0000001).</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000037</td>
<td><a href="../nucl_acid_ext/">nucl_acid_ext</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the material separation to recover the nucleic acid fraction from a sample</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000038</td>
<td><a href="../nucl_acid_amp/">nucl_acid_amp</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A link to a literature reference, electronic resource or a standard operating procedure (SOP), that describes the enzymatic amplification (PCR, TMA, NASBA) of specific nucleic acids</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000039</td>
<td><a href="../lib_size/">lib_size</a></td>
<td>0..1</td>
<td>Total number of clones in the library prepared for the project</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000040</td>
<td><a href="../lib_reads_seqd/">lib_reads_seqd</a></td>
<td>0..1</td>
<td>Total number of clones sequenced from the library</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000041</td>
<td><a href="../lib_layout/">lib_layout</a></td>
<td>0..1</td>
<td>Specify whether to expect single, paired, or other configuration of reads</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000042</td>
<td><a href="../lib_vector/">lib_vector</a></td>
<td>0..1</td>
<td>Cloning vector type(s) used in construction of libraries</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000043</td>
<td><a href="../lib_screen/">lib_screen</a></td>
<td>0..1</td>
<td>Specific enrichment or screening methods applied before and/or after creating libraries</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000048</td>
<td><a href="../adapters/">adapters</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Adapters provide priming sequences for both amplification and sequencing of the sample-library fragments. Both adapters should be reported; in uppercase letters</td>
<td>{dna};{dna}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000050</td>
<td><a href="../seq_meth/">seq_meth</a></td>
<td>1..1</td>
<td>Sequencing machine used. Where possible the term should be taken from the OBI list of DNA sequencers (http://purl.obolibrary.org/obo/OBI_0400103).</td>
<td>{termLabel} {[termID]}</td>
<td>{text}</td>
</tr>
<tr>
<td>MIXS:0000053</td>
<td><a href="../tax_ident/">tax_ident</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>The phylogenetic marker(s) used to assign an organism name to the SAG or MAG</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000056</td>
<td><a href="../assembly_qual/">assembly_qual</a></td>
<td>0..1</td>
<td>The assembly quality category is based on sets of criteria outlined for each assembly quality category. For MISAG/MIMAG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities with a consensus error rate equivalent to Q50 or better. High Quality Draft:Multiple fragments where gaps span repetitive regions. Presence of the 23S, 16S and 5S rRNA genes and at least 18 tRNAs. Medium Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Low Quality Draft:Many fragments with little to no review of assembly other than reporting of standard assembly statistics. Assembly statistics include, but are not limited to total assembly size, number of contigs, contig N50/L50, and maximum contig length. For MIUVIG; Finished: Single, validated, contiguous sequence per replicon without gaps or ambiguities, with extensive manual review and editing to annotate putative gene functions and transcriptional units. High-quality draft genome: One or multiple fragments, totaling ≥ 90% of the expected genome or replicon sequence or predicted complete. Genome fragment(s): One or multiple fragments, totalling &lt; 90% of the expected genome or replicon sequence, or for which no genome size could be estimated</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000057</td>
<td><a href="../assembly_name/">assembly_name</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Name/version of the assembly provided by the submitter that is used in the genome browsers and in the community</td>
<td>{text} {text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000058</td>
<td><a href="../assembly_software/">assembly_software</a></td>
<td>1..1</td>
<td>Tool(s) used for assembly, including version number and parameters</td>
<td>{software};{version};{parameters}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000059</td>
<td><a href="../annot/">annot</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Tool used for annotation, or for cases where annotation was provided by a community jamboree or model organism database rather than by a specific submitter</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000060</td>
<td><a href="../number_contig/">number_contig</a></td>
<td>0..1</td>
<td>Total number of contigs in the cleaned/submitted assembly that makes up a given genome, SAG, MAG, or UViG</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000061</td>
<td><a href="../feat_pred/">feat_pred</a></td>
<td>0..1</td>
<td>Method used to predict UViGs features such as ORFs, integration site, etc.</td>
<td>{software};{version};{parameters}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000062</td>
<td><a href="../ref_db/">ref_db</a></td>
<td>0..1</td>
<td>List of database(s) used for ORF annotation, along with version number and reference to website or publication</td>
<td>{database};{version};{reference}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000063</td>
<td><a href="../sim_search_meth/">sim_search_meth</a></td>
<td>0..1</td>
<td>Tool used to compare ORFs with database, along with version and cutoffs used</td>
<td>{software};{version};{parameters}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000064</td>
<td><a href="../tax_class/">tax_class</a></td>
<td>0..1</td>
<td>Method used for taxonomic classification, along with reference database used, classification rank, and thresholds used to classify new genomes</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000069</td>
<td><a href="../compl_score/">compl_score</a></td>
<td>0..1</td>
<td>Completeness score is typically based on either the fraction of markers found as compared to a database or the percent of a genome found as compared to a closely related reference genome. High Quality Draft: &gt;90%, Medium Quality Draft: &gt;50%, and Low Quality Draft: &lt; 50% should have the indicated completeness scores</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000070</td>
<td><a href="../compl_software/">compl_software</a></td>
<td>0..1</td>
<td>Tools used for completion estimate, i.e. checkm, anvi'o, busco</td>
<td>{software};{version}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000091</td>
<td><a href="../associated_resource/">associated_resource</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>A related resource that is referenced, cited, or otherwise associated to the sequence.</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000090</td>
<td><a href="../sop/">sop</a></td>
<td>0..1 <em>recommended</em> RECOMMENDED</td>
<td>Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences</td>
<td>{PMID}</td>
<td>{DOI}</td>
</tr>
<tr>
<td>MIXS:0000018</td>
<td><a href="../depth/">depth</a></td>
<td>0..1</td>
<td>The vertical distance below local surface, e.g. for sediment or soil samples depth is measured from sediment or soil surface, respectively. Depth can be reported as an interval for subsurface samples.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000094</td>
<td><a href="../alt/">alt</a></td>
<td>0..1</td>
<td>Altitude is a term used to identify heights of objects such as airplanes, space shuttles, rockets, atmospheric balloons and heights of places such as atmospheric layers and clouds. It is used to measure the height of an object which is above the earth's surface. In this context, the altitude measurement is the vertical distance between the earth's surface above sea level and the sampled position in the air</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000093</td>
<td><a href="../elev/">elev</a></td>
<td>0..1</td>
<td>Elevation of the sampling site is its height above a fixed reference point, most commonly the mean sea level. Elevation is mainly used when referring to points on the earth's surface, while altitude is used for points above the surface, such as an aircraft in flight or a spacecraft in orbit.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000113</td>
<td><a href="../temp/">temp</a></td>
<td>1..1</td>
<td>Temperature of the sample at the time of sampling.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000988</td>
<td><a href="../hcr/">hcr</a></td>
<td>1..1</td>
<td>Main Hydrocarbon Resource type. The term "Hydrocarbon Resource" HCR defined as a natural environmental feature containing large amounts of hydrocarbons at high concentrations potentially suitable for commercial exploitation. This term should not be confused with the Hydrocarbon Occurrence term which also includes hydrocarbon-rich environments with currently limited commercial interest such as seeps, outcrops, gas hydrates etc. If "other" is specified, please propose entry in "additional info" field</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000989</td>
<td><a href="../hc_produced/">hc_produced</a></td>
<td>1..1</td>
<td>Main hydrocarbon type produced from resource (i.e. Oil, gas, condensate, etc). If "other" is specified, please propose entry in "additional info" field</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000290</td>
<td><a href="../basin/">basin</a></td>
<td>1..1</td>
<td>Name of the basin (e.g. Campos)</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000291</td>
<td><a href="../field/">field</a></td>
<td>0..1</td>
<td>Name of the hydrocarbon field (e.g. Albacora)</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000303</td>
<td><a href="../reservoir/">reservoir</a></td>
<td>0..1</td>
<td>Name of the reservoir (e.g. Carapebus)</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000393</td>
<td><a href="../hcr_temp/">hcr_temp</a></td>
<td>0..1</td>
<td>Original temperature of the hydrocarbon resource</td>
<td>{float} - {float} {unit}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000394</td>
<td><a href="../tvdss_of_hcr_temp/">tvdss_of_hcr_temp</a></td>
<td>0..1</td>
<td>True vertical depth subsea (TVDSS) of the hydrocarbon resource where the original temperature was measured (e.g. 1345 m).</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000395</td>
<td><a href="../hcr_pressure/">hcr_pressure</a></td>
<td>0..1</td>
<td>Original pressure of the hydrocarbon resource</td>
<td>{float} - {float} {unit}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000397</td>
<td><a href="../tvdss_of_hcr_press/">tvdss_of_hcr_press</a></td>
<td>0..1</td>
<td>True vertical depth subsea (TVDSS) of the hydrocarbon resource where the original pressure was measured (e.g. 1578 m).</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000990</td>
<td><a href="../lithology/">lithology</a></td>
<td>0..1</td>
<td>Hydrocarbon resource main lithology (Additional information: http://petrowiki.org/Lithology_and_rock_type_determination). If "other" is specified, please propose entry in "additional info" field</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000992</td>
<td><a href="../depos_env/">depos_env</a></td>
<td>0..1</td>
<td>Main depositional environment (https://en.wikipedia.org/wiki/Depositional_environment). If "other" is specified, please propose entry in "additional info" field</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000993</td>
<td><a href="../hcr_geol_age/">hcr_geol_age</a></td>
<td>0..1</td>
<td>Geological age of hydrocarbon resource (Additional info: https://en.wikipedia.org/wiki/Period_(geology)). If "other" is specified, please propose entry in "additional info" field</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000406</td>
<td><a href="../hcr_fw_salinity/">hcr_fw_salinity</a></td>
<td>0..1</td>
<td>Original formation water salinity (prior to secondary recovery e.g. Waterflooding) expressed as TDS</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000407</td>
<td><a href="../sulfate_fw/">sulfate_fw</a></td>
<td>0..1</td>
<td>Original sulfate concentration in the hydrocarbon resource</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000408</td>
<td><a href="../vfa_fw/">vfa_fw</a></td>
<td>0..1</td>
<td>Original volatile fatty acid concentration in the hydrocarbon resource</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001008</td>
<td><a href="../prod_start_date/">prod_start_date</a></td>
<td>0..1</td>
<td>Date of field's first production</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000452</td>
<td><a href="../prod_rate/">prod_rate</a></td>
<td>0..1</td>
<td>Oil and/or gas production rates per well (e.g. 524 m3 / day)</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000453</td>
<td><a href="../water_prod_rate/">water_prod_rate</a></td>
<td>0..1</td>
<td>Water production rates per well (e.g. 987 m3 / day)</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000454</td>
<td><a href="../water_cut/">water_cut</a></td>
<td>1..1</td>
<td>Current amount of water (%) in a produced fluid stream; or the average of the combined streams</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000455</td>
<td><a href="../iwf/">iwf</a></td>
<td>1..1</td>
<td>Proportion of the produced fluids derived from injected water at the time of sampling. (e.g. 87%)</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001009</td>
<td><a href="../add_recov_method/">add_recov_method</a></td>
<td>1..1</td>
<td>Additional (i.e. Secondary, tertiary, etc.) recovery methods deployed for increase of hydrocarbon recovery from resource and start date for each one of them. If "other" is specified, please propose entry in "additional info" field</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001010</td>
<td><a href="../iw_bt_date_well/">iw_bt_date_well</a></td>
<td>0..1</td>
<td>Injection water breakthrough date per well following a secondary and/or tertiary recovery</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001011</td>
<td><a href="../biocide/">biocide</a></td>
<td>0..1</td>
<td>List of biocides (commercial name of product and supplier) and date of administration</td>
<td>{text};{text};{timestamp}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000456</td>
<td><a href="../biocide_admin_method/">biocide_admin_method</a></td>
<td>0..1</td>
<td>Method of biocide administration (dose, frequency, duration, time elapsed between last biociding and sampling) (e.g. 150 mg/l; weekly; 4 hr; 3 days)</td>
<td>{float} {unit};{Rn/start_time/end_time/duration};{duration}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001012</td>
<td><a href="../chem_treatment/">chem_treatment</a></td>
<td>0..1</td>
<td>List of chemical compounds administered upstream the sampling location where sampling occurred (e.g. Glycols, H2S scavenger, corrosion and scale inhibitors, demulsifiers, and other production chemicals etc.). The commercial name of the product and name of the supplier should be provided. The date of administration should also be included</td>
<td>{text};{text};{timestamp}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000457</td>
<td><a href="../chem_treat_method/">chem_treat_method</a></td>
<td>0..1</td>
<td>Method of chemical administration(dose, frequency, duration, time elapsed between administration and sampling) (e.g. 50 mg/l; twice a week; 1 hr; 0 days)</td>
<td>{float} {unit};{Rn/start_time/end_time/duration};{duration};{duration}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000136</td>
<td><a href="../samp_loc_corr_rate/">samp_loc_corr_rate</a></td>
<td>0..1</td>
<td>Metal corrosion rate is the speed of metal deterioration due to environmental conditions. As environmental conditions change corrosion rates change accordingly. Therefore, long term corrosion rates are generally more informative than short term rates and for that reason they are preferred during reporting. In the case of suspected MIC, corrosion rate measurements at the time of sampling might provide insights into the involvement of certain microbial community members in MIC as well as potential microbial interplays</td>
<td>{float} - {float} {unit}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000296</td>
<td><a href="../samp_well_name/">samp_well_name</a></td>
<td>0..1</td>
<td>Name of the well (e.g. BXA1123) where sample was taken</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000297</td>
<td><a href="../win/">win</a></td>
<td>0..1</td>
<td>A unique identifier of a well or wellbore. This is part of the Global Framework for Well Identification initiative which is compiled by the Professional Petroleum Data Management Association (PPDM) in an effort to improve well identification systems. (Supporting information: https://ppdm.org/ and http://dl.ppdm.org/dl/690)</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000998</td>
<td><a href="../samp_type/">samp_type</a></td>
<td>1..1</td>
<td>The type of material from which the sample was obtained. For the Hydrocarbon package, samples include types like core, rock trimmings, drill cuttings, piping section, coupon, pigging debris, solid deposit, produced fluid, produced water, injected water, swabs, etc. For the Food Package, samples are usually categorized as food, body products or tissues, or environmental material. This field accepts terms listed under environmental specimen (http://purl.obolibrary.org/obo/GENEPIO_0001246).</td>
<td>{termLabel} {[termID]}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000999</td>
<td><a href="../samp_subtype/">samp_subtype</a></td>
<td>0..1</td>
<td>Name of sample sub-type. For example if "sample type" is "Produced Water" then subtype could be "Oil Phase" or "Water Phase". If "other" is specified, please propose entry in "additional info" field</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001015</td>
<td><a href="../samp_collect_point/">samp_collect_point</a></td>
<td>1..1</td>
<td>Sampling point on the asset were sample was collected (e.g. Wellhead, storage tank, separator, etc). If "other" is specified, please propose entry in "additional info" field</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000412</td>
<td><a href="../pressure/">pressure</a></td>
<td>0..1</td>
<td>Pressure to which the sample is subject to, in atmospheres</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000753</td>
<td><a href="../oxy_stat_samp/">oxy_stat_samp</a></td>
<td>0..1</td>
<td>Oxygenation status of sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000463</td>
<td><a href="../samp_preserv/">samp_preserv</a></td>
<td>0..1</td>
<td>Preservative added to the sample (e.g. Rnalater, alcohol, formaldehyde, etc.). Where appropriate include volume added (e.g. Rnalater; 2 ml)</td>
<td>{text};{float} {unit}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000410</td>
<td><a href="../samp_transport_cond/">samp_transport_cond</a></td>
<td>0..1</td>
<td>Sample transport duration (in days or hrs) and temperature the sample was exposed to (e.g. 5.5 days; 20 ¬∞C)</td>
<td>{float} {unit};{float} {unit}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000110</td>
<td><a href="../samp_store_temp/">samp_store_temp</a></td>
<td>0..1</td>
<td>Temperature at which sample was stored, e.g. -80 degree Celsius</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000116</td>
<td><a href="../samp_store_dur/">samp_store_dur</a></td>
<td>0..1</td>
<td>Duration for which the sample was stored</td>
<td>{duration}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000755</td>
<td><a href="../samp_store_loc/">samp_store_loc</a></td>
<td>0..1</td>
<td>Location at which sample was stored, usually name of a specific freezer/room</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000103</td>
<td><a href="../organism_count/">organism_count</a></td>
<td>0..*</td>
<td>Total cell count of any organism (or group of organisms) per gram, volume or area of sample, should include name of organism followed by count. The method that was used for the enumeration (e.g. qPCR, atp, mpn, etc.) Should also be provided. (example: total prokaryotes; 3.5e7 cells per ml; qpcr)</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000099</td>
<td><a href="../org_count_qpcr_info/">org_count_qpcr_info</a></td>
<td>0..1</td>
<td>If qpcr was used for the cell count, the target gene name, the primer sequence and the cycling conditions should also be provided. (Example: 16S rrna; FWD:ACGTAGCTATGACGT REV:GTGCTAGTCGAGTAC; initial denaturation:90C_5min; denaturation:90C_2min; annealing:52C_30 sec; elongation:72C_30 sec; 90 C for 1 min; final elongation:72C_5min; 30 cycles)</td>
<td>{text};FWD:{dna};REV:{dna};initial denaturation:degrees_minutes;denaturation:degrees_minutes;annealing:degrees_minutes;elongation:degrees_minutes;final elongation:degrees_minutes; total cycles</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0001001</td>
<td><a href="../ph/">ph</a></td>
<td>0..1</td>
<td>Ph measurement of the sample, or liquid portion of sample, or aqueous phase of the fluid</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000183</td>
<td><a href="../salinity/">salinity</a></td>
<td>0..1</td>
<td>The total concentration of all dissolved salts in a liquid or solid sample. While salinity can be measured by a complete chemical analysis, this method is difficult and time consuming. More often, it is instead derived from the conductivity measurement. This is known as practical salinity. These derivations compare the specific conductance of the sample to a salinity standard such as seawater.</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000421</td>
<td><a href="../alkalinity/">alkalinity</a></td>
<td>0..1</td>
<td>Alkalinity, the ability of a solution to neutralize acids to the equivalence point of carbonate or bicarbonate</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000298</td>
<td><a href="../alkalinity_method/">alkalinity_method</a></td>
<td>0..1</td>
<td>Method used for alkalinity measurement</td>
<td>{text}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000423</td>
<td><a href="../sulfate/">sulfate</a></td>
<td>1..1</td>
<td>Concentration of sulfate in the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000424</td>
<td><a href="../sulfide/">sulfide</a></td>
<td>1..1</td>
<td>Concentration of sulfide in the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000419</td>
<td><a href="../tot_sulfur/">tot_sulfur</a></td>
<td>0..1</td>
<td>Concentration of total sulfur in the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000425</td>
<td><a href="../nitrate/">nitrate</a></td>
<td>1..1</td>
<td>Concentration of nitrate in the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000426</td>
<td><a href="../nitrite/">nitrite</a></td>
<td>0..1</td>
<td>Concentration of nitrite in the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000427</td>
<td><a href="../ammonium/">ammonium</a></td>
<td>0..1</td>
<td>Concentration of ammonium in the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000102</td>
<td><a href="../tot_nitro/">tot_nitro</a></td>
<td>0..1</td>
<td>Total nitrogen concentration of water samples, calculated by: total nitrogen = total dissolved nitrogen + particulate nitrogen. Can also be measured without filtering, reported as nitrogen</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000139</td>
<td><a href="../diss_iron/">diss_iron</a></td>
<td>0..1</td>
<td>Concentration of dissolved iron in the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000428</td>
<td><a href="../sodium/">sodium</a></td>
<td>0..1</td>
<td>Sodium concentration in the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000429</td>
<td><a href="../chloride/">chloride</a></td>
<td>0..1</td>
<td>Concentration of chloride in the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000430</td>
<td><a href="../potassium/">potassium</a></td>
<td>0..1</td>
<td>Concentration of potassium in the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000431</td>
<td><a href="../magnesium/">magnesium</a></td>
<td>0..1</td>
<td>Concentration of magnesium in the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000432</td>
<td><a href="../calcium/">calcium</a></td>
<td>0..1</td>
<td>Concentration of calcium in the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000105</td>
<td><a href="../tot_iron/">tot_iron</a></td>
<td>0..1</td>
<td>Concentration of total iron in the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000433</td>
<td><a href="../diss_org_carb/">diss_org_carb</a></td>
<td>0..1</td>
<td>Concentration of dissolved organic carbon in the sample, liquid portion of the sample, or aqueous phase of the fluid</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000434</td>
<td><a href="../diss_inorg_carb/">diss_inorg_carb</a></td>
<td>0..1</td>
<td>Dissolved inorganic carbon concentration in the sample, typically measured after filtering the sample using a 0.45 micrometer filter</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000106</td>
<td><a href="../diss_inorg_phosp/">diss_inorg_phosp</a></td>
<td>0..1</td>
<td>Concentration of dissolved inorganic phosphorus in the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000117</td>
<td><a href="../tot_phosp/">tot_phosp</a></td>
<td>0..1</td>
<td>Total phosphorus concentration in the sample, calculated by: total phosphorus = total dissolved phosphorus + particulate phosphorus</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000150</td>
<td><a href="../suspend_solids/">suspend_solids</a></td>
<td>0..*</td>
<td>Concentration of substances including a wide variety of material, such as silt, decaying plant and animal matter; can include multiple substances</td>
<td>{text};{float} {unit}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000435</td>
<td><a href="../density/">density</a></td>
<td>0..1</td>
<td>Density of the sample, which is its mass per unit volume (aka volumetric mass density)</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000436</td>
<td><a href="../diss_carb_dioxide/">diss_carb_dioxide</a></td>
<td>0..1</td>
<td>Concentration of dissolved carbon dioxide in the sample or liquid portion of the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000438</td>
<td><a href="../diss_oxygen_fluid/">diss_oxygen_fluid</a></td>
<td>0..1</td>
<td>Concentration of dissolved oxygen in the oil field produced fluids as it contributes to oxgen-corrosion and microbial activity (e.g. Mic).</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000152</td>
<td><a href="../vfa/">vfa</a></td>
<td>0..1</td>
<td>Concentration of Volatile Fatty Acids in the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000153</td>
<td><a href="../benzene/">benzene</a></td>
<td>0..1</td>
<td>Concentration of benzene in the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000154</td>
<td><a href="../toluene/">toluene</a></td>
<td>0..1</td>
<td>Concentration of toluene in the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000155</td>
<td><a href="../ethylbenzene/">ethylbenzene</a></td>
<td>0..1</td>
<td>Concentration of ethylbenzene in the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000156</td>
<td><a href="../xylene/">xylene</a></td>
<td>0..1</td>
<td>Concentration of xylene in the sample</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000157</td>
<td><a href="../api/">api</a></td>
<td>1..1</td>
<td>API gravity is a measure of how heavy or light a petroleum liquid is compared to water (source: https://en.wikipedia.org/wiki/API_gravity) (e.g. 31.1¬∞ API)</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000120</td>
<td><a href="../tan/">tan</a></td>
<td>0..1</td>
<td>Total Acid Number¬†(TAN) is a measurement of acidity that is determined by the amount of¬†potassium hydroxide¬†in milligrams that is needed to neutralize the acids in one gram of oil.¬†It is an important quality measurement of¬†crude oil. (source: https://en.wikipedia.org/wiki/Total_acid_number)</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000126</td>
<td><a href="../viscosity/">viscosity</a></td>
<td>0..1</td>
<td>A measure of oil's resistance¬†to gradual deformation by¬†shear stress¬†or¬†tensile stress (e.g. 3.5 cp; 100 ¬∞C)</td>
<td>{float} {unit};{float} {unit}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000127</td>
<td><a href="../pour_point/">pour_point</a></td>
<td>0..1</td>
<td>Temperature at which a liquid becomes semi solid and loses its flow characteristics. In crude oil a high¬†pour point¬†is generally associated with a high paraffin content, typically found in crude deriving from a larger proportion of plant material. (soure: https://en.wikipedia.org/wiki/pour_point)</td>
<td>None</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000131</td>
<td><a href="../saturates_pc/">saturates_pc</a></td>
<td>0..1</td>
<td>Saturate, Aromatic, Resin and Asphaltene¬†(SARA) is an analysis method that divides¬†crude oil¬†components according to their polarizability and polarity. There are three main methods to obtain SARA results. The most popular one is known as the Iatroscan TLC-FID and is referred to as IP-143 (source: https://en.wikipedia.org/wiki/Saturate,_aromatic,_resin_and_asphaltene)</td>
<td>{text};{float} {unit}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000133</td>
<td><a href="../aromatics_pc/">aromatics_pc</a></td>
<td>0..1</td>
<td>Saturate, Aromatic, Resin and Asphaltene¬†(SARA) is an analysis method that divides¬†crude oil¬†components according to their polarizability and polarity. There are three main methods to obtain SARA results. The most popular one is known as the Iatroscan TLC-FID and is referred to as IP-143 (source: https://en.wikipedia.org/wiki/Saturate,_aromatic,_resin_and_asphaltene)</td>
<td>{text};{float} {unit}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000134</td>
<td><a href="../resins_pc/">resins_pc</a></td>
<td>0..1</td>
<td>Saturate, Aromatic, Resin and Asphaltene¬†(SARA) is an analysis method that divides¬†crude oil¬†components according to their polarizability and polarity. There are three main methods to obtain SARA results. The most popular one is known as the Iatroscan TLC-FID and is referred to as IP-143 (source: https://en.wikipedia.org/wiki/Saturate,_aromatic,_resin_and_asphaltene)</td>
<td>{text};{float} {unit}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000135</td>
<td><a href="../asphaltenes_pc/">asphaltenes_pc</a></td>
<td>0..1</td>
<td>Saturate, Aromatic, Resin and Asphaltene¬†(SARA) is an analysis method that divides¬†crude oil¬†components according to their polarizability and polarity. There are three main methods to obtain SARA results. The most popular one is known as the Iatroscan TLC-FID and is referred to as IP-143 (source: https://en.wikipedia.org/wiki/Saturate,_aromatic,_resin_and_asphaltene)</td>
<td>{text};{float} {unit}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000752</td>
<td><a href="../misc_param/">misc_param</a></td>
<td>0..*</td>
<td>Any other measurement performed or parameter collected, that is not listed here</td>
<td>{text};{float} {unit}</td>
<td>.</td>
</tr>
<tr>
<td>MIXS:0000300</td>
<td><a href="../additional_info/">additional_info</a></td>
<td>0..1</td>
<td>Information that doesn't fit anywhere else. Can also be used to propose new entries for fields with controlled vocabulary</td>
<td>{text}</td>
<td>.</td>
</tr>
</tbody>
</table>
<h2 id="usages">Usages</h2>
<h2 id="identifier-and-mapping-information">Identifier and Mapping Information</h2></div>
            </div>
        </div>

        <footer class="col-md-12">
            <hr>
            <p>Documentation built with <a href="https://www.mkdocs.org/">MkDocs</a>.</p>
        </footer>
        <script>
            var base_url = "..",
                shortcuts = {"help": 191, "next": 78, "previous": 80, "search": 83};
        </script>
        <script src="../js/base.js" defer></script>
        <script src="../search/main.js" defer></script>

        <div class="modal" id="mkdocs_search_modal" tabindex="-1" role="dialog" aria-labelledby="searchModalLabel" aria-hidden="true">
    <div class="modal-dialog modal-lg">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="searchModalLabel">Search</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
                <p>From here you can search these documents. Enter your search terms below.</p>
                <form>
                    <div class="form-group">
                        <input type="search" class="form-control" placeholder="Search..." id="mkdocs-search-query" title="Type search term here">
                    </div>
                </form>
                <div id="mkdocs-search-results" data-no-results-text="No results found"></div>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div><div class="modal" id="mkdocs_keyboard_modal" tabindex="-1" role="dialog" aria-labelledby="keyboardModalLabel" aria-hidden="true">
    <div class="modal-dialog">
        <div class="modal-content">
            <div class="modal-header">
                <h4 class="modal-title" id="keyboardModalLabel">Keyboard Shortcuts</h4>
                <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">&times;</span><span class="sr-only">Close</span></button>
            </div>
            <div class="modal-body">
              <table class="table">
                <thead>
                  <tr>
                    <th style="width: 20%;">Keys</th>
                    <th>Action</th>
                  </tr>
                </thead>
                <tbody>
                  <tr>
                    <td class="help shortcut"><kbd>?</kbd></td>
                    <td>Open this help</td>
                  </tr>
                  <tr>
                    <td class="next shortcut"><kbd>n</kbd></td>
                    <td>Next page</td>
                  </tr>
                  <tr>
                    <td class="prev shortcut"><kbd>p</kbd></td>
                    <td>Previous page</td>
                  </tr>
                  <tr>
                    <td class="search shortcut"><kbd>s</kbd></td>
                    <td>Search</td>
                  </tr>
                </tbody>
              </table>
            </div>
            <div class="modal-footer">
            </div>
        </div>
    </div>
</div>

    </body>
</html>
